require 'multi_json'
require 'minitest/mock'
require 'minitest/stub_any_instance'
require_relative '../test_case'

##
# WARNING
#
# This class does not test much. There is full test coverage for Resource Index in the http://github.com/ncbo/resource_index project.
# What happens here is that we stub the methods from the Resource Index project so that they always return known JSON.
# This JSON is stored in the constants below (RESOURCES, COUNTS, etc). This means that if the output of the Resource Index API
# changes, these tests will not fail. We would need to get the new JSON from the modified code and replace what is here.
# However, these tests do cover some basic functionality like 404 responses and that the JSON ouput isn't mangled by the serializer.
class TestResourceIndexController < TestCase
  RESOURCES = %q([{"name":"Adverse Event Reporting System Data","acronym":"AERS","mainField":"AERS_DRUG_NAMES","homepage":"http://www.aersdata.com/","lookupURL":"http://www.aersdata.com/Quicksearch/QuickAersdata_list.php?a=search&value=1&SearchField=1&SearchOption=Equals&SearchFor=","description":"Adverse Event Reporting System (AERS) Databases are validated and normalized to provide consistent results to our users that depend on data such as this for research into Adverse Events reported to the FDA by Consumers, Doctors and many other health care related professionals.","logo":"http://www.aersdata.com/images/newidex.jpg","count":1172881,"updated":"2012-03-06 00:13:39 -0800","completed":"2014-01-26 20:46:17 -0800","fields":{"AERS_DRUG_INDI":{"name":"DRUG_INDI","ontology":null,"weight":1.0},"AERS_DRUG_NAMES":{"name":"DRUG_NAMES","ontology":null,"weight":1.0},"AERS_DRUG_CHAR":{"name":"DRUG_CHAR","ontology":null,"weight":0.9},"AERS_DRUG_ADMIN_ROUTE":{"name":"DRUG_ADMIN_ROUTE","ontology":null,"weight":0.9},"AERS_REAC":{"name":"REAC","ontology":null,"weight":1.0}}},{"name":"AgingGenesDB (via NIF)","acronym":"AGDB","mainField":"AGDB_Gene_Name","homepage":"http://www.uwaging.org/genesdb/","lookupURL":"http://uwaging.org/genesdb/search?anyword=","description":"This database provides a searchable and browsable list of aging related genes and their effects on longevity. Other information can also be found including: organism, aging phenotype, allele type, strain, gene function, phenotypes, mutant, and homologs. The Biology of Aging at the University of Washington is the home of the Nathan Shock Center of Excellence in the Basic Biology of Aging and the Genetic Approaches to Aging Training Grant. The University of Washington is a major center for research in Gerontology, and these programs help to enrich the infrastructure for this work.","logo":"http://neurolex.org/w/images/7/71/Aginggenesdb.png","count":346,"updated":"2012-09-04 22:59:14 -0700","completed":"2013-09-26 00:02:41 -0700","fields":{"AGDB_Keywords":{"name":"Keywords","ontology":null,"weight":0.7},"AGDB_Organism":{"name":"Organism","ontology":null,"weight":0.9},"AGDB_Primary_Reference":{"name":"Primary_Reference","ontology":null,"weight":0.4},"AGDB_Gene_Name":{"name":"Gene_Name","ontology":null,"weight":1.0},"AGDB_Other_Phenotypes":{"name":"Other_Phenotypes","ontology":null,"weight":0.7},"AGDB_Aging_Phenotype":{"name":"Aging_Phenotype","ontology":null,"weight":0.9},"AGDB_Gene_function":{"name":"Gene_function","ontology":null,"weight":0.7},"AGDB_Description":{"name":"Description","ontology":null,"weight":0.7}}},{"name":"Antibody Registry (via NIF)","acronym":"ABR","mainField":"ABR_Antibody_Name","homepage":"http://antibodyregistry.org/","lookupURL":"http://antibodyregistry.org/","description":"The Antibody Registry aims to serve the scientific community by providing a very large list of antibodies (>900,000) as well as a set of unique identifiers for commercial and non-commercial antibody reagents allowing scientists to more easily keep track of data generated using a specific antibody. Sourcing and availability information are based on a semantic annotation pipeline supported by the Domeo web annotation toolkit, http://annotationframework.org/. This public registry contains antibody product information organized according to genes, species, reagent types (antibodies, recombinant proteins, ELISA, siRNA, cDNA clones). Data is provided in many formats so that authors of biological papers, text mining tools and funding agencies can quickly and accurately identify the antibody reagents they and their colleagues used. The Antibody Registry allows any user to submit a new antibody or set of antibodies to the registry via a web form, or via a spreadsheet upload.","logo":"http://neurolex.org/w/images/f/fe/BA45EBE0-E863-4A48-8495-91C8D4DAF22E.jpg","count":1510381,"updated":"2013-06-27 04:54:49 -0700","completed":"2014-03-20 06:00:46 -0700","fields":{"ABR_Vendor":{"name":"Vendor","ontology":null,"weight":0.0},"ABR_Cat_Num":{"name":"Cat_Num","ontology":null,"weight":0.0},"ABR_Antibody_ID":{"name":"Antibody_ID","ontology":null,"weight":0.0},"ABR_Comments":{"name":"Comments","ontology":null,"weight":0.0},"ABR_Source_Organsim":{"name":"Source_Organsim","ontology":null,"weight":0.8},"ABR_Clonality":{"name":"Clonality","ontology":null,"weight":0.0},"ABR_Antibody_Name":{"name":"Antibody_Name","ontology":null,"weight":1.0},"ABR_Antibody_Target":{"name":"Antibody_Target","ontology":null,"weight":0.9},"ABR_Clone_ID":{"name":"Clone_ID","ontology":null,"weight":0.0}}},{"name":"ArrayExpress","acronym":"AE","mainField":"AE_name","homepage":"http://www.ebi.ac.uk/arrayexpress/","lookupURL":"http://www.ebi.ac.uk/arrayexpress/experiments/","description":"ArrayExpress is a public repository for microarray data, which is aimed at storing MIAME-compliant data in accordance with MGED recommendations. The ArrayExpress Data Warehouse stores gene-indexed expression profiles from a curated subset of experiments in the repository.","logo":"http://www.ebi.ac.uk/microarray-as/aer/include/aelogo.png","count":35877,"updated":"2013-02-05 21:35:46 -0800","completed":"2013-09-26 05:13:05 -0700","fields":{"AE_name":{"name":"name","ontology":null,"weight":1.0},"AE_description":{"name":"description","ontology":null,"weight":0.8},"AE_species":{"name":"species","ontology":1132,"weight":1.0},"AE_experiment_type":{"name":"experiment_type","ontology":1136,"weight":0.9}}},{"name":"ARRS GoldMiner","acronym":"GM","mainField":"GM_title","homepage":"http://goldminer.arrs.org/","lookupURL":"http://goldminer.arrs.org/link.php?id=","description":"ARRS GoldMiner provides instant access to images published in selected peer-reviewed radiology journals. This new, web-based system allows viewers to search for images by findings, anatomy, imaging technique, and patient age and sex.","logo":"http://goldminer.arrs.org/image/goldminer.gif","count":1155,"updated":"2012-03-06 05:56:19 -0800","completed":"2013-09-30 22:08:42 -0700","fields":{"GM_caption":{"name":"caption","ontology":null,"weight":0.8},"GM_title":{"name":"title","ontology":null,"weight":1.0}}},{"name":"AutDB (via NIF)","acronym":"AUTDB","mainField":"AUTDB_Animal_Model","homepage":"http://autism.mindspec.org/autdb/Welcome.do","lookupURL":"http://autism.mindspec.org/animalmodel/","description":"AutDB is a public database of data from scientific publications for autism spectrum disorders. It also includes several interactive modules that highlight the genes implicated in autism: Human Gene, Animal Models, and Protein Interactions.","logo":"http://autism.mindspec.org/autdb/images/logo1.png","count":268,"updated":"2012-10-11 04:27:40 -0700","completed":"2013-09-26 00:01:28 -0700","fields":{"AUTDB_Aliases":{"name":"Aliases","ontology":null,"weight":0.7},"AUTDB_Animal_Model":{"name":"Animal_Model","ontology":null,"weight":1.0},"AUTDB_Phenotype_Profile":{"name":"Phenotype_Profile","ontology":null,"weight":0.9},"AUTDB_References_for_Phenotype":{"name":"References_for_Phenotype","ontology":null,"weight":0.5},"AUTDB_Gene_Name":{"name":"Gene_Name","ontology":null,"weight":1.0},"AUTDB_Gene_Symbol":{"name":"Gene_Symbol","ontology":null,"weight":1.0}}},{"name":"BioGRID (via NIF)","acronym":"BGRD","mainField":"BGRD_Interactor_A_and_Interactor_B","homepage":"http://thebiogrid.org/","lookupURL":"http://thebiogrid.org/search.php?search=","description":"An online interaction repository of raw protein and genetic interactions from major model organism species, with data compiled through comprehensive curation efforts. The current version searches over 36,000 articles containing over 560,000 interactions. Complete coverage of the entire literature is maintained for budding yeast (S. cerevisiae), fission yeast (S. pombe) and thale cress (A. thaliana), and efforts to expand curation across multiple metazoan species are underway.","logo":"http://neurolex.org/w/images/2/27/BioGRID.PNG","count":583616,"updated":"2013-08-06 00:55:45 -0700","completed":"2014-03-21 16:38:25 -0700","fields":{"BGRD_Interaction_Types":{"name":"Interaction_Types","ontology":null,"weight":0.5},"BGRD_Source_Database":{"name":"Source_Database","ontology":null,"weight":0.0},"BGRD_Organism_B":{"name":"Organism_B","ontology":null,"weight":0.9},"BGRD_Confidence_Values":{"name":"Confidence_Values","ontology":null,"weight":0.0},"BGRD_Organism_A":{"name":"Organism_A","ontology":null,"weight":0.9},"BGRD_Interactor_B":{"name":"Interactor_B","ontology":null,"weight":0.9},"BGRD_Interaction_Detection_Method":{"name":"Interaction_Detection_Method","ontology":null,"weight":0.5},"BGRD_Interactor_A":{"name":"Interactor_A","ontology":null,"weight":1.0},"BGRD_Interactor_A_and_Interactor_B":{"name":"Interactor_A_and_Interactor_B","ontology":null,"weight":0.0}}},{"name":"BioModels","acronym":"BIOM","mainField":"BIOM_Name","homepage":"http://www.ebi.ac.uk/biomodels-main/","lookupURL":"http://www.ebi.ac.uk/biomodels-main/","description":"BioModels Database is a repository of peer-reviewed, published, computational models. These mathematical models are primarily from the field of systems biology, but more generally are those of biological interest. This resource allows biologists to store, search and retrieve published mathematical models. In addition, models in the database can be used to generate sub-models, can be simulated online, and can be converted between different representational formats. This resource also features programmatic access via Web Services.","logo":"http://www.ebi.ac.uk/biomodels/icons/logo_small_border.png","count":863,"updated":"2012-11-27 01:28:30 -0800","completed":"2013-09-25 23:45:51 -0700","fields":{"BIOM_Pathways":{"name":"Pathways","ontology":null,"weight":0.7},"BIOM_Name":{"name":"Name","ontology":null,"weight":1.0},"BIOM_Publication_Id":{"name":"Publication_Id","ontology":null,"weight":0.5},"BIOM_Species":{"name":"Species","ontology":null,"weight":0.7},"BIOM_Notes":{"name":"Notes","ontology":null,"weight":0.9},"BIOM_Go_Terms":{"name":"Go_Terms","ontology":1070,"weight":0.7}}},{"name":"Biositemaps","acronym":"BSM","mainField":"BSM_name","homepage":"http://www.ncbcs.org/biositemaps/","lookupURL":"http://biositemaps.bioontology.org/editor/BiositemapBrowser.html?file=","description":"Biositemaps represent a mechanism for computational biologists and bio-informaticians to openly broadcast and retrieve meta-data about biomedical data, tools and services (i.e., biomedical resources) over the Internet.","logo":"http://www.ncbcs.org/biositemaps/images/BioSitemaps_Logo_alone.jpg","count":2434,"updated":"2012-03-06 00:17:48 -0800","completed":"2013-10-02 09:41:17 -0700","fields":{"BSM_description":{"name":"description","ontology":null,"weight":0.8},"BSM_keywords":{"name":"keywords","ontology":null,"weight":1.0},"BSM_name":{"name":"name","ontology":null,"weight":1.0},"BSM_url":{"name":"url","ontology":null,"weight":0.0},"BSM_resource_type":{"name":"resource_type","ontology":1104,"weight":0.8}}},{"name":"caArray","acronym":"CARY","mainField":"CARY_Title","homepage":"https://array.nci.nih.gov/caarray/home.action","lookupURL":"https://array-stage.nci.nih.gov/caarray/project/","description":"caArray is an open-source, web and programmatically accessible array data management system. caArray guides the annotation and exchange of array data using a federated model of local installations whose results are shareable across the cancer Biomedical Informatics Grid (caBIGâ„¢). caArray furthers translational cancer research through acquisition, dissemination and aggregation of semantically interoperable array data to support subsequent analysis by tools and services on and off the Grid. As array technology advances and matures, caArray will extend its logical library of assay management.","logo":"https://array.nci.nih.gov/caarray/images/logo_caarray.gif","count":182,"updated":"2012-06-08 02:13:14 -0700","completed":"2013-09-26 00:00:00 -0700","fields":{"CARY_Disease_State":{"name":"Disease_State","ontology":null,"weight":0.8},"CARY_Tissue_Sites":{"name":"Tissue_Sites","ontology":null,"weight":0.8},"CARY_Description":{"name":"Description","ontology":null,"weight":0.8},"CARY_Cell_Types":{"name":"Cell_Types","ontology":null,"weight":0.8},"CARY_Title":{"name":"Title","ontology":null,"weight":1.0},"CARY_Organism":{"name":"Organism","ontology":null,"weight":0.9}}},{"name":"caNanoLab","acronym":"CANANO","mainField":"CANANO_Organization","homepage":"http://cananolab.nci.nih.gov/caNanoLab/home.jsp","lookupURL":"NA","description":"caNanoLab is a data sharing portal designed to facilitate information sharing in the biomedical nanotechnology research community to expedite and validate the use of nanotechnology in biomedicine.","logo":"http://cananolab.nci.nih.gov/caNanoLab/images/appLogo-nanolab.gif","count":895,"updated":"2012-03-08 04:53:19 -0800","completed":"2013-10-03 23:07:20 -0700","fields":{"CANANO_Nanoparticle_Entity_Type":{"name":"Nanoparticle_Entity_Type","ontology":null,"weight":0.8},"CANANO_Sample":{"name":"Sample","ontology":null,"weight":0.1},"CANANO_Function":{"name":"Function","ontology":null,"weight":0.65},"CANANO_Chemical_Association":{"name":"Chemical_Association","ontology":null,"weight":0.7},"CANANO_Point_of_contact":{"name":"Point_of_contact","ontology":null,"weight":0.95},"CANANO_Condition":{"name":"Condition","ontology":null,"weight":0.35},"CANANO_Publication":{"name":"Publication","ontology":null,"weight":0.9},"CANANO_url":{"name":"url","ontology":null,"weight":0.0},"CANANO_Activation_Method":{"name":"Activation_Method","ontology":null,"weight":0.45},"CANANO_Chemical_Composition":{"name":"Chemical_Composition","ontology":null,"weight":0.85},"CANANO_Author":{"name":"Author","ontology":null,"weight":0.95},"CANANO_Keyword":{"name":"Keyword","ontology":null,"weight":0.35},"CANANO_Instruments_and_Techniques":{"name":"Instruments_and_Techniques","ontology":null,"weight":0.5},"CANANO_Datum":{"name":"Datum","ontology":null,"weight":0.4},"CANANO_biological_Characterization":{"name":"biological_Characterization","ontology":null,"weight":0.55},"CANANO_Physicochemical_Characterization":{"name":"Physicochemical_Characterization","ontology":null,"weight":0.6},"CANANO_Functionalizing_Entity_Type":{"name":"Functionalizing_Entity_Type","ontology":null,"weight":0.75},"CANANO_Organization":{"name":"Organization","ontology":null,"weight":1.0},"CANANO_Protocol":{"name":"Protocol","ontology":null,"weight":0.3}}},{"name":"Cell Centered Database (via NIF)","acronym":"CCDB","mainField":"CCDB_ProjectName","homepage":"http://ccdb.ucsd.edu","lookupURL":"http://ccdb.ucsd.edu/sand/main?event=displaySum&mpid=","description":"The Cell Centered Database is a publicly accessible resource for high resolution 2D, 3D and 4D data from light and electron microscopy, including correlated imaging. Techniques range from wide field mosaics taken with multiphoton microscopy to 3D reconstructions of cellular ultrastructure using electron tomography. Contributions from the community are welcome. The CCDB was designed around the process of reconstruction from 2D micrographs, capturing key steps in the process from experiment to analysis. The CCDB refers to the set of images taken from the as the Microscopy Product. The microscopy product refers to a set of related 2D images taken by light (epifluorescence, transmitted light, confocal or multiphoton) or electron microscopy (conventional or high voltage transmission electron microscopy). These image sets may comprise a tilt series, optical section series, through focus series, serial sections, mosaics, time series or a set of survey sections taken in a single microscopy session that are not related in any systematic way. A given set of data may be more than one product, for example, it is possible for a set of images to be both a mosaic and a tilt series. The Microscopy Product ID serves as the accession number for the CCDB. All microscopy products must belong to a project and be stored along with key specimen preparation details. Each project receives a unique Project ID that groups together related microscopy products. Many of the datasets come from published literature, but publication is not a prerequisite for inclusion in the CCDB. Any datasets that are of high quality and interest to the scientific community can be included in the CCDB. The goal of the CCDB project is to make unique and valuable datasets available to the scientific community for visualization, reuse and reanalysis. Data in the CCDB can be accessed by performing a Search or by browsing through our Gallery. Data in the CCDB may be downloaded freely and reused for non-profit use within the terms of our usage agreement.","logo":"http://neurolex.org/w/images/a/a4/Ccdb.jpg","count":431,"updated":"2012-06-26 04:06:46 -0700","completed":"2013-09-26 00:03:44 -0700","fields":{"CCDB_CellType":{"name":"CellType","ontology":null,"weight":0.9},"CCDB_ProjectName":{"name":"ProjectName","ontology":null,"weight":1.0},"CCDB_Microscope_Type":{"name":"Microscope_Type","ontology":null,"weight":0.5},"CCDB_Age":{"name":"Age","ontology":null,"weight":0.5},"CCDB_Region":{"name":"Region","ontology":null,"weight":0.9},"CCDB_Species":{"name":"Species","ontology":null,"weight":0.8}}},{"name":"CellImageLibrary (via NIF)","acronym":"CIL","mainField":"CIL_Image_description","homepage":"http://cellimages.ascb.org/","lookupURL":"http://cellimagelibrary.org/images/","description":"The Cell an Image Library is a freely accessible, easy-to-search, public repository of vetted and annotated images, videos, and animations of cells from a variety of organisms, showcasing cell architecture, intracellular functionalities, and both normal and abnormal processes.","logo":"http://neurolex.org/w/images/1/17/The_Cell_-_an_image_library.PNG","count":4282,"updated":"2014-03-24 02:46:11 -0700","completed":"2014-03-24 03:07:30 -0700","fields":{"CIL_CellType":{"name":"CellType","ontology":null,"weight":0.0},"CIL_CellLine":{"name":"CellLine","ontology":null,"weight":0.0},"CIL_Image_description":{"name":"Image_description","ontology":null,"weight":1.0},"CIL_Cellular_component":{"name":"Cellular_component","ontology":null,"weight":0.9},"CIL_Organism":{"name":"Organism","ontology":null,"weight":0.5},"CIL_Investigator_name":{"name":"Investigator_name","ontology":null,"weight":0.7}}},{"name":"ClinicalTrials.gov","acronym":"CT","mainField":"CT_brief_title","homepage":"http://clinicaltrials.gov/","lookupURL":"http://clinicaltrials.gov/ct/show/","description":"ClinicalTrials.gov provides regularly updated information about federally and privately supported clinical research in human volunteers. ClinicalTrials.gov gives you information about a trial's purpose, who may participate, locations, and phone numbers for more details. The information provided on ClinicalTrials.gov should be used in conjunction with advice from health care professionals. Before searching, you may want to learn more about clinical trials.","logo":"http://clinicaltrials.gov/ct2/html/images/frame/title.gif","count":122591,"updated":"2012-03-08 21:00:13 -0800","completed":"2013-10-02 07:18:39 -0700","fields":{"CT_intervention":{"name":"intervention","ontology":null,"weight":0.6},"CT_official_title":{"name":"official_title","ontology":null,"weight":1.0},"CT_brief_summary":{"name":"brief_summary","ontology":null,"weight":0.8},"CT_condition":{"name":"condition","ontology":null,"weight":0.6},"CT_brief_title":{"name":"brief_title","ontology":null,"weight":1.0},"CT_detailed_description":{"name":"detailed_description","ontology":null,"weight":0.7}}},{"name":"Conserved Domain Database (CDD)","acronym":"CDD","mainField":"CDD_title","homepage":"http://www.ncbi.nlm.nih.gov/Structure/cdd/cdd.shtml","lookupURL":"http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=","description":"The Conserved Domain Database (CDD) contains protein domain models imported from outside sources, such as Pfam and SMART, and curated at NCBI. CDD contains over 12,000 such models and is linked to other NCBI databases, including protein sequences, bibliographic citations, and taxonomy.","logo":"http://www.ncbi.nlm.nih.gov/Structure/cdd/cdd2.gif","count":46346,"updated":"2012-03-09 03:25:52 -0800","completed":"2013-10-23 02:03:22 -0700","fields":{"CDD_abstract":{"name":"abstract","ontology":null,"weight":0.6},"CDD_title":{"name":"title","ontology":null,"weight":1.0},"CDD_uid":{"name":"uid","ontology":null,"weight":0.0}}},{"name":"Coriell Cell Repository (via NIF)","acronym":"CCR","mainField":"CCR_Collection","homepage":"http://ccr.coriell.org/","lookupURL":"http://ccr.coriell.org/Sections/Search/Sample_Detail.aspx?Ref=","description":"The Coriell Cell Repositories provide essential research reagents to the scientific community by establishing, verifying, maintaining, and distributing cell cultures and DNA derived from cell cultures.","logo":"http://neurolex.org/w/images/thumb/5/56/CCR.PNG/180px-CCR.PNG","count":13842,"updated":"2013-07-30 04:20:28 -0700","completed":"2014-03-21 02:20:28 -0700","fields":{"CCR_Collection":{"name":"Collection","ontology":null,"weight":0.0},"CCR_Sample_Availability":{"name":"Sample_Availability","ontology":null,"weight":0.0},"CCR_Reference":{"name":"Reference","ontology":null,"weight":0.0},"CCR_Biopsy_source":{"name":"Biopsy_source","ontology":null,"weight":1.0},"CCR_Genetic_Mutation":{"name":"Genetic_Mutation","ontology":null,"weight":0.5},"CCR_Disease_Description":{"name":"Disease_Description","ontology":null,"weight":0.9},"CCR_Comments":{"name":"Comments","ontology":null,"weight":0.7},"CCR_Relation_to_proband":{"name":"Relation_to_proband","ontology":null,"weight":0.0}}},{"name":"CTD ChemDisease (via NIF)","acronym":"CTDCD","mainField":"CTDCD_Disease","homepage":"http://ctdbase.org/","lookupURL":"http://ctdbase.org/detail.go?type=disease&acc=","description":"A public database that enhances understanding about the effects of environmental chemicals on human health. In detail, it contains information about gene/protein-disease associations, chemical-disease associations, interactions between chemicals and genes/proteins, as well as the related pathways.","logo":"http://neurolex.org/w/images/b/bb/CTD.PNG","count":5377,"updated":"2013-08-09 02:37:31 -0700","completed":"2014-03-18 00:13:11 -0700","fields":{"CTDCD_Inference_Score":{"name":"Inference_Score","ontology":null,"weight":0.0},"CTDCD_Direct_Evidence":{"name":"Direct_Evidence","ontology":null,"weight":0.0},"CTDCD_Chemical_Name":{"name":"Chemical_Name","ontology":null,"weight":0.9},"CTDCD_Disease":{"name":"Disease","ontology":null,"weight":1.0},"CTDCD_References":{"name":"References","ontology":null,"weight":0.0},"CTDCD_Inference_Gene_Symbol":{"name":"Inference_Gene_Symbol","ontology":null,"weight":0.9},"CTDCD_OMIM_IDs":{"name":"OMIM_IDs","ontology":null,"weight":0.0}}},{"name":"CTD ChemGene (via NIF)","acronym":"CTDCG","mainField":"CTDCG_Gene_Symbol","homepage":"http://ctdbase.org/","lookupURL":"http://ctdbase.org/detail.go?type=gene&acc=","description":"A public database that enhances understanding about the effects of environmental chemicals on human health. In detail, it contains information about gene/protein-disease associations, chemical-disease associations, interactions between chemicals and genes/proteins, as well as the related pathways.","logo":"http://neurolex.org/w/images/b/bb/CTD.PNG","count":6185,"updated":"2013-08-02 00:01:50 -0700","completed":"2014-03-23 23:46:37 -0700","fields":{"CTDCG_Chemical_Name":{"name":"Chemical_Name","ontology":null,"weight":0.9},"CTDCG_Gene_Forms":{"name":"Gene_Forms","ontology":null,"weight":0.5},"CTDCG_Type_of_Interactions":{"name":"Type_of_Interactions","ontology":null,"weight":0.8},"CTDCG_Interaction":{"name":"Interaction","ontology":null,"weight":0.9},"CTDCG_References":{"name":"References","ontology":null,"weight":0.0},"CTDCG_Gene_Symbol":{"name":"Gene_Symbol","ontology":null,"weight":1.0},"CTDCG_Organism":{"name":"Organism","ontology":null,"weight":0.7}}},{"name":"CTD DiseasePathway (via NIF)","acronym":"CTDDP","mainField":"CTDDP_Disease_Name","homepage":"http://ctdbase.org/","lookupURL":"http://ctdbase.org/detail.go?type=disease&acc=","description":"A public database that enhances understanding about the effects of environmental chemicals on human health. In detail, it contains information about gene/protein-disease associations, chemical-disease associations, interactions between chemicals and genes/proteins, as well as the related pathways.","logo":"http://neurolex.org/w/images/b/bb/CTD.PNG","count":3385,"updated":"2013-08-21 04:11:51 -0700","completed":"2014-03-23 23:21:19 -0700","fields":{"CTDDP_Inference_Gene_Symbol":{"name":"Inference_Gene_Symbol","ontology":null,"weight":0.9},"CTDDP_Disease_Name":{"name":"Disease_Name","ontology":null,"weight":0.9},"CTDDP_Pathway_Name":{"name":"Pathway_Name","ontology":null,"weight":1.0}}},{"name":"Database of Genotypes and Phenotypes","acronym":"GAP","mainField":"GAP_study_name","homepage":"http://www.ncbi.nlm.nih.gov/sites/entrez?Db=gap","lookupURL":"http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=","description":"The database of Genotypes and Phenotypes (dbGaP) was developed to archive and distribute the results of studies that have investigated the interaction of genotype and phenotype. Such studies include genome-wide association studies, medical sequencing, molecular diagnostic assays, as well as association between genotype and non-clinical traits.","logo":"http://www.ncbi.nlm.nih.gov/projects/gap/images/dbGaP_logo_final.jpg","count":269,"updated":"2012-02-05 20:41:07 -0800","completed":"2013-10-23 02:04:57 -0700","fields":{"GAP_study_disease_list_msh":{"name":"study_disease_list_msh","ontology":1351,"weight":1.0},"GAP_study_disease_list_snomedct":{"name":"study_disease_list_snomedct","ontology":1353,"weight":1.0},"GAP_study_name":{"name":"study_name","ontology":null,"weight":1.0},"GAP_uid":{"name":"uid","ontology":null,"weight":0.0},"GAP_study_disease_list_ncit":{"name":"study_disease_list_ncit","ontology":1032,"weight":1.0}}},{"name":"Drug Related Gene Database (via NIF)","acronym":"DRGDB","mainField":"DRGDB_Gene_Name","homepage":"https://confluence.crbs.ucsd.edu/display/NIF/DRG","lookupURL":"http://www.ncbi.nlm.nih.gov/pubmed/","description":"The Related Gene (DRG) Database gene expression data are extracted from published journal articles that test hypotheses relevant to the neuroscience of addiction and addictive behavior. Data types include the effects of a particular drug, strain, or knock out on a particular gene, in a particular anatomical region.","logo":"http://neurolex.org/w/images/a/ae/DRG.PNG","count":47840,"updated":"2013-01-20 22:53:32 -0800","completed":"2013-09-26 08:21:46 -0700","fields":{"DRGDB_Probe_ID":{"name":"Probe_ID","ontology":null,"weight":0.0},"DRGDB_Protocol_Type":{"name":"Protocol_Type","ontology":null,"weight":0.0},"DRGDB_Exp_vs_Control":{"name":"Exp_vs_Control","ontology":null,"weight":0.8},"DRGDB_Treatment":{"name":"Treatment","ontology":null,"weight":0.9},"DRGDB_Gene_Name":{"name":"Gene_Name","ontology":null,"weight":1.0},"DRGDB_Publication":{"name":"Publication","ontology":null,"weight":0.0},"DRGDB_Brain_region":{"name":"Brain_region","ontology":null,"weight":0.9},"DRGDB_Table":{"name":"Table","ontology":null,"weight":0.0},"DRGDB_Expression":{"name":"Expression","ontology":null,"weight":0.0}}},{"name":"DrugBank","acronym":"DBK","mainField":"DBK_name","homepage":"http://www.drugbank.ca/","lookupURL":"http://www.drugbank.ca/drugs/","description":"DrugBank is offered to the public as a freely available resource. Use and re-distribution of the data, in whole or in part, for commercial purposes requires explicit permission of the authors and explicit acknowledgment of the source material (DrugBank) and the original publication.","logo":"http://www.drugbank.ca/images/drugbankcover.png","count":4774,"updated":"2012-03-12 23:12:19 -0700","completed":"2013-10-01 04:35:02 -0700","fields":{"DBK_indication":{"name":"indication","ontology":null,"weight":0.8},"DBK_brandnames":{"name":"brandnames","ontology":null,"weight":0.8},"DBK_ChEBI_ID":{"name":"ChEBI_ID","ontology":1007,"weight":1.0},"DBK_drug_category":{"name":"drug_category","ontology":null,"weight":0.8},"DBK_name":{"name":"name","ontology":null,"weight":1.0},"DBK_synonyms":{"name":"synonyms","ontology":null,"weight":0.8},"DBK_target_go_classification":{"name":"target_go_classification","ontology":1070,"weight":1.0}}},{"name":"Entrez Gene (via NIF)","acronym":"EG","mainField":"EG_Gene_Symbol","homepage":"http://www.ncbi.nlm.nih.gov/gene","lookupURL":"http://www.ncbi.nlm.nih.gov/gene?term=","description":"EntrezGene (NCBI Gene) is a searchable database of genes, from RefSeq genomes, and defined by sequence and/or located in the NCBI Map Viewer. The content of EntrezGene represents the result of curation and automated integration of data from NCBI's Reference Sequence project (RefSeq), from collaborating model organism databases, and from many other databases available from NCBI.","logo":"http://neurolex.org/w/images/f/f9/Ncbi.png","count":8124584,"updated":"2013-07-08 01:03:28 -0700","completed":"2014-04-01 19:32:34 -0700","fields":{"EG_Gene_Symbol":{"name":"Gene_Symbol","ontology":null,"weight":1.0},"EG_Gene_ID":{"name":"Gene_ID","ontology":null,"weight":0.0},"EG_Description":{"name":"Description","ontology":null,"weight":0.9},"EG_Type":{"name":"Type","ontology":null,"weight":0.0},"EG_Organism":{"name":"Organism","ontology":null,"weight":0.7},"EG_Synonyms":{"name":"Synonyms","ontology":null,"weight":0.9}}},{"name":"GEMMA (via NIF)","acronym":"GEMMA","mainField":"GEMMA_Source","homepage":"http://www.chibi.ubc.ca/Gemma/home.html","lookupURL":"http://www.chibi.ubc.ca/Gemma/expressionExperiment/showExpressionExperiment.html?id=","description":"Gemma is a database and software system for the meta-analysis of gene expression data. Gemma contains data from hundreds of public microarray data sets, referencing hundreds of published papers. Users can search, access and visualize coexpression and differential expression results.","logo":"http://neurolex.org/w/images/0/08/Gemma.gif","count":634,"updated":"2013-01-07 01:26:17 -0800","completed":"2013-10-02 08:09:38 -0700","fields":{"GEMMA_differentially_expressed_genes_in_listed_tissues":{"name":"differentially_expressed_genes_in_listed_tissues","ontology":null,"weight":0.9},"GEMMA_Array_Platform":{"name":"Array_Platform","ontology":null,"weight":0.4},"GEMMA_Tissue":{"name":"Tissue","ontology":null,"weight":0.9},"GEMMA_Organism":{"name":"Organism","ontology":null,"weight":0.9},"GEMMA_Description":{"name":"Description","ontology":null,"weight":0.7},"GEMMA_Source":{"name":"Source","ontology":null,"weight":1.0}}},{"name":"Gene Expression Omnibus DataSets","acronym":"GEO","mainField":"GEO_title","homepage":"http://www.ncbi.nlm.nih.gov/geo/","lookupURL":"http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=","description":"A gene expression/molecular abundance repository supporting  MIAME compliant data submissions, and a curated, online resource for gene expression data browsing, query and retrieval.","logo":"http://www.ncbi.nlm.nih.gov/projects/geo/img/geo_main.gif","count":31483,"updated":"2012-03-09 02:13:30 -0800","completed":"2013-10-23 08:36:34 -0700","fields":{"GEO_title":{"name":"title","ontology":null,"weight":1.0},"GEO_summary":{"name":"summary","ontology":null,"weight":0.8},"GEO_organism":{"name":"organism","ontology":1132,"weight":1.0},"GEO_uid":{"name":"uid","ontology":null,"weight":0.0}}},{"name":"GeneNetwork (via NIF)","acronym":"GENET","mainField":"GENET_Gene_Name","homepage":"http://www.genenetwork.org/","lookupURL":"http://www.genenetwork.org/webqtl/main.py?FormID=showDatabase&database=","description":"The GeneNetwork consists of a set of linked resources for systems genetics. It has been designed for multiscale integration of networks of genes, transcripts, and traits such as toxicity, cancer susceptibility, and behavior. This open resource combines more than 25 years of legacy data generated by hundreds of scientists with full genome sequence and deep transcriptome data sets. WebQTL is the leading GeneNetwork module, and has been optimized for on-line analysis of traits that are controlled by combinations of allelic variants and environmental factors. WebQTL exploits several permanent genetic reference populations (GRP) of mouse (BXD, LXS, etc.), rat (HXB), and Arabidopsis (BayXSha). Each GRP is accompanied by dense genetic maps used to locate modifiers that cause downstream differences in expression and higher-order phenotypes, including disease susceptibility.","logo":"http://neurolex.org/w/images/a/a2/GeneNetwork.png","count":386156,"updated":"2012-10-10 06:16:52 -0700","completed":"2014-02-13 02:36:05 -0800","fields":{"GENET_Organism":{"name":"Organism","ontology":null,"weight":0.8},"GENET_Tissue":{"name":"Tissue","ontology":null,"weight":0.9},"GENET_Chromosome_location":{"name":"Chromosome_location","ontology":null,"weight":0.7},"GENET_Gene_Name":{"name":"Gene_Name","ontology":null,"weight":1.0},"GENET_Originating_database":{"name":"Originating_database","ontology":null,"weight":0.5}}},{"name":"Integrated Disease View (via NIF)","acronym":"IDV","mainField":"IDV_Name","homepage":"http://www.ncbi.nlm.nih.gov/pubmedhealth","lookupURL":"","description":"The NIF Integrated Disease View is a virtual database currently indexing authoritative information on disease and treatment options from NINDS Disorder List - http://www.ninds.nih.gov/disorders/disorder_index.htm and PubMed Health - http://www.ncbi.nlm.nih.gov/pubmedhealth","logo":"http://neurolex.org/w/images/e/ee/NIF_Integrated_Disease.PNG","count":2104,"updated":"2013-01-20 22:28:08 -0800","completed":"2013-09-26 00:23:24 -0700","fields":{"IDV_Research":{"name":"Research","ontology":null,"weight":0.5},"IDV_Prognosis":{"name":"Prognosis","ontology":null,"weight":0.7},"IDV_Description":{"name":"Description","ontology":null,"weight":0.9},"IDV_Symptoms":{"name":"Symptoms","ontology":null,"weight":0.9},"IDV_Treatment":{"name":"Treatment","ontology":null,"weight":0.9},"IDV_Name":{"name":"Name","ontology":null,"weight":1.0},"IDV_Synonym":{"name":"Synonym","ontology":null,"weight":0.9}}},{"name":"Integrated Videos (via NIF)","acronym":"NIV","mainField":"NIV_Title","homepage":"http://videocast.nih.gov/","lookupURL":"","description":"The NIF Integrated Video View is a virtual database currently indexing video and other multimedia content from NIH VideoCasting and Podcasting, JoVE: Journal of Visualized Experiments, The Guardian: Science Videos and Biointeractive. JoVE is a peer reviewed, PubMed indexed journal devoted to the publication of biological research in a video format and NIH video is a vast archive of scientific talks given at various NIH meetings and functions.","logo":"http://neurolex.org/w/images/e/e0/NIF_Integrated_Video.PNG","count":8980,"updated":"2012-12-02 22:50:25 -0800","completed":"2013-09-26 01:21:53 -0700","fields":{"NIV_Author":{"name":"Author","ontology":null,"weight":0.0},"NIV_Category":{"name":"Category","ontology":null,"weight":0.0},"NIV_Runtime":{"name":"Runtime","ontology":null,"weight":0.0},"NIV_Description":{"name":"Description","ontology":null,"weight":0.9},"NIV_Database":{"name":"Database","ontology":null,"weight":0.0},"NIV_Title":{"name":"Title","ontology":null,"weight":1.0},"NIV_Date_published":{"name":"Date_published","ontology":null,"weight":0.0}}},{"name":"InterNano Process Database","acronym":"INPD","mainField":"INPD_Name","homepage":"http://www.internano.org","lookupURL":"http://www.internano.org/component/option,com_process/task,view/id,","description":"The InterNano Process Database is a knowledge base of techniques for processing nanoscale materials, devices, and structures that includes step-by-step descriptions, images, notes on methodology and environmental variables, and associated references and patent information. The purpose of the Process Database is to facilitate the sharing of appropriate process knowledge across laboratories. The processes included here have been previously published or patented.","logo":"http://www.internano.org/images/InterNano.png","count":10,"updated":"2013-07-29 02:40:46 -0700","completed":"2014-03-18 01:53:38 -0700","fields":{"INPD_Terms":{"name":"Terms","ontology":null,"weight":0.9},"INPD_Description":{"name":"Description","ontology":null,"weight":0.7},"INPD_Name":{"name":"Name","ontology":null,"weight":1.0}}},{"name":"MICAD","acronym":"MICAD","mainField":"MICAD_Name","homepage":"http://www.micad.nih.gov/","lookupURL":"http://www.ncbi.nlm.nih.gov/bookshelf/br.fcgi?book=micad&part=","description":"Molecular Imaging and Contrast Agent Database","logo":"http://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/bookshelf/thumbs/th-micad-lrg.png","count":823,"updated":"2012-03-06 05:56:20 -0800","completed":"2013-09-25 23:24:22 -0700","fields":{"MICAD_Agent_Category":{"name":"Agent_Category","ontology":null,"weight":1.0},"MICAD_Animal_Studies":{"name":"Animal_Studies","ontology":null,"weight":1.0},"MICAD_Signal_Source":{"name":"Signal_Source","ontology":null,"weight":1.0},"MICAD_Target":{"name":"Target","ontology":null,"weight":1.0},"MICAD_Name":{"name":"Name","ontology":null,"weight":1.0},"MICAD_Target_Category":{"name":"Target_Category","ontology":null,"weight":1.0},"MICAD_References":{"name":"References","ontology":null,"weight":1.0},"MICAD_Synonym":{"name":"Synonym","ontology":null,"weight":1.0},"MICAD_Human_Studies":{"name":"Human_Studies","ontology":null,"weight":1.0},"MICAD_Abbreviated":{"name":"Abbreviated","ontology":null,"weight":1.0},"MICAD_Background":{"name":"Background","ontology":null,"weight":1.0},"MICAD_Synthesis":{"name":"Synthesis","ontology":null,"weight":1.0},"MICAD_Detection_Method":{"name":"Detection_Method","ontology":null,"weight":1.0},"MICAD_In_Vitro":{"name":"In_Vitro","ontology":null,"weight":1.0}}},{"name":"ModelDB (via NIF)","acronym":"MDB","mainField":"MDB_title","homepage":"http://senselab.med.yale.edu/ModelDB/","lookupURL":"http://senselab.med.yale.edu/ModelDB/ShowModel.asp?model=","description":"ModelDB provides an accessible location for storing and efficiently retrieving computational neuroscience models. ModelDB is tightly coupled with NeuronDB. Models can be coded in any language for any environment. Model code can be viewed before downloading and browsers can be set to auto-launch the models. ModelDB is a curated database of published models in the broad domain of computational neuroscience. It addresses the need for access to such models in order to evaluate their validity and extend their use. It can handle computational models expressed in any textual form, including procedural or declarative languages (e.g. C , XML dialects) and source code written for any simulation environment. The model source code doesn't even have to reside inside ModelDB; it just has to be available from some publicly accessible online repository or WWW site. ModelDB is curated in order to maximize the scientific utility of its contents. The ideal model entry would contain \"original\" (author-written) source code, especially if it works and reproduces at least one figure from a published article. Original source code has tremendous value because it is what the authors used to generate the simulation results from which they derived their published insights and conclusions. High quality \"third party\" re-implementations of published models are also relevant, especially those involving models that are of wide interest.","logo":"http://neurolex.org/w/images/7/74/Modeldb.png","count":707,"updated":"2012-05-04 04:46:54 -0700","completed":"2013-09-26 05:14:19 -0700","fields":{"MDB_transmitters":{"name":"transmitters","ontology":null,"weight":0.8},"MDB_title":{"name":"title","ontology":null,"weight":1.0},"MDB_receptors":{"name":"receptors","ontology":null,"weight":0.8},"MDB_cellTypes":{"name":"cellTypes","ontology":null,"weight":1.0},"MDB_modelTypes":{"name":"modelTypes","ontology":null,"weight":1.0},"MDB_channels":{"name":"channels","ontology":null,"weight":0.8}}},{"name":"NIH RePORTER (via NIF)","acronym":"RPORT","mainField":"RPORT_Project_Title","homepage":"http://report.nih.gov/","lookupURL":"http://projectreporter.nih.gov/project_info_description.cfm?projectnumber=","description":"The RePORTER resource has replaced the CRISP database. RePORT is a searchable database of federally funded biomedical research projects conducted at universities, hospitals, and other research institutions. The Research Portfolio Online Reporting Tools (RePORT) Website provides a central point of access to reports, data, and analyses of NIH research.","logo":"http://neurolex.org/w/images/9/94/Report.png","count":102443,"updated":"2013-02-07 23:17:26 -0800","completed":"2013-10-25 04:10:07 -0700","fields":{"RPORT_Grant_Code":{"name":"Grant_Code","ontology":null,"weight":0.0},"RPORT_Project_Title":{"name":"Project_Title","ontology":null,"weight":1.0},"RPORT_PI_Names":{"name":"PI_Names","ontology":null,"weight":0.0},"RPORT_Funding_Institute":{"name":"Funding_Institute","ontology":null,"weight":0.0},"RPORT_Abstract":{"name":"Abstract","ontology":null,"weight":0.9},"RPORT_Funding_Year":{"name":"Funding_Year","ontology":null,"weight":0.0},"RPORT_Project_Number":{"name":"Project_Number","ontology":null,"weight":0.0},"RPORT_Address":{"name":"Address","ontology":null,"weight":0.0}}},{"name":"Online Mendelian Inheritance in Man","acronym":"OMIM","mainField":"OMIM_title","homepage":"http://www.ncbi.nlm.nih.gov/omim","lookupURL":"http://www.ncbi.nlm.nih.gov/omim/","description":"OMIM is a comprehensive, authoritative, and timely compendium of human genes and genetic phenotypes. The full-text, referenced overviews in OMIM contain information on all known mendelian disorders and over 12,000 genes. OMIM focuses on the relationship between phenotype and genotype.","logo":"http://www.ncbi.nlm.nih.gov/entrez/query/static/gifs/entrez_omim.gif","count":22186,"updated":"2012-03-09 03:49:31 -0800","completed":"2013-10-23 05:30:26 -0700","fields":{"OMIM_title":{"name":"title","ontology":null,"weight":1.0},"OMIM_locus":{"name":"locus","ontology":null,"weight":0.0},"OMIM_uid":{"name":"uid","ontology":null,"weight":0.0},"OMIM_altTitles":{"name":"altTitles","ontology":null,"weight":0.7}}},{"name":"Pathway Commons","acronym":"PC","mainField":"PC_name","homepage":"http://www.pathwaycommons.org/pc/","lookupURL":"http://www.pathwaycommons.org/pc/record2.do?id=","description":"Pathway Commons is a convenient point of access to biological pathway information collected from public pathway databases, which you can browse or search.","logo":"http://www.pathwaycommons.org/pc/jsp/images/pathwaycommons/pc_logo.gif","count":2323,"updated":"2012-03-13 00:13:39 -0700","completed":"2013-10-02 09:37:19 -0700","fields":{"PC_comment":{"name":"comment","ontology":null,"weight":0.9},"PC_organism":{"name":"organism","ontology":1132,"weight":1.0},"PC_name":{"name":"name","ontology":null,"weight":1.0}}},{"name":"PDSP Ki database (via NIF)","acronym":"PDSP","mainField":"PDSP_Receptor","homepage":"http://pdsp.med.unc.edu/pdsp.php","lookupURL":"http://pdsp.med.unc.edu/pdsp.php?knowID=retreive+this+value+only&kiKey=","description":"The PDSP Ki database is a unique resource in the public domain which provides information on the abilities of drugs to interact with an expanding number of molecular targets. The Ki database serves as a data warehouse for published and internally-derived Ki, or affinity, values for a large number of drugs and drug candidates at an expanding number of G-protein coupled receptors, ion channels, transporters and enzymes. The query interface is designed to let you search by any field, or combination of them to refine your search criteria. The flexible user interface also provides for customized data mining.","logo":"http://neurolex.org/w/images/a/a3/PDSPKI.gif","count":47304,"updated":"2012-06-26 04:15:11 -0700","completed":"2013-09-26 06:28:31 -0700","fields":{"PDSP_Organism":{"name":"Organism","ontology":null,"weight":0.9},"PDSP_Ligand":{"name":"Ligand","ontology":null,"weight":0.9},"PDSP_unit_nM":{"name":"unit_nM","ontology":null,"weight":0.5},"PDSP_Structure":{"name":"Structure","ontology":null,"weight":0.9},"PDSP_Receptor":{"name":"Receptor","ontology":null,"weight":1.0},"PDSP_Hotligand":{"name":"Hotligand","ontology":null,"weight":0.5}}},{"name":"PharmGKB [Disease]","acronym":"PGDI","mainField":"PGDI_diseaseName","homepage":"http://www.pharmgkb.org/","lookupURL":"https://www.pharmgkb.org/do/serve?objId=","description":"PharmGKB curates information that establishes knowledge about the relationships among drugs, diseases and genes, including their variations and gene products.","logo":"https://www.pharmgkb.org/images/header/title.png","count":837,"updated":"2012-03-05 22:28:11 -0800","completed":"2013-09-25 23:02:14 -0700","fields":{"PGDI_diseaseRelatedGenes":{"name":"diseaseRelatedGenes","ontology":null,"weight":0.5},"PGDI_diseaseRelatedPathways":{"name":"diseaseRelatedPathways","ontology":null,"weight":0.5},"PGDI_diseaseRelatedPhenotypeDatasets":{"name":"diseaseRelatedPhenotypeDatasets","ontology":null,"weight":0.4},"PGDI_diseaseName":{"name":"diseaseName","ontology":null,"weight":1.0},"PGDI_diseaseRelatedDrugs":{"name":"diseaseRelatedDrugs","ontology":null,"weight":0.5},"PGDI_diseaseAlternateNames":{"name":"diseaseAlternateNames","ontology":null,"weight":0.8}}},{"name":"PharmGKB [Drug]","acronym":"PGDR","mainField":"PGDR_drugName","homepage":"http://www.pharmgkb.org/","lookupURL":"https://www.pharmgkb.org/do/serve?objId=","description":"PharmGKB curates information that establishes knowledge about the relationships among drugs, diseases and genes, including their variations and gene products.","logo":"https://www.pharmgkb.org/images/header/title.png","count":1643,"updated":"2012-03-05 22:28:10 -0800","completed":"2013-09-25 22:55:30 -0700","fields":{"PGDR_drugRelatedPhenotypeDatasets":{"name":"drugRelatedPhenotypeDatasets","ontology":null,"weight":0.4},"PGDR_drugTradeNames":{"name":"drugTradeNames","ontology":null,"weight":0.9},"PGDR_drugRelatedDiseases":{"name":"drugRelatedDiseases","ontology":null,"weight":0.5},"PGDR_drugName":{"name":"drugName","ontology":null,"weight":1.0},"PGDR_drugRelatedGenes":{"name":"drugRelatedGenes","ontology":null,"weight":0.5},"PGDR_drugRelatedPathways":{"name":"drugRelatedPathways","ontology":null,"weight":0.5},"PGDR_drugGenericNames":{"name":"drugGenericNames","ontology":null,"weight":0.9}}},{"name":"PharmGKB [Gene]","acronym":"PGGE","mainField":"PGGE_geneName","homepage":"http://www.pharmgkb.org/","lookupURL":"https://www.pharmgkb.org/do/serve?objId=","description":"PharmGKB curates information that establishes knowledge about the relationships among drugs, diseases and genes, including their variations and gene products.","logo":"https://www.pharmgkb.org/images/header/title.png","count":1242,"updated":"2012-01-24 01:50:10 -0800","completed":"2013-09-25 22:59:06 -0700","fields":{"PGGE_geneName":{"name":"geneName","ontology":null,"weight":1.0},"PGGE_geneSymbol":{"name":"geneSymbol","ontology":null,"weight":0.9},"PGGE_geneAlternateSymbols":{"name":"geneAlternateSymbols","ontology":null,"weight":0.9},"PGGE_geneRelatedDiseases":{"name":"geneRelatedDiseases","ontology":null,"weight":0.5},"PGGE_geneRelatedPathways":{"name":"geneRelatedPathways","ontology":null,"weight":0.5},"PGGE_geneRelatedDrugs":{"name":"geneRelatedDrugs","ontology":null,"weight":0.5},"PGGE_geneRelatedPhenotypeDatasets":{"name":"geneRelatedPhenotypeDatasets","ontology":null,"weight":0.4}}},{"name":"PubChem","acronym":"PCM","mainField":"PCM_MeSHHeadingList","homepage":"http://pubchem.ncbi.nlm.nih.gov/","lookupURL":"http://pubchem.ncbi.nlm.nih.gov/summary/summary.cgi?cid=","description":"PubChem provides information on the biological activities of small molecules. It is a component of NIH's Molecular Libraries Roadmap Initiative.","logo":"http://pubchem.ncbi.nlm.nih.gov/images/pubchemlogob.gif","count":121349,"updated":"2012-03-09 03:57:46 -0800","completed":"2013-10-23 05:18:37 -0700","fields":{"PCM_PharmActionList":{"name":"PharmActionList","ontology":null,"weight":0.8},"PCM_SynonymList":{"name":"SynonymList","ontology":null,"weight":0.9},"PCM_MeSHHeadingList":{"name":"MeSHHeadingList","ontology":1351,"weight":1.0},"PCM_MeSHTermList":{"name":"MeSHTermList","ontology":1351,"weight":1.0}}},{"name":"PubMed","acronym":"PM","mainField":"PM_title","homepage":"http://www.pubmed.gov","lookupURL":"http://www.ncbi.nlm.nih.gov/pubmed/","description":"PubMed is a service of the U.S. National Library of Medicine  that includes over 17 million citations from MEDLINE and other life science journals for biomedical articles back to the 1950s. PubMed includes links to full text articles and other related resources.","logo":"http://www.ncbi.nlm.nih.gov/entrez/query/static/gifs/iconsml.gif","count":1158025,"updated":"2011-10-24 00:51:44 -0700","completed":"2014-02-06 23:10:54 -0800","fields":{"PM_title":{"name":"title","ontology":null,"weight":1.0},"PM_keywords":{"name":"keywords","ontology":null,"weight":0.7},"PM_meshheadings":{"name":"meshheadings","ontology":1351,"weight":0.7},"PM_abstract":{"name":"abstract","ontology":null,"weight":0.8}}},{"name":"PubMedHealth Drugs (via NIF)","acronym":"PMH","mainField":"PMH_Name","homepage":"http://www.ncbi.nlm.nih.gov/pubmedhealth","lookupURL":"http://www.ncbi.nlm.nih.gov/pubmedhealth/","description":"PubMed Health is a consumer health Web site produced by the National Center for Biotechnology Information (NCBI), a division of the National Library of Medicine (NLM) at the National Institutes of Health (NIH). PubMed Health provides up-to-date information on diseases, conditions, injuries, drugs, supplements, treatment options, and healthy living, with a special focus on comparative effectiveness research from institutions around the world.","logo":"http://neurolex.org/w/images/8/8c/PubMed_Health.PNG","count":1158,"updated":"2012-10-22 04:35:52 -0700","completed":"2013-09-25 23:59:02 -0700","fields":{"PMH_If_I_forgot_a_dose":{"name":"If_I_forgot_a_dose","ontology":null,"weight":0.8},"PMH_How_to_use":{"name":"How_to_use","ontology":null,"weight":0.9},"PMH_Other_information":{"name":"Other_information","ontology":null,"weight":0.8},"PMH_Storage_conditions":{"name":"Storage_conditions","ontology":null,"weight":0.7},"PMH_Side_effects":{"name":"Side_effects","ontology":null,"weight":0.9},"PMH_In_case_of_emergency":{"name":"In_case_of_emergency","ontology":null,"weight":0.9},"PMH_Special_dietary_instructions":{"name":"Special_dietary_instructions","ontology":null,"weight":0.7},"PMH_Why_is_this_medication_prescribed":{"name":"Why_is_this_medication_prescribed","ontology":null,"weight":0.8},"PMH_Name":{"name":"Name","ontology":null,"weight":1.0},"PMH_Other_uses":{"name":"Other_uses","ontology":null,"weight":0.7}}},{"name":"PubMedHealth Tests (via NIF)","acronym":"PMHT","mainField":"PMHT_Test_Name","homepage":"http://www.ncbi.nlm.nih.gov/pubmedhealth","lookupURL":"http://www.ncbi.nlm.nih.gov/pubmedhealth/","description":"PubMed Health, a consumer health Web site, provides up-to-date information on diseases, conditions, injuries, drugs, supplements, treatment options, and healthy living, with a special focus on comparative effectiveness research from institutions around the world. PubMed Health includes consumer guides summarizing comparative effectiveness research, fact sheets on diseases and conditions, information on drugs and supplements, encyclopedic overviews of health topics, links to external Web sites. PubMed Health has a special focus on comparative effectiveness research, in particular that research which evaluates the available evidence of the benefits and harms of different treatment options for different groups of people. In Comparative Effectiveness Research, experts often synthesize the evidence from dozens, or even hundreds, of individual studies.","logo":"http://neurolex.org/w/images/8/8c/PubMed_Health.PNG","count":832,"updated":"2012-12-13 22:04:28 -0800","completed":"2013-09-26 05:23:07 -0700","fields":{"PMHT_Why_the_test_is_performed":{"name":"Why_the_test_is_performed","ontology":null,"weight":0.9},"PMHT_How_to_prepare_for_the_test":{"name":"How_to_prepare_for_the_test","ontology":null,"weight":0.9},"PMHT_Risks":{"name":"Risks","ontology":null,"weight":0.9},"PMHT_Normal_Values":{"name":"Normal_Values","ontology":null,"weight":0.9},"PMHT_Special_considerations":{"name":"Special_considerations","ontology":null,"weight":0.8},"PMHT_What_abnormal_results_mean":{"name":"What_abnormal_results_mean","ontology":null,"weight":0.9},"PMHT_Test_Name":{"name":"Test_Name","ontology":null,"weight":1.0},"PMHT_How_the_test_will_feel":{"name":"How_the_test_will_feel","ontology":null,"weight":0.9},"PMHT_Before_the_Procedure":{"name":"Before_the_Procedure","ontology":null,"weight":0.6},"PMHT_How_the_test_is_performed":{"name":"How_the_test_is_performed","ontology":null,"weight":0.9},"PMHT_Description":{"name":"Description","ontology":null,"weight":0.7},"PMHT_After_the_Procedure":{"name":"After_the_Procedure","ontology":null,"weight":0.6},"PMHT_Prognosis":{"name":"Prognosis","ontology":null,"weight":0.6}}},{"name":"Reactome","acronym":"REAC","mainField":"REAC_name","homepage":"http://www.reactome.org/","lookupURL":"http://www.reactome.org/cgi-bin/eventbrowser?DB=gk_current&ID=","description":"A curated knowledgebase of biological pathways.","logo":"http://www.reactome.org/icons/R-purple.png","count":208093,"updated":"2012-03-08 00:23:55 -0800","completed":"2013-10-03 21:30:23 -0700","fields":{"REAC_goCellCompartiment":{"name":"goCellCompartiment","ontology":1070,"weight":0.2},"REAC_participants":{"name":"participants","ontology":null,"weight":0.8},"REAC_goBiologicalProcess":{"name":"goBiologicalProcess","ontology":1070,"weight":0.7},"REAC_name":{"name":"name","ontology":null,"weight":1.0}}},{"name":"ResearchCrossroads","acronym":"RXRD","mainField":"RXRD_title","homepage":"http://www.researchcrossroads.org/","lookupURL":"http://www.researchcrossroads.org/grants/","description":"Centralizing scientific and medical funding data so that researchers gain recognition for their work and funders make better investments.","logo":"http://www.researchcrossroads.org/templates/ja_zeolite/images/logo.png","count":1033651,"updated":"2012-03-06 21:03:25 -0800","completed":"2013-10-01 04:25:48 -0700","fields":{"RXRD_title":{"name":"title","ontology":null,"weight":1.0}}},{"name":"Stanford Microarray Database","acronym":"SMD","mainField":"SMD_exptname","homepage":"http://smd.stanford.edu/","lookupURL":"http://smd.stanford.edu/cgi-bin/data/viewDetails.pl?exptid=","description":"The Stanford Microarray Database (SMD) serves as a microarray research database for Stanford investigators and their collaborators.","logo":"http://smd.stanford.edu/images/logo_art.gif","count":18838,"updated":"2012-03-12 23:11:10 -0700","completed":"2013-10-02 09:33:22 -0700","fields":{"SMD_description":{"name":"description","ontology":null,"weight":0.7},"SMD_subcategory":{"name":"subcategory","ontology":null,"weight":0.8},"SMD_organism":{"name":"organism","ontology":1132,"weight":1.0},"SMD_category":{"name":"category","ontology":null,"weight":0.9},"SMD_exptname":{"name":"exptname","ontology":null,"weight":1.0}}},{"name":"ToxinDB (via NIF)","acronym":"TOXIN","mainField":"TOXIN_toxinName","homepage":"http://www.t3db.org/","lookupURL":"http://t3db.org/toxins/","description":"Toxin and Toxin Target Database (T3DB) is a unique bioinformatics resource that combines detailed toxin data with comprehensive toxin target information. The database currently houses over 2900 toxins described by over 34200 synonyms, including pollutants, pesticides, drugs, and food toxins, which are linked to over 1300 corresponding toxin target records. Altogether there are over 33800 toxins, toxin target associations. Each toxin record (ToxCard) contains over 50 data fields and holds information such as chemical properties and descriptors, toxicity values, molecular and cellular interactions, and medical information. This information has been extracted from over 5600 sources, which include other databases, government documents, books, and scientific literature. The focus of the T3DB is on providing mechanisms of toxicity and target proteins for each toxin. This dual nature of the T3DB, in which toxin and toxin target records are interactively linked in both directions, makes it unique from existing databases. It is also fully searchable and supports extensive text, sequence, chemical structure, and relational query searches.","logo":"http://neurolex.org/w/images/9/91/T3DB.png","count":2946,"updated":"2012-06-22 00:27:52 -0700","completed":"2013-09-26 01:07:00 -0700","fields":{"TOXIN_treatment":{"name":"treatment","ontology":null,"weight":0.7},"TOXIN_synonyms":{"name":"synonyms","ontology":null,"weight":0.8},"TOXIN_mechanism_of_Action":{"name":"mechanism_of_Action","ontology":null,"weight":0.7},"TOXIN_toxinName":{"name":"toxinName","ontology":null,"weight":1.0},"TOXIN_exposure":{"name":"exposure","ontology":null,"weight":0.8},"TOXIN_description":{"name":"description","ontology":null,"weight":0.9},"TOXIN_metabolism":{"name":"metabolism","ontology":null,"weight":0.7},"TOXIN_health_Effects":{"name":"health_Effects","ontology":null,"weight":0.9}}},{"name":"UniProt KB","acronym":"UPKB","mainField":"UPKB_geneSymbol","homepage":"http://www.uniprot.org/","lookupURL":"http://www.uniprot.org/uniprot/","description":"The mission of UniProt is to provide the scientific community with a comprehensive, high-quality and freely accessible resource of protein sequence and functional information.","logo":"http://www.uniprot.org/images/logo.gif","count":533515,"updated":"2012-02-27 02:41:23 -0800","completed":"2013-11-03 16:12:14 -0800","fields":{"UPKB_naturalVariant":{"name":"naturalVariant","ontology":null,"weight":0.7},"UPKB_goAnnotationList":{"name":"goAnnotationList","ontology":1070,"weight":1.0},"UPKB_geneSymbol":{"name":"geneSymbol","ontology":null,"weight":1.0},"UPKB_cellularComponent":{"name":"cellularComponent","ontology":null,"weight":0.7},"UPKB_proteinName":{"name":"proteinName","ontology":null,"weight":0.7},"UPKB_molecularFunction":{"name":"molecularFunction","ontology":null,"weight":0.7},"UPKB_mutagenesis":{"name":"mutagenesis","ontology":null,"weight":0.7},"UPKB_biologicalProcess":{"name":"biologicalProcess","ontology":null,"weight":0.7},"UPKB_organism":{"name":"organism","ontology":null,"weight":0.8}}},{"name":"WikiPathways","acronym":"WP","mainField":"WP_name","homepage":"http://www.wikipathways.org/index.php/WikiPathways","lookupURL":"http://www.wikipathways.org/index.php/Pathway:","description":"WikiPathways was established to facilitate the contribution and maintenance of pathway information by the biology community. WikiPathways is an open, collaborative platform dedicated to the curation of biological pathways.","logo":"http://www.wikipathways.org/skins/common/images/earth-or-pathway_text3_beta.png","count":1721,"updated":"2012-03-09 04:26:43 -0800","completed":"2013-10-01 04:27:16 -0700","fields":{"WP_name":{"name":"name","ontology":null,"weight":1.0},"WP_organism":{"name":"organism","ontology":1132,"weight":0.9}}}])
  COUNTS = %q({"AGDB":0,"AERS":0,"ABR":0,"GM":0,"AUTDB":0,"AE":296,"BGRD":0,"BIOM":1,"BSM":0,"CARY":0,"CANANO":103,"CCDB":0,"CIL":0,"CT":119,"CDD":0,"CCR":0,"CTDCD":0,"CTDCG":0,"CTDDP":0,"GAP":0,"DRGDB":0,"DBK":0,"EG":0,"GEMMA":1,"GENET":0,"IDV":0,"NIV":2,"INPD":0,"MICAD":2,"MDB":0,"PC":0,"OMIM":0,"PGDI":0,"PDSP":0,"PGDR":0,"PGGE":0,"PCM":0,"PMH":0,"PMHT":0,"REAC":0,"UPKB":0,"SMD":143,"TOXIN":0,"WP":0,"GEO":96,"RPORT":86,"RXRD":49,"PM":1259})
  SEARCH = <<-EOS
    {"page":1,"pageCount":6,"prevPage":null,"nextPage":2,"links":{"nextPage":"http://localhost:9393/resource_index/resources/AE/search?classes[BRO]=http%3A%2F%2Fbioontology.org%2Fontologies%2FBiomedicalResourceOntology.owl%23Microarray_Scanner&page=2","prevPage":null},"collection":[{"AE_description":"Malaria is caused by Plasmodium parasites, which are transmitted via the bites of infected Anopheline mosquitoes. Midgut invasion is a major bottleneck for Plasmodium development inside the mosquito vectors as a rapidly responding immune system recognizes ookinetes and recruits killing factors from the midgut and surrounding tissues, dramatically reducing the population of invading ookinetes before they can successfully traverse the midgut epithelium. Understanding molecular details of the parasite-vector interactions requires precise measurement of nascent protein synthesis in the mosquito during Plasmodium infection. Current expression profiling primarily monitors alterations in steady-state levels of mRNA, but does not address the equally critical issue of whether the proteins encoded by the mRNAs are actually synthesized. In this study, we used sucrose density gradient centrifugation to isolate actively translating Anopheles gambiae mRNAs based upon their association with polyribosomes (polysomes). The proportion of individual gene transcripts associated with polysomes, which is determined by RNA deep sequencing, reflects mRNA translational status. This approach led to identification of 1017 mosquito transcripts that were primarily regulated at the translational level after ingestion of Plasmodium falciparum-infected blood. Caspar, a negative regulator of the NF-kappaB transcription factor Rel2, appears to be substantially activated at the translational levels during Plasmodium infection. In addition, transcripts of Dcr1, Dcr2 and Drosha, which are involved in small RNA biosynthesis, exhibited enhanced associations with polysomes after P. falciparum challenge. This observation suggests that mosquito microRNAs may play an important role in reactions against Plasmodium invasion. We analyzed both total cellular mRNAs and mRNAs that are associated with polysomes to simultaneously monitor transcriptomes and nascent protein synthesis in the mosquito. This approach provides more accurate information regarding the rate of protein synthesis, and identifies some mosquito factors that might have gone unrecognized because expression of these proteins is regulated mainly at the translational level rather than at the transcriptional level after mosquitoes ingest a Plasmodium-infected blood meal. Midguts from female Anopheles gambiae mosquitoes were dissected at around 24 h after ingestion of P. falciparum-infected blood. Mosquitoes fed on uninfected blood were used as control. A portion of the midguts were used for isolation of total cellular RNA. Extracts of the remaining midguts were fractionated over sucrose density gradients, and fifteen fractions were collected from the top of each gradient. Non-polysomal fractions, as well as polysomal fractions were combined, respectively, to obtain two RNA pools per gradient for three independent experiments. The first, last and the sample between non-polysomal and polysomal, were all discarded to ensure pure pools from each set. A fourth experiment was conducted where the polysomal and non-polysomal samples were not pooled, to be used later for qRT-PCR. The mRNA levels of individual mosquito genes in polysome (PS) fractions, nonpolysome (NP) fractions and unfractionated steady-state (total) RNA were determined using high throughput RNA deep sequencing. Each RNA pool generated 2.1-9.8 million raw read .  Signal intensities of transcripts from each PS pool were compared to those of transcripts from a matching NP pool. To quantify the translational status of individual mRNA species, we define the relative PS loading [PL=P/(P+NP)] as the extent of mRNA association with polysomes. PL values were compared between the mosquitoes fed on P. falciparum-infected or -uninfected blood.","AE_experiment_type":"EFO\u001fhttp://www.ebi.ac.uk/efo/EFO_0003738","AE_species":"NCBITAXON\u001fhttp://purl.obolibrary.org/obo/NCBITaxon_7165","AE_name":"Translational regulation of Anopheles gambiae mRNAs in the midgut during Plasmodium falciparum infection"},{"AE_description":"RNA helicase A (RHA) binds its target transcripts at the post-transcriptional control element (PCE) located in the 5â€™ untranslated region (UTR). This interaction represents an â€œRNA switchâ€ that regulates protein synthesis. Down regulation of RHA by siRNAs was used to identify transcripts with RHA-dependent translation. Reduced accumulation of RNA in polysomes was monitored with microarrays. Cytoplasmic lysates of cells treated with RHA targeted or non-silencing control siRNAs were separated by sucrose density gradient centrifugation. Ribosomal RNA profiles were generated, containing heavy polysomes were collected, and RNA was extracted.","AE_experiment_type":"EFO\u001fhttp://www.ebi.ac.uk/efo/EFO_0002768","AE_species":"NCBITAXON\u001fhttp://purl.obolibrary.org/obo/NCBITaxon_9534","AE_name":"Changes in polysome loading as a consequence of RHA downregulation [Agilent]"},{"AE_description":"Diethylstilbestrol (DES) is a synthetic estrogen that has been banned for use in humans, but still is employed in livestock and aquaculture operations in some parts of the world.  Detectable concentrations of DES in effluent and surface waters have been reported to range from slightly below 1 to greater than 10 ng/L.  Little is known, however, concerning the toxicological potency of DES in fish.  In this study, sexually-mature fathead minnows (Pimephales promelas) of both sexes were exposed to 1, 10 or 100 ng DES/L water in a flow-through system.  Tissue concentrations of DES and changes in a number of estrogen-responsive endpoints, including alterations to the hepatic transcriptome in females, were measured in the fish at the end of a 4-d exposure, and after a 4-d depuration/recovery period in clean water. The objectives of the study were to measure accumulation of DES in fish tissues, characterize effects of the chemical on physiological endpoints related to reproductive performance, and to evaluate DES's potency relative to concentrations reported in aquatic environments. The current series includes n=12 microarrays associated with exposure-phase samples (collected from fish exposed continuously for 4 d). Fish were exposed to 0, 1, 10, or 100 ng DES/L delivered in a continuous flow (45 ml/min) of sand filtered, UV treated, Lake Superior Water, without the use of carrier solvents. Flow of DES into the test system was initiated without fish present and continued for 1 d.  Exposures then were started by placing three male and three female sexually-mature fathead minnows into each tank. There were eight replicate tanks for the control and 100 ng/L treatments, and six replicate tanks for the 1 and 10 ng/L groups.  The fish were held at 25+1ÂºC under a 16:8 L:D photoperiod and were fed frozen adult brine shrimp twice daily (San Francisco Bay Brand, Newark, CA, USA).  Fish used for the experiment were from an on-site culture, and all procedures involving animals conformed to guidelines approved by the local Animal Care and Use Committee. After the 4-d exposure, fish from three tanks from each treatment (n=9 per sex) were sampled for determination of various biological endpoints.  At this time fish from two additional tanks from the control and 100 ng/L treatments also were sampled for determination of DES tissue residues (n=6 per sex).  Delivery of DES to the test system was subsequently stopped, and fish from the remaining three tanks per treatment group were sampled 4 d later (n= 9 per sex). The fathead minnows were euthanized with buffered MS-222 (Argent, Redmond, VA, USA) and weighed.  Fish were scored for occurrence and relative expression of nuptial tubercles, which in males can be reduced by ER agonists.  Blood was collected from the caudal vein/artery with a microhematocrit tube, and plasma was separated by centrifugation and stored at -80ÂºC.  Gonads were removed from males and a subsample (8.5Â±3.0 mg; meanÂ±SD) was immediately immersed in tissue culture media for determination of ex vivo steroid production. Livers were removed from both sexes and flash-frozen for gene expression analyses using microarray (females) or real-time QPCR (males). Hepatic transcripts from three females per treatment group, in each phase of the experiment (except n=2 from recovery phase 10 ng/L) were analyzed using a custom 15,000 feature microarray (GEO Platform Accession GPL9248). Data sets for the exposure phase (n=12 microarrays) and recovery phase (n=11) were normalized independently using Fastlo (Ballman et al., 2004) implemented in R (http://www.r-project.org/), but analyzed using parallel approaches.","AE_experiment_type":"EFO\u001fhttp://www.ebi.ac.uk/efo/EFO_0002768","AE_species":"NCBITAXON\u001fhttp://purl.obolibrary.org/obo/NCBITaxon_90988","AE_name":"A Short-Term Study Investigating the Estrogenic Potency of Diethylstilbesterol in the Fathead Minnow (Pimephales promelas): Exposure Phase"},{"AE_description":"In order to further our understanding of the metabolic network of the malaria parasite, Plasmodium falciparum, we carried out a concurrent transcriptomic and metabolomic study of the parasite's intraerythrocytic developmental cycle.  These microarray data were generated to compare the expression levels of metabolic enzymes to the concentrations of their associated metabolites over the 48-hour life cycle. Sorbitol-synchronized Plasmodium falciparum (3D7 strain) was grown in tissue culture flasks in incubators according to standard protocols.  Immediately after reinvasion, and at 8-hour intervals thereafter, parasites were harvested by centrifugation.  For each timepoint, 0.5 mL of packed RBC (10% parasitemia) was pelleted by centrifugation, washed once in PBS and flash-frozen in liquid nitrogen.  Total RNA isolation and amino-allyl cDNA labeling were as previously described (Bozdech et al., 2003).  A pool of 3D7 total RNA from all intraerythrocytic developmental stages was generated and used as the reference sample.  For DNA microarray hybridization, pool cDNA was coupled to Cy3 dye, while cDNA from an individual timepoint was coupled to Cy5 dye.  DNA microarrays were scanned using an Axon 4200A scanner and images analyzed using Axon GenePix software (Axon Instruments, Union City, CA, USA).  Microarray data were stored and analyzed using our in-house database PUMAdb (Princeton University MicroArray database).  All data for individual arrays were normalized by a global normalization using unflagged features with >= 65% of pixels one or more standard deviations over local background.  All unflagged spots were selected and extracted for further analysis.  Data were filtered to remove oligos more than 1 datapoint missing across the timeseries, log2 transformed, mean centered, ordered by the timing of their peak expression level, and visualized with Java Treeview (Saldanha, 2004) (Table S3).","AE_experiment_type":"EFO\u001fhttp://www.ebi.ac.uk/efo/EFO_0002768","AE_species":"","AE_name":"Transcriptomic and metabolomic profiles of the Plasmodium falciparum developmental cycle"},{"AE_description":"Goal was to identify yeast genes whose expression changed as a function of the shift from growth in bulk culture to growth in an air-liquid interfacial biofilm. Experiment Overall Design: Cells were grown 24 h at 30 C in YEPD to a density of about 500,000,000 cells/ml. At harvest, sugar was found to be depleted as measured by an enzymatic dip stick (Diastic, Bayer). Cells were pelleted and washed twice in sterile distilled water by centrifugation and then diluted 10-fold into 100 ml of Flor medium (YNB + 4% ethanol + leucine + histidine + uracil) in triplicate 250 ml beakers. Cultures were then grown statically in Flor medium at 27 C. Within a few hours following inoculation, the bulk liquid appeared to be clear, no biofilm was evident by visual observation, and a layer of settled cells was evident at the bottom of the beakers. After 48 h, a visible air-liquid interfacial biofilm covered the entire surface while the thickness of the layer of settled cells appeared unchanged. After 48 h, biofilm cells (floaters) were collected by aseptic aspiration. Once the biofilm cells were removed, cells at the bottom of the beaker (sinkers) were collected similarly. Cells from both populations were washed once in sterile distilled water by centrifugation prior to RNA isolation. Significant cell clumping was evident in both populations of cells by microscopic observation. While cell viability was estimated by plating on YEPD, the resultant cfu/ml values could not be directly correlated with cell counts in a hemacytometer because clumps of cells containing at least one viable cell presumably produced only a single colony. Further, counting individual cells accurately in the numerous clumps containing large numbers of cells was not possible. Nonetheless, when cells were counted (clumps were counted as single cells) and compared to cfu/ml for the same suspension, the cfu/ml values were about 2-fold higher than the corresponding values for cell counts using the hemacytometer for both biofilm and bottom layer cells.","AE_experiment_type":"EFO\u001fhttp://www.ebi.ac.uk/efo/EFO_0002768","AE_species":"NCBITAXON\u001fhttp://purl.obolibrary.org/obo/NCBITaxon_4932","AE_name":"Transcription profiling of S. cerevisiaeAir-liquid interfacial biofilm vs planktonic S. cerevisiae cells"},{"AE_description":"Samples and RNA preparation: The series is composed of 4 hybridizations for analysis of differentially expressed genes in the liver of ciprofibrate dosed vs. control rats. Rats (200-250 g) were given (by gastric intubation) tap water (control group, n=5) or ciprofibrate (50 mg/kg body weight, once daily for 60 days, n=4). Liver total RNA extraction was performed first by ultracentrifugation on a cesium chloride cushion and then using TRIZOL Reagent (GIBCO BRL Life Technologies, New York, NY). Labeling and hybridization: One microgram of total RNA from the control pool (n=5), and from each ciprofibrate dosed rat was reverse transcribed and labeled with Cy3- and Cy5-attached dendrimer, respectively, using 3DNA Array Detection Submicro kit as described in the manufacturer's protocols  (Genisphere, Montvale, NJ). The labeling reactions were also spiked with exogenous control RNAs (Stratagene) at expected Cy5/Cy3 ratios of 1.0, 2.0, 3.0, 4.0, 5.0, 6.0,  0.5, 0.33, 0.25 and 0.125 for control spike 1, 2, 3, 4, 5, 6, 8, 9, 10 and 7, respectively. The Cy3 and Cy5 labeled samples (combined volume 5 microliter) were mixed with 30 microliter of the hybridization solution containing 0.25 M NaPO4, 1 mM EDTA, 4.5% SDS, 1 x SSC, 2 x Denhardt's solution, 0.25 microliter antifade reagent (Genisphere) and 0.2 microgram mouse COT-1 DNA (GIBCO BRL Life Technologies), added onto the array and covered with 22 x 50mm cover slip. The array was assembled into humidified hybridization chamber (Corning) and hybridized at 65oC for 18 h by submerging in a water bath. Post-hybridization washes were done once in 2 x SSC, 0.2% SDS at 55oC for 15 min, then in 2 x SSC at room temperature for 15 min, and finally with 0.2 x SSC at room temperature for 15 min. Scanning, image analysis and data processing: Arrays were scanned at 10 micrometer resolution with a confocal laser scanner constructed in-house in collaboration with NEMKO (Trondheim, Norway) according to a prototype developed at NHGRI (http://www.nhgri.gov/DIR/LCG/15K(HTML/). Microarray image analysis was done with MicroArray Suite software version 2.1 with globally normalization (Scanalytics, Inc., Fairfax VA). First, spots undetected (target intensity=1) in at least one array were excluded. Then, spots with signal intensity (averages of the 4 arrays) less than 300 in both channels were removed. The fluorescence intensity ratios (Cy5/Cy3) for genes measured from probes spotted in duplicate were averaged for each slide. Then, mean ratios of the 4 hybridizations were calculated for each gene, and differentially regulated genes were determined using 1.4 fold change cutoff values (1.4 for up-regulated or 0.7 for down-regulated genes).","AE_experiment_type":"","AE_species":"NCBITAXON\u001fhttp://purl.obolibrary.org/obo/NCBITaxon_10116","AE_name":"Ciprofibrate stimulated gene expression in rats"},{"AE_description":"B-cell chronic lymphocytic leukemia (B-CLL) is characterized by a highly variable clinical course that reflects its heterogeneous genomic pattern. To better define molecular subtypes of the disease, we performed SNP and gene expression profiling microarray analyses in a panel of early stage (Binet A) patients. A clustering analysis of genomic profiles identified four significant groups mainly driven by del(13)(q14) and trisomy 12. Notably, patients with del(13)(q14) were grouped in two separate clusters based on the presence of a biallelic loss and the extension of the deletion. The shorter monoallelic deleted 13q14 region was found to be 635 kb in length, not encompassing the mir-15a/16-1 locus. Interestingly, the mir-15a and mir-16 expression was found to be significantly down-regulated only in patients with biallelic loss. Furthermore, a multiclass supervised analysis identified a different transcriptional signatures in the two genomic subgroups with del(13)(q14). Finally, an integrative approach identified 93 transcripts, mainly mapped to chromosome 12 and 13q12-q14.3, whose expression was significantly correlated with the DNA copy number. Overall, our data further support the notion that transcription deregulation in B-CLL could be mostly due to a gene dosage effect and underscore the presence of two distinct molecular types of 13q14 deleted patients with potential clinical relevance. Transcriptional analysis of 60 B-CLL patients, and genotyping analysis of 100 B-CLL patients, in early stage disease (Binet A).  This series of microarray experiments contains the gene expression profiles of purified B-cell chronic lymphocytic leukemia (B-CLL) cells obtained from 60 newly diagnosed patients (Binet stage A). Peripheral blood mononuclear cells from B-CLL patients were isolated by Ficoll-Hypaque density-gradient centrifugation and the proportion of CD5/CD19/CD23 triple positive B cells in the suspension was determined by direct immunofluorescence performed using a FACS-sort flow cytometer with antibodies to: CD19 FITC/PE, CD23 PE and CD5 Cy-Chrome. If B-CLL cells were less than 90%, T cells, NK cells and monocytes were removed by negative selection using CD3, CD56, CD16, and CD14 monoclonal antibody treatment followed by magnetic beads. 3 micrograms of total RNA was processed and, in accordance with the manufacturer's protocols, 15 micrograms of fragmented biotin-labelled cRNA were hybridized on GeneChip Human Genome U133A Arrays (Affymetrix Inc.). The arrays were scanned using the GeneChip Scanner 3000 7G. The images were acquired using Affymetrix GeneChip Operating Software (GCOS version 1.4) and the probe level data converted to expression values using the Bioconductor function for the Robust Multi-Array average (RMA) procedure (Irizarry et al, 2003), in which perfect match intensities are background adjusted and quantile-quantile normalised.  This series of microarray experiments contains the genome-wide profiles of purified B-cell chronic lymphocytic leukemia (B-CLL) cells obtained from 100 newly diagnosed patients (Binet stage A). Peripheral blood mononuclear cells from B-CLL patients were isolated by Ficoll-Hypaque density-gradient centrifugation and the proportion of CD5/CD19/CD23 triple-positive B cells in the suspension was determined by direct immunofluorescence performed using a FACS-sort flow cytometer with antibodies to: CD19 FITC/PE, CD23 PE and CD5 Cy-Chrome. If B-CLL cells were less than 90%, T cells, NK cells and monocytes were removed by negative selection using CD3, CD56, CD16, and CD14 monoclonal antibody treatment followed by magnetic beads. 250 nanograms of genomic DNA was processed and, in accordance with the manufacturer's protocols, 90 micrograms of fragmented biotin-labeled DNA were hybridized on GeneChip Human Mapping 250K NspI Arrays (Affymetrix Inc.). The arrays were scanned using the GeneChip Scanner 3000 7G. The images were acquired using Affymetrix GeneChip Operating Software (GCOS version 1.4). The raw data for individual SNPs were extracted from CEL files and converted into signal intensities using GTYPE 4.1 and Affymetrix Copy Number Analysis Tool (CNAT 4.0.1) softwares using the Hidden Markov Model algorithm with a genomic smoothing window set to 0. After the pre-processing, piecewise constant estimates of the underlying local DNA copy number (CN) variation was calculated using the DNAcopy Bioconductor package, which looks for optimal breakpoints using circular binary segmentation (CBS).","AE_experiment_type":"EFO\u001fhttp://www.ebi.ac.uk/efo/EFO_0000749","AE_species":"NCBITAXON\u001fhttp://purl.obolibrary.org/obo/NCBITaxon_9606","AE_name":"Genomic approach in B-cell Chronic Lymphocytic Leukemia: molecularly distinct subgroups of patients with 13q14 deletion"},{"AE_description":"We have identified a population of adipocytes in fat tissue that arise from bone marrow-derived progenitor cells. We used microarrays to compare the global gene expression patterns of the bone marrow progenitor-derived adipocytes as well as conventional white and brown adipocytes to evaluate the relationship between these adipocyte subpopulations. Gonadal fat tissue (for white adipocytes) and intrascapular fat tissue (for brown adipocytes) was digested with collagenase and adipocytes were recovered by centrifugation/flotation.  Bone marrow derived adipocytes were isolated from the adipocyte fraction of gonadal fat tissue from mice receiving bone marrow tranplants from donors expressing either green fluorescent protein (GFP) or beta-galactosidase (LacZ) by flow cytometry.","AE_experiment_type":"EFO\u001fhttp://www.ebi.ac.uk/efo/EFO_0002768","AE_species":"NCBITAXON\u001fhttp://purl.obolibrary.org/obo/NCBITaxon_10090","AE_name":"Expression Data from Bone Marrow Progenitor-Derived Adipocytes, White Adipocytes and Brown Adipocytes."},{"AE_description":"Experiment Design:,â€¢Type of experiment: time course after GAL (galactose) induction,â€¢Experimental factor: time, genotype,â€¢Number of hybridizations performed in the experiment: 24 (eight conditions with three biological replicates each),Samples used, extract preparation and labeling:,â€¢The origin of the biological sample:,Species: S. cerevisiae,Cell type:,wild-type (YSS37, IFH1-13myc-TRP1, MATa),UASIFH1::HIS3-UASGAL1 (YSS89, UASGAL1-IFH1-13myc-TRP1, MATa),â€¢Growth conditions: log phase (2x107/mL) at 30Â°C in YPLG medium, then addition of Galactose to 2%,â€¢Treatments: all cells were harvested by rapid centrifugation at 4Â°C then flash frozen in liquid nitrogen.","AE_experiment_type":"EFO\u001fhttp://www.ebi.ac.uk/efo/EFO_0002768","AE_species":"NCBITAXON\u001fhttp://purl.obolibrary.org/obo/NCBITaxon_4932","AE_name":"Transcription profiling time series IFH1 overexpression by galactose induction in S. cerevisiae"},{"AE_description":"Inflammatory mechanisms and immune cells are involved in acute coronary syndromes (ACS) and may lead to acute plaque rupture. However, the local expression of the different genes potentially involved is largely unknown. We therefore performed an Affymetrix analysis of genes expressed in white blood cells obtained from an occluding coronary thrombus or peripheral blood of patients with ST-elevation myocardial infarction. Thrombi of ACS patients were harvested from the site of coronary occlusion. Leukocytes were isolated by Ficoll centrifugation. Peripheral blood leukocytes (PBL) were treated in a similar fashion and mRNA was extracted from both cells.","AE_experiment_type":"EFO\u001fhttp://www.ebi.ac.uk/efo/EFO_0002768","AE_species":"NCBITAXON\u001fhttp://purl.obolibrary.org/obo/NCBITaxon_9606","AE_name":"Differential gene expression in thrombus-derived white blood cells of patients with acute coronary syndrome"},{"AE_description":"The objective of this study was to understand the genetic mechanisms of Vitamin-A-Deficiency (VAD)-induced arrest of spermatogonial stem-cell differentiation. Vitamin A and its derivatives (the retinoids) participate in many physiological processes including vision, cellular differentiation and reproduction. VAD affects spermatogenesis, the subject of our present study. Spermatogenesis is a highly regulated process of differentiation and complex morphologic alterations that, in the postnatal testis, leads to the formation of sperm in the seminiferous epithelium. VAD causes early cessation of spermatogenesis, characterized by degeneration of meiotic germ cells, leading to seminiferous tubules containing mostly type A spermatogonia and Sertoli cells. In this study, we investigated the molecular basis of VAD on spermatogenesis in mice. We used adult Balb/C mice fed with a Control or VAD diet for an extended period of time (8-28 weeks) and selected two time points (18 and 25 weeks) for microarray analysis.  To understand the effect of VAD on the spermatogonial stem cell transcriptome, we studied isolated pure populations of spermatogonia from control and vitamin-A-deficient mice from two representative time points (18 and 25 weeks) using Affymetrix GeneChip microarrays. We identified target genes involved in the arrest of spermatogonial differentiation and spermatogenesis. Our results establish a better understanding of the chronology and magnitude of the consequences of VAD on mouse testes and add to the current knowledge of the molecular regulatory mechanisms of germ cell development. Spermatogonia were isolated by the STATPUT procedure with minor modifications. VAD mice were sacrificed at 18 and 25 weeks of VAD treatment for spermatogonia isolation. Decapsulated testes from 2 mice were suspended in RPMI medium containing collagenase (0.6 mg/ml), Hyaluronidase (0.12 mg/ml) and DNAse I (1.25 mg/ml) and incubated at 37ÂºC for 30 minutes in a shaking water bath. The tissues were then allowed to come down the tube and the supernatant (containing interstitial cells) was removed. The pellet was incubated with 3 ml of 0.25% Trypsin /EDTA (GIBCO, Invitrogen, USA), in the presence of DNAse I (0.2 mg/ml) at 37ÂºC for 15 minutes in a shaking water bath. The dispersed cells were washed twice with RPMI medium containing 10% heat inactivated FBS to neutralize the protease activity, and filtered through a sterile 0.22 nylon to remove any undigested fragments. Cells of the dissociated seminiferous epithelium were then plated overnight in a 34ÂºC 3% CO2 incubator. The following day, germ cells, in suspension, were separated by sedimentation with use of a 2-4% BSA gradient. The cells were allowed to sediment for a standard period of 2.5 h, and fractions of 2-ml volume were collected. The cells of each fraction were examined under a phase contrast microscope, and fractions containing cells of similar size and morphology were pooled and spun down by low-speed centrifugation. Purity of spermatogonia was estimated and was routinely higher than 90%. Total RNA was extracted from the isolated germ cells using TRIzol Â® Reagent, and cleaned with RNeasy minicolumns. RNA content was determined by measurement of optical density at 260 nm. Only the RNA samples showing an OD 260/280 ratio higher than 1.8 were used for microarray hybridization. Raw expression values in Affymetrix CEL file format were generated by GeneChip Operating Software.","AE_experiment_type":"EFO\u001fhttp://www.ebi.ac.uk/efo/EFO_0002768","AE_species":"NCBITAXON\u001fhttp://purl.obolibrary.org/obo/NCBITaxon_10090","AE_name":"Effect of vitamin A deficiency (VAD) on mouse spermatogonial transcriptome profiles"},{"AE_description":"Transcriptional profiling of an El Tor biotype crp mutant The virulent V. cholerae El Tor Ogawa strain C7258 (Peru isolate 1991) and an isogenic deletion mutant (WL7258) lacking DNA sequences encoding the cAMP receptor protein were grown in LB medium to optical density at 600 nm of 1.5. The cultures were chilled in ice, cells quickly collected by centrifugation and total RNA imediately extracted. RNAwas extracted and purified using the Trizol plus RNA purification system (Invitrogen) followed RNEasy miniElute cleanup (Qiagen). RNA samples were conserved at - 80 C and used within a week. 4 replicates","AE_experiment_type":"","AE_species":"NCBITAXON\u001fhttp://purl.obolibrary.org/obo/NCBITaxon_666","AE_name":"Vibrio cholerae: Wild type El Tor biotype versus crp mutant"},{"AE_description":"Self-renewing undifferentiated nephron progenitors express Six2, a transcription factor that is required for their maintenance as undifferentiated progenitors. Differentiation of nephron progenitors is triggered by Wnt/b-catenin signaling. In order to understand how Six2 and Wnt signaling counteract each other, we performed ChIP-seq of Six2 and b-catenin in mesenchymal nephron progenitor cells. Nephron progenitors were FACS-isolated from BAC transgenic Six2GFPcre-positive embryonic kidneys at E16.5. For Six2 ChIP, freshly FACS isolated Six2+ cells were used. For b-catenin ChIP, FACS isolated Six2+ cells were aggregated by centrifugation at 850g for 5min and incubated in 10%FBS/DMEM containing 4uM BIO for 24hrs.","AE_experiment_type":"EFO\u001fhttp://www.ebi.ac.uk/efo/EFO_0002692","AE_species":"NCBITAXON\u001fhttp://purl.obolibrary.org/obo/NCBITaxon_10090","AE_name":"Genome-wide maps of Six2 and b-catenin in mesenchymal nephron progenitors"},{"AE_description":"Alterations of the short arm of chromosome 12 (12p) occur in various hematological malignancies and ETV6 and CDKN1B, which are located on 12p, have been implicated as leukemogenic genes of interest. Design and Methods: We selected 7 patients with myeloid malignancies and small 12p deletions by FISH encompassing only the region centromeric of ETV6 and further evaluated them by SNP microarrays. Results: The minimal deleted region (MDR) contained nine genes only. These genes were subsequently analyzed by microarray expression profiling in an independent cohort of 781 cases mostly with different hematological malignancies or without any hematological malignancy: CREBL2, MANSC1, and CDKN1B were expressed in >25% of cases, while the other 6 genes were expressed in a minority of cases only. As CDKN1B is a cell cycle regulator and functions as a tumor suppressor gene, this gene was selected for further expression studies in 286 AML patients. In cases with low CDKN1B expression (expression level <1,160; 1st quartile), RUNX1-RUNX1T1 (11/83; 13.3%; vs. 5/203; 2.5%; p=0.001), PML-RARA rearrangements (11/83; 13.3%; vs. 4/203; 2.0%; p<0.001), 11q23/MLL rearrangements (6/83; 7.2%; vs. 4/203; 2.0%; p=0.038), and FLT3-TKD mutations (7/63; 11.1% vs. 6/167; 3.6%; p=0.047) were more frequently observed than in patients with intermediate/high expression (2nd-4th quartiles). Low CDKN1B expressers had longer median OS and EFS (not reached vs. 14.9 months; p=0.005 and 31.0 vs. 9.7 months; p=0.013, respectively) than intermediate/high expressers. Conclusions: CDKN1B is an interesting candidate gene as a potential biomarker for acute myeloid leukemia prognostication. Affymetrix SNP arrays were performed according to the manufacturer's directions using DNA extracted from mononuclear cells after Ficoll density centrifugation from bone marrow or peripheral blood samples.","AE_experiment_type":"EFO\u001fhttp://www.ebi.ac.uk/efo/EFO_0000749","AE_species":"NCBITAXON\u001fhttp://purl.obolibrary.org/obo/NCBITaxon_9606","AE_name":"CDKN1B encoding the cyclin-dependent kinase inhibitor 1B (p27) is located in the minimally deleted region of 12p abnormalities in myeloid malignancies and its low expression is a favorable prognostic marker in acute myeloid leukemia"},{"AE_description":"In a screen for singlet oxygen resistant mutants, a mutant called sor1 was isolated with increased resistance to different oxidative and electrophilic stress conditions. This mutant was found to have a constitutive high expression of three known singlet oxygen response genes, a glutathione peroxidase GPX5 and two glutathione-S-transferase GSTS1 and GSTS2. The sor1 mutation was mapped to the gene of a 393-amino acid protein for which blast searches revealed only poor homology to other proteins but predicted a putative basic leucine zipper DNA binding domain indicating that it might function as transcription factor to activate gene expression during oxidative stress.Global expression analysis of sor1 compared to the untreated wild-type enabled to identify additional genes differentially regulated in the sor1 mutant including several other genes involved in oxidative stress response and detoxification of endogenous xenobiotics. The sor1 mutant and the corresponding wild-type strain 4A+ were grown mixotrophically in a Tris-acetate phosphate in three independent biological replicates. Then the cells of each replicate were harvested by centrifugation and total RNA was isolated using the RNeasy Mini Kit (Qiagen). DNA microarrays were performed using the â€˜One-Color Microarray-Based Gene Expression Analysisâ€™ system and a custom made 4 Ã— 44 K â€˜Chlamydomonas Whole Genome DNA Microarraysâ€™ (Agilent Technologies) containing 15143 specific probes designed based on the Chlamydomonas version 4.0 transcript models provided by the DOE Joint Genome Institute (JGI), with an average of three replicates for each probe","AE_experiment_type":"EFO\u001fhttp://www.ebi.ac.uk/efo/EFO_0002768","AE_species":"NCBITAXON\u001fhttp://purl.obolibrary.org/obo/NCBITaxon_3055","AE_name":"Expression analysis of the singlet oxygen resistant 1 (sor1) mutant"},{"AE_description":"Estrogen is a reproductive steroid hormone that has both beneficial and detrimental effects on the cardiovascular system. Selective estrogen receptor modulators (SERMs) exhibit partial estrogen agonist/antagonist activity in estrogen target tissues.  Since we still have an incomplete picture of the gene targets of estrogen in the vasculature, and our understanding of SERM gene targets in the vasculature is even less well-developed, it is important to broaden the available information on this subject.  The present study tested the hypothesis that estrogens (ethinyl estradiol, estradiol benzoate, and equilin) and SERMs (tamoxifen and raloxifene) cause differential gene and protein expression in the vasculature.  DNA microarray and real-time RT-PCR were used to investigate gene expression in the mesenteric arteries of estrogen and SERM treated ovariectomized rats. The genes shown to be differentially expressed included stearoyl-CoA desaturase (SCD), soluble epoxide hydrolase (sEH), secreted frizzled related protein-4 (SFRP-4), insulin-like growth factor-1 (IGF-1), phospholipase A2 group 1B (PLA2-G1B), and fatty acid synthase (FAS).  Western blot further confirmed the differential expression of sEH, SFRP-4, FAS, and SCD protein.  These results reveal that estrogens and SERMs cause differential gene and protein expression in the mesenteric artery.  Consequently, the use of these agents may be associated with a unique profile of functional and structural changes in the mesenteric arterial circulation.   Goal of the experiment:  To examine the effects of three formulations of estrogen (ethinyl estradiol, equilin, and estradiol benzoate) and of the selective estrogen receptor modulators, tamoxifen and raloxifene, on gene expression in the mesenteric arteries of the rat.  Keywords:  rat, artery, gene expression, mesentery, estrogen, tamoxifen, raloxifene  Experimental factors:  hormone treatment, drug treatment  Experimental design:  Prepubertal female Sprague Dawley rats (Harlan, Indianapolis, IN) were ovariectomized at 4 weeks of age, prior to the initiation of regular ovarian cycles.  The animals were allowed to recover for 2 weeks before the experimental protocol was initiated.  The rats were randomly divided into 6 groups (n=3-4/group) and treated by daily gavage for 3 weeks with the vehicle (20% 2-hydroxypropyl-?-cyclodextrin, Sigma, St Louis, MO), ethinyl estradiol (EE, 0.15 mg/kg; Sigma), estradiol benzoate (EB, 0.15 mg/kg; Sigma), equilin (EQ, 0.15 mg/kg; Sigma), tamoxifen (TAM, 3 mg/kg; Sigma), or raloxifene (RAL, 3 mg/kg; Eli Lilly & Co., Indianapolis, IN).   After 3 weeks of treatment, the rats were euthanized and the mesentery obtained.  Total RNA was extracted from each individual sample and used for the synthesis of labeled cRNA for microarray analysis. After the final gavage treatment on day 21, the rats were deeply anesthetized with isoflurane and euthanized by decapitation.  The mesenteric arterial tree was isolated and harvested and stored in RNAlater (Ambion) at -80ÂºC.  The mesenteric arteries were dissected free from veins and surrounding fat and homogenized in TRI reagent (MRC, Cincinnati, OH). Mesenteric artery RNA was extracted and purified using an RNeasy Mini Kit (Qiagen).  Any potentially contaminating DNA was removed by treatment with RNase-free DNase (Qiagen) on the RNeasy column.  Total RNA was quantified and RNA integrity was assessed using the Agilent Bioanalyzer (Santa Clara, CA).  Following analysis, the RNA was stored at -80ÂºC until it was used for gene expression studies. DNA microarray:  CodeLink Whole Rat Genome Bioarrays (Applied Microarrays, Tempe, AZ) were used.  Five ?g total RNA was used from each mesenteric arterial sample for the microarray analysis.  Complementary RNA was synthesized per company protocol (Ambion).  A set of bacterial RNAs was added to each total RNA sample and carried through the synthesis and labeling reactions as controls and for norÂ¬malization of the data.  The microarrays were scanned with the GenePix 4000B (Axon Instruments, Union City, CA) with GenePix Pro 5.0 software (Axon Instruments).  Analysis of the microarÂ¬rays was performed with the CodeLink Expression Analysis software (Applied Microarrays) and Acuity 3.1 software (Axon), and statistical analysis was performed with GeneSpring 7.1 software (Agilent, Santa Clara, CA).  A sample of n=3 was completed with the microarrays for each treatment.  Quality control steps:  The cRNA that was synthesized from each mesenteric artery treatment sample was used for hybridization to a single CodeLink (Applied Microarrays) whole rat genome microarray.  Only one sample was hybridized with each slide and only one dye (streptavidin-Alexa 647) was used so no dye swaps were used.  Bacterial control spikes were used per manufacturer's instructions.  Samples used, extract preparation and labeling: The origin of each biological sample:  The samples were isolated mesenteric arteries from individual Sprague Dawley rats that had been treated by daily gavage for 3 weeks with the vehicle (20% 2-hydroxypropyl-?-cyclodextrin, Sigma, St Louis, MO), ethinyl estradiol (EE, 0.15 mg/kg; Sigma), estradiol benzoate (EB, 0.15 mg/kg; Sigma), equilin (EQ, 0.15 mg/kg; Sigma), tamoxifen (TAM, 3 mg/kg; Sigma), or raloxifene (RAL, 3 mg/kg; Eli Lilly & Co., Indianapolis, IN).    Manipulations of biological samples and protocols used:  Ovariectomized female rats were treated by daily gavage for 3 weeks with the vehicle (20% 2-hydroxypropyl-?-cyclodextrin, Sigma, St Louis, MO), ethinyl estradiol (EE, 0.15 mg/kg; Sigma), estradiol benzoate (EB, 0.15 mg/kg; Sigma), equilin (EQ, 0.15 mg/kg; Sigma), tamoxifen (TAM, 3 mg/kg; Sigma), or raloxifene (RAL, 3 mg/kg; Eli Lilly & Co., Indianapolis, IN).   After the final gavage treatment on day 21, the rats were deeply anesthetized with isoflurane and euthanized by decapitation.  The mesenteric arterial tree was isolated, harvested, and stored in RNAlater (Ambion) at -80ÂºC.  The mesenteric arteries were dissected free from veins and surrounding fat and homogenized in TRI reagent (MRC, Cincinnati, OH).  Experimental factor:  hormone treatment, drug treatment   Technical protocols:  The mesenteric arteries were minced with scalpels and TRI reagent (MRC) was added.  The tissue was homogenized with a Polytron homogenizer.  Bromochloropropane and sodium acetate were added, and the samples were centrifuged to separate the phases.  The RNA-containing layer was removed and the RNA purified on an RNeasy extraction column (Qiagen).  The sample was treated with an on-column DNase treatment (RNase-free DNase, Qiagen).  The purity and quantity were evaluated by an Agilent Bioanalyzer using the RNA 6000 Nanoassay LabChip.  Labeled cRNA was prepared using the Message Amp II-Biotin Enhanced Kit from Ambion, following the manufacturer's Instruction Protocol.  5.0 microgram of rat mesenteric artery total RNA was mixed with bacterial control RNA spikes and primed with T7 oligo(dT) primer for 10 min at 70C.  (The bacterial control spikes included araB, entF, fixB, gnd, hisB, and leuB.)  The first strand of cDNA was synthesized with first strand buffer, dNTP mix, RNase inhibitor, and reverse transcriptase for 2 h at 42C.  The second strand cDNA synthesis reaction was prepared using second strand buffer, dNTP mix, DNA polymerase mix, and RNase H; the reaction was carried out for 2h at 16C.  The double-stranded cDNA was purified on spin columns included in the Ambion kit.  The double-stranded cDNA was mixed with T7 reaction buffer, T7 ATP, T7 GTP, T7 UTP, T7 CTP, biotin-11-UTP, and T7 enzyme mix for the synthesis of cRNA.  The cRNA synthesis reaction was terminated after 14h at 37C by purifying the cRNA on spin columns from the Ambion kit.  The concentration of cRNA was determined by spectrophotometry.    Hybridization procedures and parameters:  10 micrograms of biotinylated cRNA was mixed with fragmentation buffer and heated to 94C for 20 min.  The fragmented cRNA was mixed with CodeLink hybridization buffer, loaded on the microarray slides (CodeLink Whole Rat Genome Bioarrays), and hybridized for 18 hours at 37C.   The slides were washed in 0.75x TNT (Tris-HCl, NaCl, Tween-20) at 46C for 1h then incubated with streptavidin-Alexa 647 fluorescent dye for 30 min at room temperature.  The streptavidin-Alexa 647 fluor was prepared in TNB blocking buffer (0.1M Tris-HCl, 0.15M NaCL, 0.5% NEN Blocking Reagent-PerkinElmer)  The slides were then washed 4 times for 5 min each in 1x TNT and twice in 0.05% Tween 20 for 5 sec each.  The slides were dried by centrifugation and scanned in an Axon GenePix 4000B scanner.","AE_experiment_type":"EFO\u001fhttp://www.ebi.ac.uk/efo/EFO_0002768","AE_species":"NCBITAXON\u001fhttp://purl.obolibrary.org/obo/NCBITaxon_10116","AE_name":"Estrogens and selective estrogen receptor modulators regulate gene and protein expression in the mesenteric arteries"},{"AE_description":"The cytokine IL-2 determines T cell fate by controlling T cell proliferation and differentiation, but the expression files of IL-2 regulated genes are not defined; Using Affymetrix mouse genome array and mouse T cell line CTLL-2, we analyzed the expression profiles of IL-2 regulated genes over 10 time points in 24 hours after IL-2 treatment. A total of 2,813 genes were differentially expressed (?2-fold change) at least at one time point in response to IL-2 stimulation. Clustering analyses showed that these genes consist of 9 clusters, suggesting that the IL-2 regulated genes in mouse T cells have 9 expression patterns. These genes involve in many biological processes, including immune response, signal transduction, apoptosis, cell cycle, transcription, transport, biosynthesis and various metabolisms. Two supplementary tables are linked to this series:; Table 1:  IL-2 responsive genes over the first 24 hours post-stimulation; Table 2:  Newly identified IL-2 regulated genes function in a variety of biological processes Experiment Overall Design: Murine cytotoxic T lymphocyte cell line CTLL-2, derived from a C57BL/6 mouse, is an IL-2 dependent cell line. Cells were cultured in D10 medium in an incubator with 5% CO2 at 37oC to mid-log phase (~2 X 105 cells/mL), and collected by centrifugation, washed with phosphate buffered saline (PBS) (Invitrogen), resuspended at 1 X 106 cells/mL in the medium without IL-2 (IL-2 starvation) and cultured for 14 hours (No IL-2 stimulation, T0, 5 reps). After adding human rIL-2 (100 U/mL, Chiron, IL-2 stimulation), cells were collected at time point 0.5, 1, 2, 4, 6, 8, 10, 12, 16 and 24 hours, at least 3 biological replicates were carried out for each time point.Total RNA were extracted from the cells and was hybridized on Affymetrix GeneChip Mouse Genome 430 2.0 Array.","AE_experiment_type":"EFO\u001fhttp://www.ebi.ac.uk/efo/EFO_0002768","AE_species":"NCBITAXON\u001fhttp://purl.obolibrary.org/obo/NCBITaxon_10090","AE_name":"Transcription profiling of mouse cytotoxic T lymphocyte cell line CTLL-2 treated with interleukin-2"},{"AE_description":"Exosomes are molecular entities derived from membrane vesicles of endocytic origin secreted by most cell types. These vesicles are implicated in cell-to-cell communication, deliver proteins and mRNA molecules between cells. Recent studies have shown that exosomes are found in body fluids such as saliva, blood, urine, amniotic fluid, malignant ascites, bronchoalveolar lavage fluid, synovial fluids and breast milk. Exosomes secreted through human saliva contain mRNA may potentially be useful for diagnostic purposes. Although the exact protective mechanism of saliva RNA is a topic of debate, the consensus is that the enrichment of mRNAs in these nano-vesicles in one of the features of the biomarker discoveries. Our aim was to determine if exosomes are present in human saliva and to nano-characterize their transcriptomic content.  Exosomes were purified by differential ultracentrifugation, identified by immunoelectron microscopy, flow cytometry and western blot using a CD-63 antibody. Atomic force microscopy studies revealed ultra structural analysis of both size and density of exosomes. Microarray analysis revealed the presence of 590 mRNA core transcripts are relatively stable inside the exosomes, which can be of saliva mRNA biomarkers.  Exosomal mRNA stability was determined by detergent lyses with treatment of RNase.  Under in vitro conditions fluorescent dye labeled saliva exosomes were able to communicate between human oral keratinocytes studied by using fluorescence microscopy. The RNA from saliva exosomes can transfer their genetic information to human oral keratinocytes and alters gene expression in the new location.  Together, these results suggest that saliva is involved in mRNA trafficking via exosomes, and provides a mechanism for cargoing passenger mRNAs. Our findings are consistent with proposal that exosomes can shuttle RNAs between cells and mRNA is protected inside these vesicles may be a possible resource for biomarker discovery.  Experiment Overall Design: Human saliva exosomes were purified through differential centrifugation followed by RNA extraction and hybridization on Affymetrix microarrays. We were able to obtain normal human subjects saliva which are pooled and subjected to ultracentrifugation. The protocol was approved by UCLA Institutional review board. 1 ml of saliva exosomes were used to extract RNA followed by two rounds of amplification by Actorus Amp kit. The amplified RNA was biotin labled and hybridized with Affymetrix protocol.","AE_experiment_type":"EFO\u001fhttp://www.ebi.ac.uk/efo/EFO_0002768","AE_species":"NCBITAXON\u001fhttp://purl.obolibrary.org/obo/NCBITaxon_9606","AE_name":"Transcription profiling of human saliva exosomes"},{"AE_description":"Escherichia coli strain MG1655 was grown in a Zn-depleted custom-built chemostat.  Culture volume (120 ml), temperature (37 oC) and stirring speed (440 rpm) were maintained.  Steady state values for pH and OD600 were 6.9 and 0.6, respectively.  Chemostats were grown for 50 h to allow five culture volumes to pass through the vessel and allow an apparent (pseudo-)steady state to be reached.  At this point, ZnSO4.7H2O in water was added to a final concentration of 0.2 M in the chemostat.  A 10 ml sample of culture was taken using a polypropylene pipette tip immediately prior to Zn addition and 2.5, 7, 10 and 30 min after addition.  Samples were harvested into RNAprotect and total RNA was purified using Qiagenâ€™s RNeasy Mini kit (using the supplierâ€™s protocol) prior to use in microarray analysis.  A control experiment was also carried out in which water was added.  Biological experiments were carried out twice (i.e. control (water added) and experiment (ZnSO4 added) chemostats were grown separately twice), and a dye swap performed for each experiment, providing at least two technical repeats for each of the two biological repeats at all five time points.  Slides were analysed so that time points â€œwith Znâ€ were compared to the same time point â€œwithout Znâ€ (e.g. 30 min after Zn was added was compared to 30 min after water being added). Cells were grown in glycerol-glycerophosphate medium (GGM) lacking Zn.  Final concentrations in GGM are: MES (40.0 mM), NH4Cl (18.7 mM), KCl (13.4 mM), beta-glycerophosphate (7.64 mM), glycerol (5.00 mM), K2SO4 (4.99 mM), MgCl2 (1.00 mM), EDTA (134 microM), CaCl2.2H2O (68.0 microM), FeCl3.6H2O (18.5 microM), H3BO3 (1.62 microM), CuCl2.2H2O (587 nM), CoNO3.6H2O (344 nM), and (NH4)6Mo7O24.4H2O (80.9 nM) in MilliQ water (Millipore).  Bulk elements (MES, NH4Cl, KCl, K2SO4 and glycerol in MilliQ water at pH 7.4 (batch growth) or 7.6 (continuous culture)) were passed through a column containing Chelex-100 ion exchange resin (Bio-Rad) to remove contaminating cations.  Trace elements (with no added Zn) and a CaCl2 solution were then added to give the final concentrations shown above prior to autoclaving.  After autoclaving, MgCl2 and beta-glycerophosphate were added at the final concentrations shown above.  All chemicals were of AnalaR grade of purity or higher.  Chelex-100 was packed into a Bio-Rad Glass Econo-column (approximately 120 mm Ã— 25 mm) that had previously been soaked in 3.5% nitric acid for 5 d.  E. coli strain MG1655 was grown a custom-built chemostat made entirely of non-metal parts.  Glass growth vessels and flow-back traps were soaked extensively (approximately two months) in 10% nitric acid before being rinsed thoroughly in MilliQ water.  Vent filters (Vent Acro 50 from VWR) were connected to the vessel using PTFE tubing.  Metal-free pipette tips were used (MAXYMum Recovery Filter Tips from Axygen).  Culture volume was maintained at 120 ml using an overflow weir in the chemostat vessel.  The dilution rate (and hence the specific growth rate) was 0.1 h-1.   The vessel was inoculated using one of the side-arms.  Flasks were placed on KMO 2 Basic IKA-Werke stirrers (arbitrary setting: 400).  Stirring speed was calibrated (440 rpm) and matched between vessels using a handheld laser tachometer (Compact Instruments Ltd).  Temperature was kept constant at 37 oC.  Twice daily, a 2-ml sample was taken and used to test the pH (which stayed constant at pH 6.9), OD600 (0.6 at steady state), glycerol limitation, and contamination on nutrient agar plates.  E. coli strain MG1655 was grown in a â€œZn-freeâ€ chemostat.  Chemostats were grown for 50 h to allow five culture volumes to pass through the vessel and allow an apparent (pseudo-)steady state to be reached.   At this point, ZnSO4.7H2O in water was added to a final concentration of 0.2 M in the chemostat.  A 10 ml sample of culture was taken using a polypropylene pipette tip immediately prior to Zn addition and 2.5, 7, 10 and 30 min after addition.  Total RNA was stabilised using RNAprotect (Qiagen).  Total RNA was purified using Qiagenâ€™s RNeasy mini kit.  A control experiment was carried out in which water was added.  Microarray slides were set up so that time points â€œwith Znâ€ were compared to the same time point â€œwithout Znâ€ (e.g. 30 min after Zn was added was compared to 30 min after water being added).  For each time point, equal quantities of RNA obtained after addition of zinc and addition of water were labelled by using nucleotide analogues of dCTP containing either Cy3 or Cy5 fluorescent dyes (Perkin Elmer).  For each microarray slide, one sample was labelled with Cy3-dCTP, while the other sample was labelled with Cy5-dCTP.  RNA (15-20 micrograms) was annealed to 9 micrograms pd(N)6 random hexamers (Amersham Biosciences) by heating at 65 oC for 10 min, followed by 10 min at 22 oC.  This was supplemented with 4 microlitres of 5Ã— First-strand buffer (Invitrogen), 2 microlitres dNTP mixture (5 mM each dATP, dTTP, dGTP and 2 mM dCTP), 2 microlitres 0.1 M DTT, 2 microlitres 1 mM Cy3 or Cy5 (Perkin Elmer) and 1 microlitre Superscript II reverse transcriptase (200 U) (Invitrogen).  This was incubated at 42 oC for 2 hours.  The reaction was terminated by the addition of NaOH (5 microlitres of 1M) and HCl (5 microlitres of 1M) before the addition of TE buffer (200 microlitres).  Labelled cDNA was purified using a Qiaquick PCR purification kit (Qiagen).  The Cy3-dCTP-labelled sample was mixed with the Cy5-dCTP-labelled sample and re-suspended in 120 microlitres salt-based hybridisation buffer (supplied with the microarray slides).  This was heated at 95 oC for 3 min then cooled on ice for 3 min.  The mixture was pipetted onto a microarray slide, sealed with a GeneFrame and coverslip in a hybridization chamber and incubated for 18 h at 42 Â°C.  Following hybridization, microarray slides (minus GeneFrame and coverslip) were washed in a series of pre-warmed (37 Â°C) SSC buffers for 5 min each at 37oC: 1Ã— SSC/0.1 % SDS, 1Ã— SSC, 0.2Ã— SSC and 0.01Ã— SSC.  Microarray slides were dried by centrifugation at 500 Ã— g for 2 min before scanning.  Slides were scanned on an Affymetrix 428 scanner.  The average signal intensity and local background correction were obtained using a commercially available software package from Biodiscovery, Inc (Imagene, version 4.0 and GeneSight, version 3.5).  Spots automatically flagged as bad, negative or poor in the Imagene software were removed before the statistical analysis was carried out in GeneSight.  The mean values from each channel were log2 transformed and normalised using the Lowess method to remove intensity-dependent effects in the log2(ratios) values.  The Cy3/Cy5 fluorescent ratios were calculated from the normalized values.  Biological experiments were carried out twice times, and a dye swap performed for each experiment, providing at least two technical repeats for each of the two biological repeats at all five time points.  Data from the independent experiments were combined.  Genes that were differentially expressed ? twofold and displayed and P value of < 0.05 (as determined by a t test) were defined as being statistically significantly differentially transcribed.","AE_experiment_type":"EFO\u001fhttp://www.ebi.ac.uk/efo/EFO_0002768","AE_species":"NCBITAXON\u001fhttp://purl.obolibrary.org/obo/NCBITaxon_562","AE_name":"Transcriptional profiling of Escherichia coli during a transition from zinc starvation to surfeit"},{"AE_description":"In order to study the function of the Campylobacter jejuni Cj0138 gene, a series of experiments were carried out. Three strains were constructed: a Cj0138 knockout strain, a strain where the Cj0138 knockout was complemented in trans, and a strain with a second copy over-expressing Cj0138 from an fdxA promoter. The transcriptomes of these were all compared to the wild-type strain. The arrays are all from RNA isolated in mid-exponential growth from independent biological replicates. Batch cultures of Campylobacter jejuni NCTC 11168 were grown in 50 ml volumes of Brucella broth in 70cm tissue culture flasks. Microaerophilic conditions were generated using a MACS-VA500 microaerophilic work station (5% Oxygen, 10% Carbon dioxide, 85% Nitrogen) from Don Whitley Scientific, Ltd which also maintained the growth temperature at 37 ÂºC. RNA was extracted from C. jejuni cultures grown to an OD600 of approximately 0.4. Briefly, 0.1 volume of 5% phenol in ethanol was mixed with the broth culture, and after centrifugation RNA was isolated with Tri Reagent (Sigma) and chloroform. RNA was further purified using the RNeasy kit (Qiagen) according to the manufacturerâ€™s instructions. The RNA was treated with Turbo DNA-free (Ambion) to remove any residual DNA according to the manufacturerâ€™s instructions. RNA concentration was determined using the Nanodrop Spectrophotometer NS-1000 (Thermo Scientific).  RNA preparations of each sample type were labelled and hybridized to custom-designed Agilent microarrays. Equal quantities of RNA from test and control cultures were labelled by using nucleotide homologues of dUTP containing either Cy3 or Cy5 fluorescent dye (Perkin Elmer). For each microarray slide, the test strain was labelled with Cy3-dUTP, while the wild-type sample was labelled with Cy5-dUTP. RNA (15 Âµg) was primed with 5 Âµg pd(N)6 random hexamers (Amersham Biosciences). The Affinity Script kit (GE Healthcare) was used to produce cDNA, and hybridized (Agilent Hi-RPM Gene Expression Hybridization Kit) to the microarray slide according to the manufacturerâ€™s instructions. Microarrays were scanned at 5 Âµm using an Axon 4000A scanner, and images were acquired using GenePix Pro 3.0 software (Axon).  Related analysis reference: Holmes K., Mulholland F., Pearson B. M., Pin C., McNicholl-Kennedy J., Ketley J. M., Wells J. M. (2005) Campylobacter jejuni gene expression in response to iron limitation and the role of Fur Microbiology-SGM 151 243-257 (PMID 15632442).","AE_experiment_type":"EFO\u001fhttp://www.ebi.ac.uk/efo/EFO_0002768","AE_species":"","AE_name":"Study of Cj0138 function in Campylobacter jejuni"},{"AE_description":"Preparation of nuclear extractsD. melanogaster Schneider cells were prepared as previously described by Andrews and Faller (1991). Briefly, in a typical preparation, 8x106 cells (grown in Schneider medium plus 10% fetal calf serum; GIBCO-BRL) were pelleted and resuspended in 1,5 mL cold PBS1X; the cell suspension is then tranferred to a microfuge tube. Cells are pelleted for 10 seconds and resuspended in 400 ÂµL cold Buffer A (10mM HEPES-KOH pH 7.9 at 4Â°C, 1.5mM MgCl2, 10mM KCl, 0.5mM duthiothreitol, 0.2mM PMSF, 0.2U/microL RNasin). The cells are allowed to swell on ice for 10 minutes, and then vortexed for 10 seconds. Samples are centrifuged for 10 seconds, and the supernatant fraction is discarded. The pellet is resuspended in 100 ÂµL of cold Buffer C (20mM HEPES-KOH pH 7.9, 25% glycerol, 420mM NaCl, 1.5mM MgCl2, 0.2mM EDTA, 0.5mM dithiothreitol, 0.2mM PMSF) and incubated on ice for 20 min for high-salt extraction. Cellular debris is removed by centrifugation for 2 min at 4Â°C and the supernatant fraction is stored at â€“70Â°C. All buffers were prepared from double-distilled autoclaved water that had been treated with 0.1% DEPC (Sigma).RNA immunoprecipitation and Reverse TranscriptionFor RNA immunopreciptation, the nuclear extract was incubated whit 50Âµg of monoclonal C1A9 anti HP1 antibody and the mixture was kept at  4Â°C overnight under continuous gentle movement. One-hundred  microliters ofprotein G-Sepharose (Sigma) suspension (50% packed Sepharose in Buffer C) was added  and the incubation was continued overnight as described.The beads were pelleted by 2 min centrifugation at 240 g at 4Â°C; the pellet was washed briefly three times whit each 1mL of IP Wash Solution (150mM NaCl, 50mM Tris pH 7.5, 0.5% NP40) and the Sepharose was transferred for eluition into a fresh plastic tube , pelleted again and then the supernatant was removed completely. To elute the immunocomplexes for protein analysis, an aliquot of beads were suspended in 50ÂµL SDS-PAGE sample buffer and incubated for 10 min at 90Â°C; following centrifugation the supernatant was removed and used for Western blot for testing the presence of HP1 protein.To elute the immunoprecipitated RNAs, the pelleted beads were boiled in 200ÂµL of DEPC wather for 5 min, spun, and the supernatant recovered; 1mL of Trizol (Invitrogen) was added to 200 ÂµL of supernatant and mixed well, followed by the addition of 200ÂµL of chloroform. This mixture was incubated at 4Â°C for 5 min and then centrifuged at 12000 g for 15 min; the RNAs in the aqueous phase was precipitated whit half volume of isopropanol; after precipitation, the RNAs was resuspend in 10ÂµL of DEPC wather. Contaminating DNA was digested whit RNase-free DNase I (Sigma). The RNA purified from the previous step is used as a template to synthesize cDNA using oligo dT , random hexamers and SuperScript reverse transcriptase III (Invitrogen) according to the manufacturer's protocol.  cDNA amplification and labelingThe cDNA was used as template for a two-step random PCR amplification; in Round A, Sequenase is used to extend randomly annealed primers (Primer A) to generate templates for subsequent PCR; during Round B, the specific primer B is used to aplify the templates previously generated and finally round C consist of additional PCR cycles to inorporate the amino allyl dUTP nucleotide. About 25 ng of each cDNA sample was used for two 8 min extension with 2,7 mM Round A primer (5â€™-GTT TCC CAG TCA CGA TCN NNN NNN NN-3â€™, N being a mixture of all four nucleotides with 60% A+T and 40% G+C) at 37Â°C with 267U/ml Sequenase version 2.0 (usb). DNA was denatured at 94Â°C for 2 min and cooled to 10Â°C , and  Sequenase 2.0 was added between extensions. The resulting products were used as template for 25 cycles of PCR using 1U/100Âµl Taq polymerase (Platinum Taq Invitrogen) and 10mM Round B primer (5â€™-GTT TCC CAG TCA CGA TC-3â€™). Finally this DNA was used as template for 25  cycles PCR to inorporate the amino allyl dUTP nucleotides to which the fluorescent dye may then be attached. To remove Tris buffer  which interferes with the indirect coupling, the aminoallyl-cDNA samples were desalted by filtering through a Microcon â€“30 and then mixed with the succinimidyl esters of the Cy3 or Cy5 dyes (Amersham Biosciences) in 0,1M sodium bicarbonate buffer (pH 9); the coupling reaction was icubated overnight in the dark at room temperature.Each dye labeled sample was purified by AutoSeq MicroSpin G-50 columns (Amersham Biosciences) following the manufacterers directions.Microarrays Hybridization and washing The dried labeled cDNAs pellet were resuspended in 80Âµl of DIG Easy Hyb (Roche) hybridization buffer containg  0,5 mg/ml yeast tRNa and 0,5 mg/ml salmon sperm DNA. Finally the DNA probe was denaturated by incubation at 65Â°C for 10 min  and, after a brief centrifugation to spin down the drops,  the mixture was pipetted onto a microarray; a coverslip was applied  and the slide was placed in a microarray hybridation chamber (BioRad) and incubated overnight at 37Â°C.After hybridization, the slide was submerged in 0,01% SDS, 1%SSC until the coverslip slipped off the surface; The slide was transferred to a solution of 1%SSC and shaken at 50Â°C for 10 min, then washed once more by shaking in 0,1%SSC.Slide was dried by centrifugation for 3 min at 550 rpm and scanned with an ScanArray Lite Microarray Scanner (Packard Bioscience) with laser intensities chosen to maximize signals while avoiding pixel saturation.ScanArray express softwere was used to quantify hybridation signals; bad spots were flagged automatically by softwere and subsequently each slide was inspected manually.","AE_experiment_type":"EFO\u001fhttp://www.ebi.ac.uk/efo/EFO_0002760","AE_species":"NCBITAXON\u001fhttp://purl.obolibrary.org/obo/NCBITaxon_7227","AE_name":"Chromatin immunoprecipitation of Drosophila Schneider cells using antibodies against HP1a"},{"AE_description":"Transforming growth factor beta-1 (TGFbeta) is a tumor suppressor during the initial stage of tumorigenesis, but it can switch to a tumor promoter during neoplastic progression. Ionizing radiation (IR), both a carcinogen and a therapeutic agent, induces TGFbeta activation in vivo. We now show that IR sensitizes human mammary epithelial cells (HMEC) to undergo TGFbeta-mediated epithelial to mesenchymal transition (EMT). Non-malignant HMEC (MCF10A, HMT3522 S1 and 184v) were irradiated with 2 Gy shortly after attachment in monolayer culture, or treated with a low concentration of TGFbeta (0.4 ng/ml), or double-treated. All double-treated (IR+TGFbeta) HMEC underwent a morphological shift from cuboidal to spindle-shaped. This phenotype was accompanied by decreased expression of epithelial markers E-cadherin, beta-catenin and ZO-1, remodeling of the actin cytoskeleton, and increased expression of mesenchymal markers N-cadherin, fibronectin and vimentin. Furthermore, double-treatment increased cell motility, promoted invasion and disrupted acinar morphogenesis of cells subsequently plated in Matrigel. Neither radiation nor TGFbeta alone elicited EMT, even though IR increased chronic TGFbeta signaling and activity. Gene expression profiling revealed that double treated cells exhibit a specific 10-gene signature associated with Erk/MAPK signaling. We hypothesized that IR-induced MAPK activation primes non-malignant HMEC to undergo TGFbeta-mediated EMT. Consistent with this, Erk phosphorylation were transiently induced by irradiation, persisted in irradiated cells treated with TGFbeta, and treatment with U0126, a Mek inhibitor, blocked the EMT phenotype. Together, these data demonstrate that the interactions between radiation-induced signaling pathways elicit heritable phenotypes that could contribute to neoplastic progression. Experiment Overall Design: Nonmalignant human mammary epithelial MCF10A cells (passages 106 and 108) were seeded at cloning density in 35mm dishes (10^5 cells/dish). Cell culture medium consisted of 3ml/dish of MGEM serum free medium (Cambrex Inc.), supplemented or not with 400pg/ml recombinant Transforming Growth Factor-beta. Cells were irradiated or not 5h post plating using 160 KV X-ray with a total dose of 2Gy. Sham, IR-treated, TGFbeta-treated and double-treated (IR+TGFbeta) MCF10A cells were harvested 8 days post-IR. Briefly, cells were washed with PBS, denatured in Trizol, scraped off the dish and subjected to chloroform extraction. After centrifugation, the upper phase was precipitated with an equal volume of isopropanol. RNA precipitates were resuspended in RNase free water and further purified on RNeasy columns (Qiagen, Germany). RNA quality was assessed on an Agilent Bio-Analyzer. The dataset analyzed by microarray included biological duplicates for each treatment in two independent experiments and three sham treated samples. Microarray data were generated at the Lawrence Berkeley National Laboratory Molecular Profiling Laboratory (http://hta.lbl.gov) using a high-throughput, automated GeneChip system (Affymetrix). Briefly, target preparation, HT_HG-U133A array plate hybridization setup, washing and staining were performed on an Affymetrix robotic system (GCAS) using version 2.1 protocols. Scanning (protocol version 2.2.09) was performed on a CCD-based high throughput scanner (Affymetrix). Samples were analyzed and clustered with the (UNO) One Color GenetrafficTM software version 3.2-12 (Iobion Informatics LLC, Stratagene, La Jolla, CA). Genes whose expression was specifically altered by treatment were defined as those in which dye ratio was more than 1.75-fold (|mean log2ratio|>0.8) from baseline in at least three out of the four treated samples compared to the three sham samples. Significance analysis tests (p<0.05) were performed using Excel between sham samples and either IR, TGFbeta or TGFbeta+IR samples.","AE_experiment_type":"EFO\u001fhttp://www.ebi.ac.uk/efo/EFO_0002768","AE_species":"NCBITAXON\u001fhttp://purl.obolibrary.org/obo/NCBITaxon_9606","AE_name":"Transcription profiling of human MCF10A cells subjected to ionizing radiation and treatment with transforming growth factor beta-1."},{"AE_description":"A total of 215 gene were found to be differentially expressed; 90 of them were up regulated and 125 genes were down regulated. Most notably the functional categories that were affected include Virulence genes, carbohydrate genes and lipid synthesis genes. Some of the virulence genes were down regulated by more than 16-32 fold. These results indicate that SDH plays an important role in the regulation of virulence. S.pyogenes strain SF370 (wild-type) was procured from ATCC (ATCC 700294). M1-SDHHbtail (SDH cytoplasmically retained) mutant strain PGB-1 was obtained using pFW5 vector as described before ( Boel G. Jin, H. and Pancholi, V. (2005). Inhibition of cell surface export of groupA streptococcal anchorless surface dehydrogenase (SDH) affects bacterial adherence and antiphagocytic properties. Infect. Immun. 73:6237-6248).  Both strains  (M1-Wild-type and PGB1(M1-SDHHbtail) were grown in Todd-Hewitt broth to their late log phase. Bacteria  were harvested by centrifugation and washed twice with sterile PBS. Total RNA was isolated from  these washed streptococci using Qiagen RNeasy Mini kit.  For synthesis and labeling of cDNA, 20 ug of total RNA, 1.5 ug of random hexamer, 0.5mM dNTPs (except that 0.2mM of dTTP was replaced by the same amount of amino-allyl dUTP to incorporate dUTP into first-strand cDNA) were used in each reverse transcriptase reaction (Superscript II Reverse Transcriptase, Invitrogen).  Purified cDNA preparations from the wild-type and PGB1 mutant strains were labeled with either Alexafluor-555 or Alexafluor-647 depending on the experimental design ( i.e. dye swap experiment). Differentially labeled probes were then combined and purified.  Using three independently isolated RNA preparations (biological replicates), a total of 8 experiments (incorporating  dye swaps) were performed. Accordingly eight hybridization measurements for this mutant were performed. Thus Exp-1 and -2 (GSM380310, GSM380320) are the technical replicates of the biological sample-1, Exp-3,-4 and -5 (GSM380321, GSM380322, GSM380331) are technical replicates of biological sample-2 and Exp-6,-7, and -8 (GSM380332,GSM380333, GSM380334) are technical replicates of the biological sample-3.   Signals of the bound reagents on the microarray spots in terms of relative fluorescence values were measured and quantified by a laser scanner (GenePix 4100 ) at 10um/pixel resolution. The resulting images were processed using Gene Pix Pro  software (version 4.0, Axon instruments). The raw data were obtained in the form of GenPix  *.gpr output files.  The web application CARMAweb (Comprehensive R based microarray analysis web service) was used for the normalization and analysis of microarray data.  All raw data (in the form of  *. GPR files) were uploaded to the web application in the data directory. Using appropriate navigation tree, background correction from the foreground signal was applied and within microarray normalization was achieved using the Lowess method.  Genes flagged as bad spots by the scanning software were excluded from the analysis and all flagged spots were given a weight of zero. The normalized data were then subjected to fold-change analysis and t-statistics using Bioconductor multtest package. CARMAweb allows to set Log2 cut-off values for both the M (regulation) and the A (average expression) values. Differentially expressed genes (Log2 values of Mutant647-red vs. wild-type555-green or Log2 values Mutant-555(green) vs Wild-type 647-red) were defined based on cut-off value >1 Log2< i.e. all genes that show a two or more-fold up- or down-regulation.  Bioconductors Multtest package provided suitable method to adjust P values according to multiple hypothesis testing problems.","AE_experiment_type":"EFO\u001fhttp://www.ebi.ac.uk/efo/EFO_0002768","AE_species":"NCBITAXON\u001fhttp://purl.obolibrary.org/obo/NCBITaxon_1314","AE_name":"Microarray analysis for differential gene expression in M1-SDHHbtail"},{"AE_description":"Gene expression profiles of bakerâ€™s yeast during initial dough-fermentation were investigated using liquid fermentation media to obtain insights at the molecular level into rapid adaptation mechanisms of bakerâ€™s yeast. Results showed that onset of fermentation caused drastic changes in gene expression profiles within 15 min. Genes involved in the tricarboxylic acid (TCA) cycle were down-regulated and genes involved in glycolysis were up-regulated, indicating a metabolic shift from respiration to fermentation. Genes involved in ethanol production (PDC genes and ADH1), in glycerol synthesis (GPD1 and HOR2), and in low-affinity hexose transporters (HXT1 and HXT3) were up-regulated at the beginning of model dough-fermentation. Among genes up-regulated at 15 min, several genes classified as transcription were down-regulated within 30 min. These down-regulated genes are involved in messenger RNA splicing and ribosomal protein biogenesis, in zinc finger transcription factor proteins, and in transcriptional regulator (SRB8, MIG1). In contrast, genes involved in amino acid metabolism and in vitamin metabolism, such as arginine biosynthesis, riboflavin biosynthesis, and thiamin biosynthesis, were subsequently up-regulated after 30 min. Interestingly, the genes involved in the unfolded protein response (UPR) pathway were also subsequently up-regulated. Our study presents the first overall description of the transcriptional response of bakerâ€™s yeast during dough-fermentation, and will thus help clarify genomic responses to various stresses during commercial fermentation processes. Experiment Overall Design: Saccharomyces cerevisiae T128 was used as a model of typical commercial bakerâ€™s yeast used in Japan. After 18 h cultivation, cells in stationary phase were collected by centrifugation (2,700 ?g for 5 min). Some of the cell pellets were suspended in 900 ml of sterilized water. Cells for no-fermentation control were harvested after the fed-butch cultivation and stored until RNA extraction. Cell pellets (11,700 OD units) were suspended in 390 ml of lequid fermentation (LF) medium in a 500-ml flask and then fermented for 300 min. To investigate gene expression profiles during initial stages of dough-fermentation, cell samples for DNA microarray analysis were obtained from each culture medium at 15 min, 30 min, and 60 min. Cells in stationary phase were then collected by centrifugation (2,700?g for 5 min), and stored until RNA extraction.","AE_experiment_type":"EFO\u001fhttp://www.ebi.ac.uk/efo/EFO_0002768","AE_species":"NCBITAXON\u001fhttp://purl.obolibrary.org/obo/NCBITaxon_4932","AE_name":"Transcription profiling of S. cerevisiae T128 was used as a model of typical commercial bakers yeast used in Japan to investigation of stress responses during commercial fermentation"},{"AE_description":"Cyclic indole-3-carbinol (I3C) tetrameric derivative (CTet) is an anticancer molecule that has been shown to exert an antiproliferative activity in both MCF-7 and MDA-MB-231 breast cancer cell lines. To characterize the molecular mechanisms leading to the inhibition of the cell proliferation, gene expression analyses were conducted. MCF-7 and MDA-MB-231 breast cancer cells were plated in 6-wells culture plates at density of 150,000 cells/well and cultured overnight. Cellular treatments were conducted at 6uM and 12 uM concentration of CTet, or vehicle control, for 24 h. Cell survival was evaluated by trypan blue dye exclusion assay and, after washing in phosphate buffered saline (PBS), the cells were pelletted by centrifugation and stored at -20Â°C with 300Âµl of RNA-later solution (Sigma-aldrich). Total RNA was purified from treated and control cells using the RNeasy plus kit (Qiagen). Biotin-labeled cRNA was synthesized using the CodeLink iExpress Assay reagent kit (GE Healthcare), following the manufacturerâ€™s protocols. Biotin-labeled cRNA obtained from each biological sample was fragmented and hybridized against three independent arrays (10 ug each) at 37Â°C for 22 h. After hybridization, the arrays were washed, stained with Cy5-streptavidin and scanned using a ScanArray GX scanner (Perkin Elmer), with a resolution of 5 um. The image files generated by the scanner were processed using the Codelink Expression Analysis software (GE Healthcare). Normalized data from the Codelink software package were analyzed with GeneSifter software (www.genesifter.net; Geospiza Inc., Seattle, WA) for statistical validation and data mining. Normalized data of the two experiments were subjected to analysis of variance (ANOVA) and 5% false discovery rate calculation (Benjamini and Hochberg). The cut-off parameters for differential gene expression were p =0.01 and fold change threshold =2.","AE_experiment_type":"EFO\u001fhttp://www.ebi.ac.uk/efo/EFO_0002768","AE_species":"NCBITAXON\u001fhttp://purl.obolibrary.org/obo/NCBITaxon_9606","AE_name":"Transcription profiling in breast cancer cell lines after Ctet treatment"},{"AE_description":"In order to study the function of the Campylobacter jejuni Cj1103 gene, a series of experiments were carried out. Three strains were constructed: a Cj1103 knockout strain, a strain where the Cj1103 knockout was complemented in trans, and a strain with a second copy over-expressing Cj1103 from an metK promoter. The transcriptomes of these were all compared to the wild-type strain. The arrays are all from RNA isolated in mid-exponential growth from independent biological replicates. Batch cultures of Campylobacter jejuni NCTC 11168 were grown in 50 ml volumes of Brucella broth in 70cm tissue culture flasks. Microaerophilic conditions were generated using a MACS-VA500 microaerophilic work station (5% Oxygen, 10% Carbon dioxide, 85% Nitrogen) from Don Whitley Scientific, Ltd which also maintained the growth temperature at 37 ÂºC. RNA was extracted from C. jejuni cultures grown to an OD600 of approximately 0.4. Briefly, 0.1 volume of 5% phenol in ethanol was mixed with the broth culture, and after centrifugation RNA was isolated with Tri Reagent (Sigma) and chloroform. RNA was further purified using the RNeasy kit (Qiagen) according to the manufacturerâ€™s instructions. The RNA was treated with Turbo DNA-free (Ambion) to remove any residual DNA according to the manufacturerâ€™s instructions. RNA concentration was determined using the Nanodrop Spectrophotometer NS-1000 (Thermo Scientific).  Two or three independent RNA preparations (biological replicates) of each sample type were labelled and hybridized to custom-designed Agilent microarrays. Equal quantities of RNA from test and control cultures were labelled by using nucleotide homologues of dUTP containing either Cy3 or Cy5 fluorescent dye (Perkin Elmer). For each microarray slide, the test strain was labelled with Cy3-dUTP, while the wild-type sample was labelled with Cy5-dUTP. RNA (15 Âµg) was primed with 5 Âµg pd(N)6 random hexamers (Amersham Biosciences). The Affinity Script kit (GE Healthcare) was used to produce cDNA, and hybridized (Agilent Hi-RPM Gene Expression Hybridization Kit) to the microarray slide according to the manufacturerâ€™s instructions. Microarrays were scanned at 5 Âµm using an Axon 4000A scanner, and images were acquired using GenePix Pro 3.0 software (Axon).  Related analysis reference: Holmes K., Mulholland F., Pearson B. M., Pin C., McNicholl-Kennedy J., Ketley J. M., Wells J. M. (2005) Campylobacter jejuni gene expression in response to iron limitation and the role of Fur Microbiology-SGM 151 243-257 (PMID 15632442).","AE_experiment_type":"EFO\u001fhttp://www.ebi.ac.uk/efo/EFO_0002768","AE_species":"","AE_name":"Study of Cj1103 function in Campylobacter jejuni"},{"AE_description":"Endometriosis is an inflammatory disease and bone marrow-derived cells are abundant in endometriotic lesions and in the peritoneal fluid of women with the disease.  This study tested the hypothesis that reciprocal communication occurs between macrophages and cultured human endometrial stromal cells and that this communication contributes to the pathology of endometriosis.  Changes in gene expression elicited by exposure to factors secreted by the opposing cell type were measured by DNA microarray to test this hypothesis.  716 named genes were differentially expressed in cultured endometrial stromal cells in response to factors secreted by macrophages.  Genes that were up-regulated included IL8/CXCL8, MMP3, phospholamban, CYR61/CCN1, CTGF/CCN2, tenascin C, and NNMT, whereas integrin alpha 6 was down-regulated.  In contrast, 15 named genes were differentially expressed in macrophages in response to factors secreted by cultured endometrial stromal cells.   The data document reciprocal communication between macrophages and endometrial stromal cells and suggest that interaction with macrophages stimulates the expression of genes in endometrial stromal cells that contribute to migration, adhesion, invasion, neovascularization and mitosis of endometrial cells that may support the establishment of endometriosis.  Experiment Design:   Goal of the experiment:  To examine reciprocal communication between a telomerase-immortalized human endometrial stromal cell line (THESC, ATCC #CRL-4003) and U-937 macrophages (ATCC #CRL-1593.2 treated with phorbol ester to stimulate differentiation), and also to assess the response of THESC cells to estrogen plus progesterone.  Keywords:  human, cultured endometrial stromal cell line, macrophage  Experimental factors:  hormone treatment and treatment with macrophage-conditioned medium of endometrial stromal cells, treatment of macrophages with endometrial stromal cell conditioned medium  Experimental design:  T HESC cells were grown in phenol red-free DMEM/F12 medium supplemented with 1% ITS+, 10% charcoal-dextran treated fetal bovine serum albumin, 1% pen/strep, and 500 ng/ml puromycin.  THESC cells were treated for 8 days with vehicle (0.1% ethanol) or estradiol (10-8 M) plus progesterone (10-7 M medroxyprogesterone acetate).  On day 6, serum was removed from the culture medium.  On day 7, half of the cells were treated with U-937 macrophage-conditioned medium for 24 hours.  The cells were harvested and RNA extracted on day 8.  The experiment was repeated with three different passages of cells (passages 9, 11, and 13) to yield an experimental n of 3.  The four groups of cells were vehicle-treated control, estradiol + progesterone, control + macrophage-conditioned medium, and estradiol + progesterone + macrophage-conditioned medium.    The monocytic cell line, U-937, was maintained in suspension in RPMI-1640 with 10% fetal bovine serum and 1% pen/strep.  The U-937 cells were treated with 32 nM phorbol myristate acetate (PMA) for 48 h to stimulate differentiation into macrophages.  The U-937 macrophages were treated with THESC conditioned medium or with non-conditioned control medium.  Total RNA was collected after 24 hours.  The experiment was repeated with three different passages of cells (passages 7, 9, and 12) to yield an experimental n of 3.  The two experimental groups were macrophages treated with non-conditioned control medium and macrophages treated with THESC-conditioned medium.  Quality control steps:  The cRNA that was synthesized from each cultured cell passage and treatment was used for hybridization to a single CodeLink (Applied Microarrays) whole human genome microarray.  Only one sample was hybridized with each slide and only one dye (streptavidin-Alexa 647) was used so no dye swaps were necessary.  Bacterial control spikes were used per manufacturer's instructions.  Samples used, extract preparation and labelling: The origin of each biological sample:  The samples were telomerase-immortalized human endometrial stromal cells and human U-937 monocytic cell line differentiation into macrophages by treatment with phorbol ester.  The cell lines were obtained from American Type Culture Collection (#CRL-4003 and #CRL-1593.2).    Manipulations of biological samples and protocols used:  T HESC cells were grown in culture in phenol red-free DMEM/F12 medium supplemented with 1% ITS+, 2% charcoal/dextran treated fetal bovine serum, 1% penicillin/streptomycin, and 500 ng/ml puromycin.  The cells were treated for 8 days with vehicle (0.1% ethanol), estradiol (10-8 M) plus progesterone (10-7 M medroxyprogesterone acetate), vehicle + macrophage-conditioned medium, or estradiol (10-8 M) + progesterone (medroxyprogesterone acetate 10-7 M) + macrophage-conditioned medium.   On day 6, serum was removed from the culture medium of all cells.  On day 7, half of the cells were treated with macrophage-conditioned medium for 24 hours.  The cells were harvested and RNA extracted on day 8.  The experiment was repeated with three different passages of cells (passages 9, 11, and 13) to yield an experimental n of 3.  The four groups of cells were vehicle-treated control, estradiol + progesterone, control + macrophage-conditioned medium, and estradiol + progesterone + macrophage-conditioned medium.  To prepare macrophage-conditioned medium, U-937 (ATCC #CRL-1593.2) cells were grown in RPMI-1640 medium with 10% fetal bovine serum until the cells covered 80% of the flask surface.  These U-937 cells were treated with 32 nM PMA for 48 h to stimulate differentiation into macrophages.  Differentiated macrophages were maintained in phenol red-free, fetal bovine serum-free DMEM/F12 medium supplemented with 1% penicillin/streptomycin for 48 hours.  The cell suspension was centrifuged and the conditioned medium was collected, filtered, and stored at -80C.  Before use, macrophage-conditioned medium was supplemented with 1% ITS+ and puromycin.      The monocytic cell line, U-937, was maintained in suspension in RPMI-1640 with 10% fetal bovine serum and 1% pen/strep.  The U-937 cells were treated with 32 nM PMA for 48 h to stimulate differentiation into macrophages.  The differentiated macrophages were treated with THESC conditioned medium or with non-conditioned control medium.  Total RNA was collected after 24 hours.    Experimental factor:  hormone treatment, treatment with macrophage-conditioned medium, and treatment with T-HESC cell conditioned medium  Technical protocols:  The cells were rinsed and TRI reagent (MRC) was added to the culture flasks.  The cells were scraped into centrifuge tubes, bromochloropropane and sodium acetate were added, and the samples were centrifuged to separate the phases.  The RNA-containing layer was removed and the RNA purified on an RNeasy extraction column (Qiagen).  The sample was treated with an on-column DNase treatment (RNase-free DNase, Qiagen).  The purity and quantity were evaluated by an Agilent Bioanalyzer using the RNA 6000 Nanoassay LabChip.  Labelled cRNA was prepared using the Message Amp II-Biotin Enhanced Kit from Ambion, following the manufacturer's Instruction Protocol.  1.0 microgram of T-HESC cell total RNA or 2 micrograms of macrophage total RNA was mixed with bacterial control RNA spikes and primed with T7 oligo(dT) primer for 10 min at 70C.  (The bacterial control spikes included araB, entF, fixB, gnd, hisB, and leuB.)  The first strand of cDNA was synthesized with first strand buffer, dNTP mix, RNase inhibitor, and reverse transcriptase for 2 h at 42C.  The second strand cDNA synthesis reaction was prepared using second strand buffer, dNTP mix, DNA polymerase mix, and RNase H; the reaction was carried out for 2h at 16C.  The double-stranded cDNA was purified on spin columns included in the Ambion kit.  The double-stranded cDNA was mixed with T7 reaction buffer, T7 ATP, T7 GTP, T7 UTP, T7 CTP, biotin-11-UTP, and T7 enzyme mix for the synthesis of cRNA.  The cRNA synthesis reaction was terminated after 14h at 37C by purifying the cRNA on spin columns from the Ambion kit.  The concentration of cRNA was determined by spectrophotometry.    Hybridization procedures and parameters:  10 micrograms of biotinylated cRNA was mixed with fragmentation buffer and heated to 94C for 20 min.  The fragmented cRNA was mixed with CodeLink hybridization buffer, loaded on the microarray slides, and hybridized for 18 hours at 37C.   The slides were washed in 0.75x TNT (Tris-HCl, NaCl, Tween-20) at 46C for 1h then incubated with streptavidin-Alexa 647 fluorescent dye for 30 min at room temperature.  The streptavidin-Alexa 647 fluor was prepared in TNB blocking buffer (0.1M Tris-HCl, 0.15M NaCL, 0.5% NEN Blocking Reagent-PerkinElmer)  The slides were then washed 4 times for 5 min each in 1x TNT and twice in 0.05% Tween 20 for 5 sec each.  The slides were dried by centrifugation and scanned in an Axon GenePix 4000B scanner.","AE_experiment_type":"EFO\u001fhttp://www.ebi.ac.uk/efo/EFO_0002768","AE_species":"NCBITAXON\u001fhttp://purl.obolibrary.org/obo/NCBITaxon_9606","AE_name":"Reciprocal communication between endometrial stromal cells and macrophages"},{"AE_description":"Core binding factor (CBF) leukemias, characterized by translocations t(8;21) or inv(16)/t(16;16) targeting the core binding factor, constitute acute myeloid leukemia (AML) subgroups with favorable prognosis. However, about 40% of patients relapse, and the current classification system does not fully reflect this clinical heterogeneity. Previously, gene expression profiling (GEP) revealed two distinct CBF leukemia subgroups displaying significant outcome differences and identified apoptotic signaling, MAPKinase signaling and chemotherapy-resistance mechanisms among the most significant differentially regulated pathways. We now tested different inhibitors of the respective pathways in a cell line model (six cell lines reflecting the CBF subgroup specific gene expression alterations), and found apoptotic signaling to be differentiating between the CBF subgroup models. In accordance, primary samples from newly diagnosed CBF AML patients (n=23) also showed differential sensitivity to in vitro treatment with a Smac mimetic such as BV6, an antagonist of inhibitor of apoptosis (IAP) proteins , and ABT-737, a BCL2 inhibitor. Furthermore, GEP revealed the BV6 resistant cases to resemble the previously identified unfavorable CBF subgroup. Thus, our current findings show deregulated IAP expression and apoptotic signaling to differentiate clinically relevant CBF subgroups, which were independent of known molecular markers, thereby providing a starting point for novel therapeutic approaches. All samples were obtained from untreated patients at the time of diagnosis. Cells used for microarray analysis were collected from the purified fraction of mononuclear cells after Ficoll density centrifugation. Routine diagnostic algorithms, including the characterization of molecular markers are performed.","AE_experiment_type":"EFO\u001fhttp://www.ebi.ac.uk/efo/EFO_0002768","AE_species":"NCBITAXON\u001fhttp://purl.obolibrary.org/obo/NCBITaxon_9606","AE_name":"Deregulated apoptosis signaling in core binding factor leukemia differentiates clinically relevant, molecular marker independent subgroups"},{"AE_description":"To measure the relative changes of gene expression upon Gis2p depletion, 50 ml of delta Gis2 cells or the respective wild-type strain BY4741 were grown in YPD at 30 degrees celsius to an OD600 of 0.5- 0.6. Cells were harvested by centrifugation and washed twice with 5ml of bidistilled water (ddH2O). RNA was isolated by hot phenol extraction for microarray analysis. 20 ug of total RNA derived from wild-type and knock-out cells were reverse transcribed in the presence of 5-(3-aminoallyl)-dUTP and natural dNTPs with a mixture of N9 and dT20V primers, and cDNAs were covalently linked to Cy3 and Cy5 NHS-monoesters (GE HealthSciences Cat# RPN5661), respectively, and competitively hybridized onto yeast oligo arrays at 42 degrees celsius for 14 hours in formamide-based hybridization buffer. Microarrays were scanned with an Axonscanner 4200 (Molecular Devices) and analyzed with GenePix Pro 5.1 (Molecular devices) before uploading into Stanford Microarray Database. Biological Replicate","AE_experiment_type":"EFO\u001fhttp://www.ebi.ac.uk/efo/EFO_0002768","AE_species":"NCBITAXON\u001fhttp://purl.obolibrary.org/obo/NCBITaxon_4932","AE_name":"Gis2 mutant versus wild-type cells"},{"AE_description":"This study uses microarray analyses to examine gene expression profiles for Mycobacterium tuberculosis (Mtb) induced by exposure in vitro to bovine lung surfactant preparations that vary in apoprotein content: (i) whole lung surfactant (WLS) containing the complete mix of endogenous lipids and surfactant proteins (SP)-A, -B, -C, and -D; (ii) extracted lung surfactant (CLSE) containing lipids plus SP-B and -C; (iii) column-purified surfactant lipids (PPL) containing no apoproteins, and (iv) purified human SP-A. Exposure to WLS evoked a multitude of transcriptional responses in Mtb, with 52 genes up-regulated and 23 genes down-regulated at 30 min exposure, plus 146 genes up-regulated and 27 genes down-regulated at 2 h.  Notably, WLS rapidly induced several membrane-associated lipases that presumptively act on surfactant lipids as substrates, and a large number of genes involved in the synthesis of phthiocerol dimycocerosate (PDIM), a cell wall component known to be important in macrophage interactions and Mtb virulence. Exposure of Mtb to CLSE, PPL, or purified SP-A caused a substantially weaker transcriptional response (? 20 genes were induced) suggesting that interactions among multiple lipid-protein components of WLS may contribute to its effects on Mtb transcription.  Whole lung surfactant (WLS) was obtained by centrifugation of lavage fluid (0.15M NaCl) from the intact lungs of freshly killed calves (ONY, Inc, Amherst, NY). Centrifugation was done at 12,000 x g at room temperature to pellet the active large aggregate surfactant fraction. CLSE was prepared by extraction of WLS into chloroform-methanol by the method of Bligh and Dyer (1959), as described in our prior work (Notter et al. 2002). CLSE is conceptually identical to the substance of the clinical surfactant InfasurfÂ®. The composition of CLSE by weight was approximately 94% phospholipid, 4.5% cholesterol, and 1.5% protein (a mixture of SP-B and SP-C) as defined previously (Notter et al. 2002). Mixed lung surfactant phospholipids (PPL) were isolated from CLSE by column chromatography with an elution solvent of 1:1 (v/v) chloroform-methanol plus 5% 0.1 N HCl. Two passes through a Sephadex LH-20 column (Pharmacia-LKB Biotechnology, Piscataway, NJ) separated phospholipids away from hydrophobic surfactant apoproteins and neutral lipids. Acid remaining in PPL fractions was removed by a second chloroform-methanol extraction. Final PPL had a protein content below the limits of detection of both the Folin-phenol assay of Lowry et al. (1951) and the amido black assay of Kaplin and Pedersen (1989) modified by the addition of 15% sodium dodecyl sulfate (SDS) to allow accurate quantitation in the presence of lipid.  Human SP-A was a gift from Dr. J. R. Wright at Duke University, and was isolated from bronchoalveolar lavage from patients with alveolar proteinosis as described previously (Wright et al. 1987).  For exposure studies with Mtb, amounts of WLS, CLSE, and PPL used per experiment were 3.3 mg in a total volume of 4 ml. In preliminary studies, this concentration was shown to induce a transcriptional response in Mtb without affecting viability after a 2 h treatment as determined by serial dilutions compared to untreated Mtb.   SP-A was studied at a level of 0.165 mg based on its putative concentration of approximately 5% in endogenous surfactant.   To determine the effects of lung surfactant on the transcriptional response of Mtb, preparations of WLS, CLSE, or PPL (in 650 Âµl PBS) were initially coated onto the bottom surface of a 25 cm2 cell culture flask for 30 min at 37oC to mimic a surfactant layer. A minimum of two independent biological replicates were studied for each surfactant preparation. Purified human SP-A was also examined for its effects on Mtb transcription, and phosphate buffered saline (PBS) was used as a â€œno treatmentâ€ control. In experiments with SP-A (in 650 Âµl PBS), the pure apoprotein was either pre-coated on the flask as done with the surfactant preparations, or was directly added and incubated with Mtb during study. Mtb was added to the flask in 3.35 ml of DMEM (supplemented with 2% FCS and 5 mM HEPES) and exposed to coated surfactant/SP-A or PBS control for 30 min or 2h at 37oC, followed by RNA isolation as detailed by Rohde et al. (2008). GTC lysis/stabilization buffer was added, and pelleted Mtb were washed and then digested with lysozyme. TRIzol (Invitrogen) was added, and organisms were disrupted using glass beads in a Mini-Bead Beater (BioSpec Products, Inc. Bartlesville, OK). After extraction of Mtb lysates with chloroform, ethanol was added to the aqueous phase before loading samples onto RNeasy mini columns (Qiagen, Valencia, CA). Subsequent steps were then performed according to the manufacturerâ€™s instructions. Following elution with RNase-free water, the RNA samples were treated with DNase (Turbo DNAfree, Applied Biosystems, Foster City, CA), and stored at -70oC until analysis.     The MessageAmpTMII-Bacteria system (Applied Biosystems) was used to amplify mycobacterial RNA for all samples [16]. Total RNA was reverse transcribed to yield double-stranded amino-allyl modified aRNA, which was labeled with Alexa Fluor 555 and Alexa Fluor 647 (Invitrogen). Excess reactive dyes were removed with a Mega Clear purification kit (Applied Biosystems).  Next, Alexa-labeled aRNA from paired samples was dried using a Speedvac, and re-suspended in hybridization buffer (5x SSC, 25% formamide, 0.15% SDS, and 25 ?g salmon sperm DNA). The mixed samples were denatured by being heated to 95oC for and then cooled to 50oC prior to loading them onto microarrays.  The arrays were hybridized for approximately 15 h at 45oC, washed and then dried by centrifugation.  Fluorescence intensity data from each array were collected with a GenePix 4000B scanner (Axon Instruments, Union City, CA) using GenePixPro3.0 software (Axon Instruments).  Image analysis, i.e. spot identification, background measurements, quality assessment and flagging were conducted with Imagene software (Version 6.0, BioDiscovery, Inc., El Segundo, CA).  Poor quality spots were filtered and removed from the analysis.  Subsequent normalization, statistical analysis (i.e. t-test), and visualization of array data was conducted using Genespring (Version 7, Agilent Technologies, Palo Alto, CA).","AE_experiment_type":"EFO\u001fhttp://www.ebi.ac.uk/efo/EFO_0002768","AE_species":"NCBITAXON\u001fhttp://purl.obolibrary.org/obo/NCBITaxon_1773","AE_name":"Transcriptional responses of Mycobacterium tuberculosis to lung surfactant"},{"AE_description":"The transcriptome of Blochmannia floridanus, the endosymbiont of the carpenter ant Camponotus floridanus, is presented during eight developmental stages of its holometabolous host by use of a whole genome DNA-macroarray. The detected transcription patterns indicate the presence of local transcription units as well as global regulatory mechanisms. Yet, the overall regulation scale is very modest, rarely exceeding a factor of three which is in line with the low number of transcriptional regulators present in this reduced genome. A large number of genes show differential expression in different life stages and a distinct expression pattern of genes possibly involved in symbiotic function as compared to housekeeping genes is apparent. However, these transcriptional changes are small as compared to the changes in the number of bacteria during host development which is highest in pupae and in young imagines. Control of replication of the bacteria in certain life stages may therefore be the decisive parameter influencing overall gene expression of Blochmannia in the animal. The few highly expressed genes like those encoding molecular chaperones exhibit a significantly higher G+C-content than moderately expressed genes.  Keywords: Host stage-dependent gene expression C. floridanus laboratory colonies were maintained under constant conditions at 25Â°C with a 12 h light cycle in artificial nests and fed twice a week with cockroaches (Nauphoeta cinerea), Bhatkar agar (Bhatkar and Whitcomb, 1970), and honey water (50% wt/wt) ad libitum. Developmental stages were defined as L1 (very young larvae, still clustered), L2 (older larvae, approx. 2-4 mm), P1 (pupae before metamorphosis), P2 (pupae after metamorphosis, still uncolored), P3 (pupae shortly before eclosion with dark abdomens), W1 (workers shortly after eclosion, not completely melanized, no aggressive behavior), W2 (fully melanized adult workers from the nest, age not distinguishable), and W3 (adult workers from isolated subcolonies, at least 4 months old). For RNA preparations, midguts of the respective life stage (L2 - W3) were dissected and bacteria were isolated by homogenization and centrifugation. Endosymbionts from small larvae (L1) were isolated from whole animals. RNA for each cDNA array experiment was isolated from individual bacterial preparations; from L1 to W2, each developmental stage was represented by at least three independent experiments and samples from at least three different ant colonies. Two ant colonies (C79 and C118) were represented in every stage from L1 to W2 with additional samples from six other colonies. Old W3 workers were collected from two isolated subcolonies derived from the same mother colony, each sampled in duplicate.","AE_experiment_type":"EFO\u001fhttp://www.ebi.ac.uk/efo/EFO_0002768","AE_species":"NCBITAXON\u001fhttp://purl.obolibrary.org/obo/NCBITaxon_203907","AE_name":"Transcriptional profiling of Blochmannia floridanus during different host stages"},{"AE_description":"A total of 163 genes were found to be differentially expressed; 38 genes of them were up regulated and 125 genes were down regulated. Most notably the functional categories that were affected include Virulence genes (18 genes  11% of the total significantly differentiated genes), carbohydrate metabolism  and cell envelop realted genes ( 31, 19.1% of the total),  and aminoacid transport genes (21genes, 12.9%).  Among the virulence-related genes, the most notable one were those belong to the production of capsule, the M protein, exotoxin, NAD glycohydrolase and Streptolysin S. Some of the capsule related genes were down regulated by more than 64 fold. These results indicate that CdhA (group A Streptococcal Cell division controlling and Chain-forming cell wall hydrolase) plays an important role in the regulation of virulence. S.pyogenes strain SF370 (wild-type) was procured from ATCC (ATCC 700294). M1-CdhA(-) mutant strain was obtained using pFW5 vector containing spectinomycin resistance marker (aad9). The mutant lacking CdhA is highly defective in cell division and growth,  lacks antiphagocytic property and attenuated for virulence. Both strains  (M1-Wild-type and M1-CdhA(-) were grown in Todd-Hewitt broth to their late log phase. Bacteria  were harvested by centrifugation and washed twice with sterile PBS. Total RNA was isolated from  these washed streptococci using Qiagen RNeasy Mini kit.  For synthesis and labeling of cDNA, 20 ug of total RNA, 1.5 ug of random hexamer, 0.5mM dNTPs (except that 0.2mM of dTTP was replaced by the same amount of amino-allyl dUTP to incorporate dUTP into first-strand cDNA) were used in each reverse transcriptase reaction (Superscript II Reverse Transcriptase, Invitrogen).  Purified cDNA preparations from the wild-type and  M1CdhA(-) mutant strains were labeled with either Alexafluor-555 or Alexafluor-647 depending on the experimental design ( i.e. dye swap experiment). Differentially labeled probes were then combined and purified.  Using three independently isolated RNA preparations (biological replicates), a total of 11 experiments (incorporating  dye swaps) were performed. Accordingly eleven hybridization measurements for this mutant were performed. Thus Exp-1 to -4 (GSM388703, GSM388717, GSM388718 and GSM388719) are the technical replicates of the biological sample-1, Exp-5 to 8(GSM388732, GSM388733, GSM388734, and GSM388735) are technical replicates of biological sample-2 and Exp-9,-10, and -11 (GSM388736,GSM388737, and GSM388738) are technical replicates of the biological sample-3.   Signals of the bound reagents on the microarray spots in terms of relative fluorescence values were measured and quantified by a laser scanner (GenePix 4100 ) at 10um/pixel resolution. The resulting images were processed using Gene Pix Pro  software (version 4.0, Axon instruments). The raw data were obtained in the form of GenPix  *.gpr output files.  The web application CARMAweb (Comprehensive R based microarray analysis web service) was used for the normalization and analysis of microarray data.  All raw data (in the form of  *. GPR files) were uploaded to the web application in the data directory. Using appropriate navigation tree, background correction from the foreground signal was applied and within microarray normalization was achieved using the Lowess method.  Genes flagged as bad spots by the scanning software were excluded from the analysis and all flagged spots were given a weight of zero. The normalized data were then subjected to fold-change analysis and t-statistics using Bioconductor multtest package. CARMAweb allows to set Log2 cut-off values for both the M (regulation) and the A (average expression) values. Differentially expressed genes (Log2 values of Mutant647-red vs. wild-type555-green or Log2 values Mutant-555(green) vs Wild-type 647-red) were defined based on cut-off value >1 Log2< i.e. all genes that show a two or more-fold up- or down-regulation.  Bioconductors Multtest package provided suitable method to adjust P values according to multiple hypothesis testing problems.","AE_experiment_type":"EFO\u001fhttp://www.ebi.ac.uk/efo/EFO_0002768","AE_species":"NCBITAXON\u001fhttp://purl.obolibrary.org/obo/NCBITaxon_1314","AE_name":"Microarray analysis for Streptococcus pyogenes M1-CdhA (-) mutant"},{"AE_description":"Background: Lactococcus garvieae is a bacterial pathogen that affects different animal species and human.  Despite the widespread distribution and emerging clinical significance of L. garvieae in both veterinary and human medicine, there is almost a complete lack of knowledge about the genetic content of this microorganism. In the present study the genomic content of L. garvieae CECT 4531 was analyzed by bioinformatic tools and microarray-based comparative genomic hybridizations (CGH) experiments, using Lactococcus lactis subsp. lactis IL1403 and Streptococcus pneumoniae TIGR4 as reference microorganisms. Results: The combination and integration of in silico analyses and in vitro (CGH) experiments performed between the reference microorganisms allowed establishing an inter-species hybridization framework with a detection threshold based on the sequence similarity ?70%. With this threshold value, 267 genes were identified as having an analogue in L. garvieae, most of which (n = 258) have been documented for the first time in this pathogen. Most of these genes are related to ribosomal, sugar metabolism or energy conversion systems. Some identified genes could be involved in the pathogenesis of L. garvieae infections.  Conclusions: In this study a comparative analysis based on microarray interspecies hybridization and the use of bioinformatic tools were used for the first time to study the genetic content of L. garvieae CECT 4531. Towards this approach, we identified 267 potentially present genes in L. garvieae CECT 4531, some of which could be involved in the pathogenesis of L. garvieae infections, such as als or mycA. These results provide the first insight into the genome content of L. garvieae.  Array-based comparative genome hybridization (CGH): The L. lactis subsp. lactis IL1403 and S. pneumoniae TIGR4 microarrays used for the CGH analysis were purchased from Eurogentec (Serain, Belgium). The L. lactis microarray contains 4608 spots: 2126 duplicated ORFs, 32 negative controls and 324 empty spots. The S. pneumoniae microarray contains 4608 spots: 2087 duplicated ORFs, 224 negative controls and 210 empty spots.  The CGH experiments were performed by means of competitive hybridizations using DNA of L. lactis subsp. lactis IL1403 or S. pneumoniae TIGR4, depending on the array as positive controls. DNAs to be hybridized on the same array were labelled with Cy3-dUTP and Cy5-dUTP, respectively. For each microarray hybridization reaction, aliquots (1â€“2 Âµg) of labelled genomic DNAs of reference (labelled with Cy3) and test (labelled with Cy5) strains, were mixed in 45 Âµl EGT hybridization solution (Eurogentec, Serain, Belgium) and denatured at 65ÂºC for 2 min. The hybridization mixture was then loaded onto a microarray slide, covered with a coverslip and incubated at 38ÂºC overnight. Following hybridization, the slides were washed in 2 X SSC, 0.5% SDS for 5 min followed by 5 min in 1 X SSC, 0.25% SDS. Finally, slides were rinsed in 0.2 X SSC and dried by centrifugation.  The results presented herein represent a compilation of sixteen separate CGH experiments: L. lactis subsp. lactis IL1403 arrays (reference microorganism) were hybridized with S. pneumoniae TIGR4 (test microorganism) (n=2); S. pneumoniae TIGR4 arrays (reference microorganism) were hybridized with L. lactis subsp. lactis IL1403 (test microorganism) (n=2); L. lactis subsp. lactis IL1403 arrays (reference microorganism) were hybridized with L.garvieae CECT 4531 (test microorganism) (n=8); S. pneumoniae TIGR4 arrays (reference microorganism) were hybridized with L.garvieae CECT 4531 (test microorganism) (n=4).  Data acquisition and analysis: The microarray was scanned after hybridization using a Scanarray HT microarray scanner (Perkin-Elmer). The signal intensity of the two fluors was determined using ImaGene software (BioDiscovery, El Segundo, CA, USA). Microarray data were analysed using ImaGene software, Microsoft Excel and an in-house designed and built Microsoft Access database [30]. Gene calling was based on a signal-to-noise ratio (SNR) > 3 for each spot. After CGH experiments, a gene was considered to show a positive result when it was present in at least three of the four CGH assays. In the case of the L. garvieae CECT 4531 hybridizations with L. lactis subsp. lactis IL1403 arrays, it was necessary to perform a greater number of assays (n=8) due to the poor quality of one of the array batches used. Thus, the criteria chosen to determine a positive result in this case was when the gene was present in at least five of the eight CGH assays.","AE_experiment_type":"EFO\u001fhttp://www.ebi.ac.uk/efo/EFO_0000749","AE_species":"NCBITAXON\u001fhttp://purl.obolibrary.org/obo/NCBITaxon_1313","AE_name":"Analysis of the genome content of Lactococcus garvieae by genomic interspecies microarray hybridization"},{"AE_description":"Cells: ACH-2, and A3.01, were obtained through the NIH AIDS Research and Reference Reagent Program, Division of AIDS, NIAID, NIH. ACH-2, is a chronically infected cell line harboring HIV-1 LAV strain, while A3.01 is the corresponding parental uninfected cell line.  Cells were grown in RPMI-1640 (Invitrogen, San Diego, CA) with 10% fetal bovine serum (FBS, Invitrogen), 5% penicillin-streptomycin (Invitrogen), and 2mM glutamine (Invitrogen).  Cells were maintained at a concentration of 1x10e6 cells/ml in T-175 flasks. Cell concentrations and cell viability were monitored throughout the experiment at all time points studied.  Cells were induced by addition of 20 ng/mL of phorbol myristyl acetate (PMA or TPA, Sigma, St Louis, MO) for one hour, after which the cells were washed with phosphate buffered saline (PBS).  Cells were harvested by centrifugation at 1000 rpm for 10 minutes, at the following times after induction, 0.5, 3, 6, 8, 12, 18, 24, 48, 72, and 96 hour(s).  HIV-infected and uninfected cells maintained and harvested in parallel with the PMA-treated cells, but not induced with PMA ( No PMA)were also harvested. Harvested cells were washed thrice with ice-cold PBS to remove media; cell pellets were snap frozen using ethanol-dry ice mixture and stored at (80Â°C for subsequent RNA extraction. Total RNA Extraction:  Total RNA was extracted using RNEasy Midiprep Kits per manufacturer's protocol (Qiagen, Valencia, CA).  RNA concentrations and purity were measured by spectrophotometry, RNA quality (absence of RNA degradation) was assessed by gel electrophoresis.  RNA concentration was adjusted to the levels required for subsequent microarray experiment protocols by concentration in a SpeedVac (Savant Instruments, Holbrook, CA).  RNA samples (6-7 Âµg/ÂµL) were stored in 100 ÂµL TE buffer at -80Â°C.  Microarray Studies:  Total RNA obtained from induced chronically infected and corresponding uninfected parental cells were used for microarray experiments.  For each time point, RNA from the induced chronically infected ACH-2 cells and RNA from the corresponding induced, uninfected A3.01 cells were compared to minimize effects due to PMA induction. Microarrays were obtained from the National Cancer Institute Microarray Facility, Advanced Technology Center (Gaithersburg, MD).  The microarrays (Hs. UniGem2) contained 10,368 cDNA spots on each glass slide. The cDNAs were selected for spotting on the slides based on their known or probable involvement in oncogenesis, signal transduction, apoptosis, immune function, inflammatory pathways, cellular transport, transcription, protein translation and other important cellular functions.  A number of expressed sequence tags (ESTs) from unknown genes homologous to known genes and cDNAs encoding housekeeping genes were also included in these gene sets.  For each time point, 50 Âµg of total RNA from PMA induced ACH-2 cells and 70 Âµg of total RNA from PMA induced A3.01 cells was labeled with Cy-3-dUTP and Cy-5-dUTP respectively.  Higher amounts of RNA were used for Cy-5 labeling to minimize the disparities in dye incorporation.  Each sample of RNA from PMA-induced, infected cells from a particular time point was compared with RNA from the corresponding PMA-induced, uninfected cells from the same time point for subsequent hybridization to the same array to ensure accurate comparisons and to eliminate inter-array variability. Following reverse transcription, the labeled cDNAs were then combined and purified using MicroCon YM-30 (Millipore, Bedford, MA) spin column filters, to remove any unincorporated nucleotides. 8-10 Âµg each of Cot-1 DNA, (Boehringer Mannheim, Indianapolis, IN), yeast tRNA (Sigma) and polyA (Amersham Biosciences) were added to the reaction mixture and heated at 100Â°C for 1 minute.  Hybridization of the labeled cDNA to the microarray was carried out at 65Â°C overnight, followed by washes with 1X SSC, 0.2X SSC and 0.05X SSC respectively.  The slides were dried by centrifugation at 1000 rpm for 3 minutes and then scanned as described below.  Microarray Scanning and Data Analysis:  The slides were scanned using an Axon GenePix 4000 scanner (Axon Instruments, Union City, CA). The photomultiplier tube values (PMT) were adjusted to obtain equivalent intensities at both wavelengths used, 635 nm and 532 nm for the Cy5 and Cy3 channels respectively.  Image analysis was performed using GenePix analysis software (Axon Instruments) and data analysis was performed using the microArray Database (mAdb) system hosted by the Center for Information Technology and Center for Cancer Research at NIH (http://nciarray.nci.nih.gov/). Keywords = HIV Keywords = latency","AE_experiment_type":"","AE_species":"NCBITAXON\u001fhttp://purl.obolibrary.org/obo/NCBITaxon_9606","AE_name":"Host Cell Gene Expression During HIV-1 Latency and Reactivation"},{"AE_description":"Cell survival in changing environments requires appropriate regulation of gene expression, including post-transcriptional gene regulatory mechanisms. Based on reporter gene studies in glucose-starved yeast, it was proposed that translationally silenced eukaryotic mRNAs accumulate in P-bodies and can return to active translation. We present evidence contradicting the notion that this model is a widespread and general phenomenon. First, genome-wide measurements of mRNA abundance, translation, and ribosome occupancy following glucose withdrawal show that most mRNAs are lost from the cell coincident with their loss from polysomes. Second, only a very limited sub-population of translationally repressed transcripts, comprising fewer than 400 genes, can be reactivated for translation upon glucose re-addition in the absence of new transcription. This highly selective post-transcriptional regulation could be a mechanism for cells to minimize the energetic costs of reversing gene-regulatory decisions in rapidly changing environments by transiently preserving a pool of transcripts whose translation is rate-limiting for growth. Sigma 1278b yeast were grown to OD600=1.0-1.1 in YPD in baffled flasks at 30C with vigorous shaking. Cells were treated with thiolutin to a final concentration of 3ug/mL, harvested by centrifugation, and resuspended in pre-warmed YPA medium lacking glucose. Cells were returned to 30C for shaking for 10 minutes. For the refed samples, glucose was added to a final concentration of 2% and samples were taken after 5 minutes. Polysome preparation and RNA isolation were as described in Experimental Procedues. A common RNA sample, polysomal RNA from cells grown in YPD, was used for one channel of each array.","AE_experiment_type":"EFO\u001fhttp://www.ebi.ac.uk/efo/EFO_0002768","AE_species":"NCBITAXON\u001fhttp://purl.obolibrary.org/obo/NCBITaxon_4932","AE_name":"Polysome-associated mRNA levels upon glucose repletion"},{"AE_description":"The faecal indicator bacterium Escherichia coli K12 was used to study the effects of the global regulators RpoS and cAMP at the transcription level using microarray technology during short-term (physiological) adaptation to slow growth under limited nutrient supply. Effects due to the absence of one global regulator were assessed by comparing the mRNA levels isolated from rpoS or cya mutants under glucose-limited continuous culture at a dilution rate of 0.3 h-1 for the rpoS mutant or 0.16 h-1 for the cya mutant with those from wt E. coli grown under the same conditions. Wild-type Escherichia coli K12 MG1655 (ATCC 700926), E. coli delta-rpoS (MG1655 delta-rpoS::Tn10; Wick et al. (2002)) and AMS2 (MG1655 delta-cya-854; Schultz et al. (1988)) were grown at 37Â°C in mineral medium. Steady-state with respect to optical density was reached approximately 20 hours after switching on the medium feed. Samples for catabolic pathway assays and transcriptome analysis were taken 40 hours after switching to the continuous culture mode, and each experiment was done in triplicate. 800 ml of culture liquid was withdrawn from the chemostat, immediately cooled down on crushed ice and cells were collected by centrifugation at 4Â°C.  Total RNA from E. coli cells was isolated (Sambrook et al., 1989). RNA in samples was quantified spectrophotometrically by measuring extinction at 260 nm, and purity (free from DNA) was checked by gel electrophoresis. Synthesis of cDNA from RNA was performed with the CyScribe First-Strand cDNA Labelling Kit (Amersham Bioscience, Little Chalfont, England). The purified cDNA was concentrated to 5 ul and was mixed with 120 ul of hybridization buffer (MWG-Biotech AG), heated to 95Â°C for 3 min and cooled down on ice for 3 min. The hybridization mixture was then added to the microarray slide and covered with a coverslip. The hybridization slide was incubated overnight at 42Â°C. After the hybridization step, the slide was washed three times, the first time for 5 min in 2x (times concentrated) SSC - 0.1 % SDS, the second time for 5 min in 1x SSC, and finally for 5 min in 0.1x SSC. SSC buffer was prepared as 20x solution containing 0.3M Na-citrate and 3M NaCl at pH 7.0. The slides were dried by centrifugation at room temperature for 2 min at 500g.  Microarray slides were scanned using the Affymetrix 428TM Array Scanner (High Wycombe, UK). Spot intensities and corresponding background signals were quantified with the Affymetrix JaguarTM software version 2.0. Further data analysis was performed with the program GeneSpring from Silicon Genetics (Redwood City, USA). Induction factors were calculated from the Cy3 and Cy5 signal intensities of the spot. Spots with signal intensity below a value of 50 were excluded from the analysis and the minimal induction factor was set to 0.01. The normalization was performed with the 50th percentile distribution of remaining spots after background correction. The mean value of the induction factors of a specific gene was calculated from three replicates. Biological experiments were carried out three times, which provided three biological repeats. Data from the independent experiment were combined, genes that were differently regulated ?3 and ?-3 (t-test, p ? 0.2) were defined as being statistically significant.","AE_experiment_type":"EFO\u001fhttp://www.ebi.ac.uk/efo/EFO_0002768","AE_species":"","AE_name":"Effect of the global regulators RpoS and cyclic-AMP/CRP on the transcriptome of Escherichia coli K12 during  carbon- and energy-limited growth"},{"AE_description":"Confluent human umbilical vein endothelial cells (HUVECs) were exposed to histamine (100 mM) for 2 hours.  Ribosomal profiling via gradient centrifugation and fractionation was used to separate monosome, or under-translated, and polysome, or actively translated, mRNA species that were then used to probe cDNA arrays, a process known as Translation State Array Analysis (TSAA).  Four samples were obtained from these experiments, control monosome, control polysome, histamine monosome and histamine polysome.  Using the normalized signal intensities from the GeneFilters, we calculated a translation index, or measure of movement of an mRNA molecule from the monosome to the polysome fraction upon stimulation.  This calculation was made as follows:  (histamine polysome/histamine monosome)/(control polysome/control monosome).  Translational indices greater than 2.5 (upregulated) or lower than 0.4 (downregulated) were chosen for further study.   HUVECs were stimulated with histamine (100 mM) for 2 hours.  Cycloheximide (CHX) was added to a final concentration of 100 ng/mL for 5 minutes and cells were then rinsed with cold HBSS containing CHX (100 ng/mL) and scraped from the dish and pelleted by centrifugation (2000 x g, 5 minutes).  The supernatant was removed and cells were carefully resuspended in 375 ?L Low Salt Buffer (LSB; 20 mM Tris, 10 mM NaCl, 3 mM MgCl2, pH 7.4) + RNasin (40 U/mL) + DTT (10 nM) for 3 minutes on ice.  125 ?L LSB Lysis buffer (LSB containing 200 mM sucrose and 1.2% Triton-X100) was then added and cells disrupted by pipetting.  The mixture was then transferred to 1.7 mL microfuge tubes and centrifuged (20,000 x g, 1 minute, 4? C) to remove nuclei and cellular debris.  The supernatant was transferred to a new tube containing 50 ?L LSB + RNasin (40 U/mL) + DTT (10 nM) and 15 ?L 5M NaCl.  This mixture was then layered onto prepoured gradients (15-50% sucrose in LSB).  Gradients were spun at 43,700 rpm for 90 minutes (4? C) and then run on a density gradient flow cell fractionator (Isco) to obtain ribosomal profiles.  Ribosomal fractions corresponding to monosomes (M) and polysomes (P) (Figure 1) were collected into Trizol LS (Invitrogen) and total RNA isolated according to the directions of the manufacturer.  Total RNA was reverse transcribed using oligo dT primers in the presence of 33P dCTP to generate labeled cDNA probes from the M and P associated mRNA fractions of control and stimulated cells and these probes used to detect specific array elements on four identical Research Genetics (ResGen) Named Human Gene cDNA GeneFilters (GF 211, Invitrogen).  These filters were washed using high stringency conditions and exposed to phosphorimaging screens for analysis using Pathways 3, proprietary software included with the ResGen filters.   Hybridization signals were normalized to the average intensity of the hybridization signals from the particular array filter to correct for differing global hybridization efficiencies between filters.  To determine if a particular hybridization signal was significantly above background levels, an initial screen of the array elements was conducted by dividing the hybridization signal of an individual cDNA spot by the background signal on particular filter.  Background signal levels were determined using the Pathways 3 analysis software that measures the signal intensity of the area between cDNA spots and averages that signal over the area of the entire array.  Array elements were considered to be distinguishable from background if the hybridization signal/background ratio was greater than 1.1.  Signal intensities from each filter were normalized by dividing the raw signal intensity of the cDNA spot by the average signal intensity for the entire filter.  The translation state (TS) of each qualifying array element was determined by dividing the normalized signal intensity of the element in the polysome fraction by the normalized signal intensity of the element in the monosome fraction:  TS=P/M.  The Translation Index (TLI), a measure of the change of translation states of individual genes, was determined by calculating the ratio of the treated cell TS to the control cell TS; TLI = TSe/TSc where e represents experimental or treated conditions and c represents controls.  In this scenario, a gene that is not regulated by translation would have a TLI value of 1, actively translated genes would have a TLI value greater than 1, whereas a poorly translated or under-translated gene would have a TLI value less than 1.  Arbitrary TLI values of 2.5 and 0.4 were used to categorize individual array elements as being translationally regulated.","AE_experiment_type":"","AE_species":"NCBITAXON\u001fhttp://purl.obolibrary.org/obo/NCBITaxon_9606","AE_name":"Translation State Array Analysis of Histamine Stimulated Human Endothelial Cells"},{"AE_description":"Confluent human umbilical vein endothelial cells (HUVECs) were exposed to TNF-alpha (100 units/mL) for 2 hours. Ribosomal profiling via gradient centrifugation and fractionation was used to separate monosome, or under-translated, and polysome, or actively-translated, mRNA species that were then used to probe cDNA arrays, a process known as Translation State Array Analysis (TSAA). Four samples were obtained from these experiments: control monosome, control polysome, TNF monosome and TNF polysome. using the normalized signal intensities from the GeneFilters, we calculated a translation index, or measure of movement of an mRNA molecule from the monosome to the polysome fraction upon stimulation. This calculation was made as follows: (TNF polysome/TNF monosome)/(control polysome/control monosome). Translational indices greater than 2.5 (upregulated) or lower than 0.4 (downregulated) were chosen for further study. HUVECs were stimulated with TNF-alpha (100 U/mL) for 2 hours.  Cycloheximide (CHX) was added to a final concentration of 100 ng/mL for 5 minutes and cells were then rinsed with cold HBSS containing CHX (100 ng/mL) and scraped from the dish and pelleted by centrifugation (2000 x g, 5 minutes).  The supernatant was removed and cells were carefully resuspended in 375 micL Low Salt Buffer (LSB; 20 mM Tris, 10 mM NaCl, 3 mM MgCl2, pH 7.4) + RNasin (40 U/mL) + DTT (10 nM) for 3 minutes on ice.  125 micL LSB Lysis buffer (LSB containing 200 mM sucrose and 1.2% Triton-X100) was then added and cells disrupted by pipetting.  The mixture was then transferred to 1.7 mL microfuge tubes and centrifuged (20,000 x g, 1 minute, 4 deg C) to remove nuclei and cellular debris.  The supernatant was transferred to a new tube containing 50 micL LSB + RNasin (40 U/mL) + DTT (10 nM) and 15 micL 5M NaCl.  This mixture was then layered onto prepoured gradients (15-50% sucrose in LSB).  Gradients were spun at 43,700 rpm for 90 minutes (4 deg C) and then run on a density gradient flow cell fractionator (Isco) to obtain ribosomal profiles.  Ribosomal fractions corresponding to monosomes (M) and polysomes (P) (Figure 1) were collected into Trizol LS (Invitrogen) and total RNA isolated according to the directions of the manufacturer.  Total RNA was reverse transcribed using oligo dT primers in the presence of 33P dCTP to generate labeled cDNA probes from the M and P associated mRNA fractions of control and stimulated cells and these probes used to detect specific array elements on four identical Research Genetics (ResGen) Named Human Gene cDNA GeneFilters (GF 211, Invitrogen).  These filters were washed using high stringency conditions and exposed to phosphorimaging screens for analysis using Pathways 3, proprietary software included with the ResGen filters.   Hybridization signals were normalized to the average intensity of the hybridization signals from the particular array filter to correct for differing global hybridization efficiencies between filters.  To determine if a particular hybridization signal was significantly above background levels, an initial screen of the array elements was conducted by dividing the hybridization signal of an individual cDNA spot by the background signal on particular filter.  Background signal levels were determined using the Pathways 3 analysis software that measures the signal intensity of the area between cDNA spots and averages that signal over the area of the entire array.  Array elements were considered to be distinguishable from background if the hybridization signal/background ratio was greater than 1.1.  Signal intensities from each filter were normalized by dividing the raw signal intensity of the cDNA spot by the average signal intensity for the entire filter.  The translation state (TS) of each qualifying array element was determined by dividing the normalized signal intensity of the element in the polysome fraction by the normalized signal intensity of the element in the monosome fraction:  TS=P/M.  The Translation Index (TLI), a measure of the change of translation states of individual genes, was determined by calculating the ratio of the treated cell TS to the control cell TS; TLI = TSe/TSc where e represents experimental or treated conditions and c represents controls.  In this scenario, a gene that is not regulated by translation would have a TLI value of 1, actively translated genes would have a TLI value greater than 1, whereas a poorly translated or under-translated gene would have a TLI value less than 1.  Arbitrary TLI values of 2.5 and 0.4 were used to categorize individual array elements as being translationally regulated.","AE_experiment_type":"","AE_species":"NCBITAXON\u001fhttp://purl.obolibrary.org/obo/NCBITaxon_9606","AE_name":"Translation State Array Analysis of TNF Stimulated Human Endothelial Cells"},{"AE_description":"The co-utilization of hexoses and pentoses derived from lignocellulose is an attractive trait for in microorganisms considered for consolidated biomass processing to biofuels. This issue was examined for the H2-producing, extremely thermophilic bacterium Caldicellulosiruptor saccharolyticus growing on oligosaccharides (composed of glucose and xylose) compared to monosaccharides. Based on the whole-genome transcriptional response analysis and comparative genomics, carbohydrate specificities for transport systems could be proposed for a number of the 24 putative carbohydrate ATP-binding cassette (ABC) transporters in the C. saccharolyticus genome. Transcriptome analysis showed different transporters were utilized for uptake of oligosaccharides than those used for monosaccharides uptake. No evidence for catabolite repression was noted for either growth on multi-sugar mixtures or in the corresponding transcriptomes. C. saccharolyticus was subcultured (overnight) seven times on the substrate of interest in modified DSMZ 640 medium before inoculating a 1-liter batch containing 0.5 gram substrate per liter. Cells were grown at 70 Â°C until mid-logarithmic phase (3-5*107) and harvested by rapid cooling to 4 Â°C and centrifugation and then stored at -80 Â°C. To elucidate the transporters plus the central carbon metabolic pathways and their regulation utilized on the different sugars, transcriptome analysis was performed after growth on xylan (oat spelt), xyloglucan, xylogluco-oligosaccharides, glucose and xylose.","AE_experiment_type":"EFO\u001fhttp://www.ebi.ac.uk/efo/EFO_0002768","AE_species":"NCBITAXON\u001fhttp://purl.obolibrary.org/obo/NCBITaxon_44001","AE_name":"Csac growth on model substrates found in lignocellulose"},{"AE_description":"Background: Peripheral blood mononuclear cells (PBMCs) are relatively easily obtainable cells in humans. Gene expression profiles of PBMCs have been shown to reflect the pathological and physiological state of a person. Recently, we showed that the nuclear receptor peroxisome proliferator-activated receptor alpha (PPAR?) has a functional role in human PBMCs during fasting. However, the extent of the role of PPAR? in human PBMCs remains unclear. In this study, we therefore performed gene expression profiling of PBMCs incubated with the specific PPAR? ligand WY14,643. Results: Incubation of PBMCs with WY14,643 for 12 hours resulted in a differential expression of 1,373 of the 13,080 genes expressed in the PBMCs. Gene expression profiles showed a clear individual response to PPAR? activation between six healthy human blood donors, which was not the result of the nutritional status of the donors. Pathway analysis showed that genes in fatty acid metabolism, primarily in ?-oxidation were up-regulated upon activation of PPAR? with WY14,643, and genes in several amino acid metabolism pathways were down-regulated. Conclusions: This study shows that PPAR? in human PBMCs regulates fatty acid and amino acid metabolism. In addition, PBMC gene expression profiles show individual responses to WY14,643 activation. We show that PBMCs are a suitable model to study changes in PPAR? activation in healthy humans. Keywords: metabolic state analysis PBMCs from six healthy Caucasian male blood donors, aged between 30 and 48 yr, were isolated directly after arrival of the buffy coat (max. 8 hours after donation) by Ficol-paque Plus density gradient centrifugation (Amersham Biosciences, Roosendaal, the Netherlands). PBMCs were incubated in RPMI1640 medium with 2 mmol/L L-glutamine, 10% fetal bovine serum and antibiotics (penicillin and streptomycin) in the presence of 5% CO2 at 37Â°C. at 1.0 x 106 cells per ml with either WY14,643 (50 ?M) or vehicle (DMSO, 0.1%) for 12 hours. Total RNA from PBMCs was labeled using a one-cycle cDNA labeling kit (Affymetrix Inc, Santa Clara, CA) and hybridized to Affymetrix Human whole genome U133 plus 2.0 arrays (Affymetrix). Sample labeling, hybridization to chips and image scanning was performed according to the manufacturerâ€™s GeneChip Expression Analysis Technical Manual (Affymetrix).","AE_experiment_type":"","AE_species":"NCBITAXON\u001fhttp://purl.obolibrary.org/obo/NCBITaxon_9606","AE_name":"Activation of peroxisome proliferator-activated receptor alpha in human peripheral blood mononuclear cells"},{"AE_description":"Confluent human umbilical vein endothelial cells (HUVECs) were exposed to Thrombin (2 U/mL) for 2 hours.  Ribosomal profiling via gradient centrifugation and fractionation was used to separate monosome, or under-translated, and polysome, or actively translated, mRNA species that were then used to probe cDNA arrays, a process known as Translation State Array Analysis (TSAA).  Four samples were obtained from these experiments, Control Monosome, Control Polysome, Thrombin Monosome, and Thrombin Polysome.  Using the normalized signal intensities from the GeneFilters, we calculated a translation index, or measure of movement of an mRNA molecule from the monosome to the polysome fraction upon stimulation.  This calculation was made as follows:  (thrombin polysome/thrombin monosome)/(control polysome/control monosome).  Translational indices greater than 2.5 (upregulated) or lower than 0.4 (downregulated) were chosen for further study.  TSAA data suggests that JunB is translationally regulated by thrombin stimulation.  Immunocytochemistry, western blotting and RT-PCR were used to verify the results of TSAA.   Polysome Profiling.  HUVECs were stimulated with thrombin (2 U/mL) for 2 hours.  Cycloheximide (CHX) was added to a final concentration of 100 ng/mL for 5 minutes and cells were then rinsed with cold HBSS containing CHX (100 ng/mL) and scraped from the dish and pelleted by centrifugation (2000 x g, 5 minutes).  The supernatant was removed and cells were carefully resuspended in 375 microL Low Salt Buffer (LSB; 20 mM Tris, 10 mM NaCl, 3 mM MgCl2, pH 7.4) + RNasin (40 U/mL) + DTT (10 nM) for 3 minutes on ice.  125 microL LSB Lysis buffer (LSB containing 200 mM sucrose and 1.2% Triton-X100) was then added and cells disrupted by pipetting.  The mixture was then transferred to 1.7 mL microfuge tubes and centrifuged (20,000 x g, 1 minute, 4 degrees C) to remove nuclei and cellular debris.  The supernatant was transferred to a new tube containing 50 microL LSB + RNasin (40 U/mL) + DTT (10 nM) and 15 microL 5M NaCl.  This mixture was then layered onto prepoured gradients (15-50% sucrose in LSB).  Gradients were spun at 43,700 rpm for 90 minutes (4 degrees C) and then run on a density gradient flow cell fractionator (Isco) to obtain ribosomal profiles.  Ribosomal fractions corresponding to monosomes (M) and polysomes (P) (Figure 1) were collected into Trizol LS (Invitrogen) and total RNA isolated according to the directions of the manufacturer.  Total RNA was reverse transcribed using oligo dT primers in the presence of 33P dCTP to generate labeled cDNA probes from the M and P associated mRNA fractions of control and stimulated cells and these probes used to detect specific array elements on four identical Research Genetics (ResGen) Named Human Gene cDNA GeneFilters (GF 211, Invitrogen).  These filters were washed using high stringency conditions and exposed to phosphorimaging screens for analysis using Pathways 3, proprietary software included with the ResGen filters.   Hybridization signals were normalized to the average intensity of the hybridization signals from the particular array filter to correct for differing global hybridization efficiencies between filters.  To determine if a particular hybridization signal was significantly above background levels, an initial screen of the array elements was conducted by dividing the hybridization signal of an individual cDNA spot by the background signal on particular filter.  Background signal levels were determined using the Pathways 3 analysis software that measures the signal intensity of the area between cDNA spots and averages that signal over the area of the entire array.  Array elements were considered to be distinguishable from background if the hybridization signal/background ratio was greater than 1.1.  Signal intensities from each filter were normalized by dividing the raw signal intensity of the cDNA spot by the average signal intensity for the entire filter.  The translation state (TS) of each qualifying array element was determined by dividing the normalized signal intensity of the element in the polysome fraction by the normalized signal intensity of the element in the monosome fraction:  TS=P/M.  The Translation Index (TLI), a measure of the change of translation states of individual genes, was determined by calculating the ratio of the treated cell TS to the control cell TS; TLI = TSe/TSc where e represents experimental or treated conditions and c represents controls.  In this scenario, a gene that is not regulated by translation would have a TLI value of 1, actively translated genes would have a TLI value greater than 1, whereas a poorly translated or under-translated gene would have a TLI value less than 1.  Arbitrary TLI values of 2.5 and 0.4 were used to categorize individual array elements as being translationally regulated. A measure of transcriptional activity was also determined from the signal intensities of the array elements.  Summing the normalized signal intensities from the M and P fractions of the experimental cells and dividing that value by the sum of the normalized signal intensities from the control fractions calculated the Transcription Index (TCI).  Therefore, TCI = (Me + Pe)/(Mc + Pc).  If transcript levels for a specific gene do not change, the TCI value for that gene will be 1.  Genes that are transcribed upon stimulation of the cells will have a TCI value greater than 1, while genes that are not transcribed or transcription is down-regulated upon stimulation will have a TCI value less than 1.  Arbitrary values of 2 and 0.5, a doubling or halving of the amount of mRNA for that element, respectively, were used to categorize individual array elements as being transcriptionally regulated.","AE_experiment_type":"","AE_species":"NCBITAXON\u001fhttp://purl.obolibrary.org/obo/NCBITaxon_9606","AE_name":"Translation State Array Analysis of Thrombin Stimulated Human Endothelial Cells"},{"AE_description":"Comparison of mRNA expression in human EPC vs. HUVEC vs. human monocytes. Cell-type specific gene expression under basal cell culture conditions (no stimulation). The hybridization was performed with three samples of EPC vs. three samples of HUVEC vs. 3 samples of CD14+ monocytes. â€¢ The origin of the biological sample:; Human endothelial progenitor cells (EPC): EPC were ex vivo cultivated from human peripheral blood-derived mononuclear cells (PBMC). PBMC were isolated by density gradient centrifugation from healthy human volunteers as previously described (Dimmeler et al., 2001). Pooled human umbilical vein endothelial cells (HUVEC) were purchased from Cambrex (Verviers, Belgium). CD14+ monocytes were purified from PBMC by positive selection with anti-CD14-microbeads (Miltenyi Biotec, Bergisch-Gladbach, Germany). Purity assessed by FACS analysis was greater than 95%. â€¢ Manipulation of biological samples and protocols used: for example, growth conditions, treatments, separation techniques:; EPC:; 8000000 PBMC/ml were plated on human fibronectin (Sigma, Taufkirchen, Germany) and maintained in endothelial basal medium (Cambrex) with EGM SingleQuots and 20% fetal calf serum (FCS). After 3 days, nonadherent cells were removed and adherent cells were incubated in fresh medium for 24 h before starting experiments. HUVEC:; HUVEC were cultured in endothelial basal medium (Cambrex) supplemented with hydrocortisone, bovine brain extract, gentamicin, amphotericin B, epidermal growth factor, and 10% FCS until the third passage according to the manufacturerâ€™s recommendations. CD14+ monocytes: No culture. â€¢ Protocol for preparing the hybridization extract: for example, the RNA or DNA extraction and purification protocol. Total RNA was extracted from EPC, HUVEC, and CD14+ monocytes using the RNeasy cleanup system (Qiagen, Hilden, Germany) according to the manufacturer's protocol. Quantity and quality of total RNA was analyzed by the 2100 Bioanalyzer system (Agilent Technologies, Waldbronn, Germany) and agarose gel electrophoresis. â€¢ Labeling protocol(s):; The detailed protocol for the sample preparation and microarray processing is available from Affymetrix (Santa Clara, CA). Briefly, 10 Âµg of purified total RNA was reverse transcribed by Superscript II reverse transcriptase (Life Technologies, Grand Island, NY) using T7-(dT)24 primer containing a T7 RNA polymerase promoter. After synthesis of the second complementary DNA (cDNA) strand, this product was used in an in vitro transcription reaction to generate biotinylated complementary RNA (cRNA). â€¢ The protocol and conditions used during hybridization, blocking and washing:; Fifteen micrograms of fragmented, biotinylated cRNA were hybridized to a HG-U95Av2 microarray (Affymetrix Inc.) for 16 hours at 45Â° C with constant rotation at 60 rpm. This high-density oligonucleotide array targets 9,670 human genes as selected from the National Center for Biotechnology Information (NCBI) Gene Bank database with a total of 12,000 oligonucleotide sets. Each microarray was used to assay a single sample. After hybridization, the microarray was washed and stained on an Affymetrix fluidics station and scanned with an argon-ion confocal laser, with a 488 nm emission and detection at 570 nm. â€¢ GeneChip image analysis was performed using the Microarray Analysis Suite 5.0 (Affymetrix, Inc.). Expression data were analyzed utilizing the GeneSpringâ„¢ software version 4.2 (Silicon Genetics Inc., San Carlos, CA).","AE_experiment_type":"EFO\u001fhttp://www.ebi.ac.uk/efo/EFO_0002768","AE_species":"NCBITAXON\u001fhttp://purl.obolibrary.org/obo/NCBITaxon_9606","AE_name":"Transcription profiling of human endothelial progenitor cells (EPC) vs. human umbilical vein endothelial cells (HUVEC) vs. CD14+ monocytes"},{"AE_description":"RNA helicase A (RHA) binds its target transcripts at the post-transcriptional control element (PCE) located in the 5â€™ untranslated region (UTR). This interaction represents an â€œRNA switchâ€ that regulates protein synthesis. Down regulation of RHA by siRNAs was used to identify transcripts with RHA-dependent translation. Reduced accumulation of RNA in polysomes was monitored with microarrays. Changes in cytoplasmic RNA steady state abundance was monitored as well. Sixty nine genes exhibit decreased transcript polysome association when subjected to RHA downregulation. A majority of the transcripts that experienced a reduction in the polysome fraction had no significant change in their cytoplasmic abundance (45 genes).  Keywords: gene expression array-based Cytoplasmic lysates of cells treated with RHA targeted or non-silencing control siRNAs were separated by sucrose density gradient centrifugation. Ribosomal RNA profiles were generated, fractions containing polysomes were collected, and RNA was extracted.","AE_experiment_type":"EFO\u001fhttp://www.ebi.ac.uk/efo/EFO_0002768","AE_species":"NCBITAXON\u001fhttp://purl.obolibrary.org/obo/NCBITaxon_9534","AE_name":"Changes in polysome loading as a consequence of RHA downregulation"},{"AE_description":"Sewage samples were collected and concentrated for Human and animal viruses. Viruses were cultured on Buffalo Green Monkey Cells (BGMK) and their genomic DNA/RNA were extracted and labeled with Cy3 and Cy5 respectively. Labeled DNA/RNA were hybridized unto the array and signals generated were analyzed to indicate the presence of target viruses. Keywords: Detection of pathogens within environmental sample (Viruses) Environmental viruses were concentrated using organic flocculation with Beef Extract supplemented with glycine. Viruses were concentrated using 2 successive rounds of centrifugation and resuspended in Sodium Phosphate buffer. Viral nucleic acid was extracted, labeled and hybridized unto the microarray to determine the presence of target viruses within the sample.","AE_experiment_type":"","AE_species":"NCBITAXON\u001fhttp://purl.obolibrary.org/obo/NCBITaxon_10239","AE_name":"Detection of Human Viruses from wastewater sewage extracts"},{"AE_description":"Wild-type Campylobacter jejuni NCTC11168 was grown in 2 homemade, parallel chemostats. Cells were grown in 125 ml volumes of Mueller-Hinton Broth where the growth rate was controlled by the dilution rate  (0.1 h-1). A gas mix of 80% nitrogen, 10% carbon dioxide and 10% oxygen was pumped into the head-space of the vessels at a rate of 0.25 l/min to obtain a microaerobic environment and the culture was maintained at 42 ÂºC via a water jacket. Cells were grown as above until steady-state had been reached. At steady-state one of the cultures was exposed to 10 uM NOC-5 & -7 (final concentration) for a period of 10 min. Samples of both the control and stressed cultures were harvested into phenol/ethanol to stabilize the RNA and total RNA was purified using Qiagenâ€™s RNeasy Mini kit (as recommended by the suppliers) prior to use in microarray analysis. Equal quantities of RNA from control and GSNO-supplemented cultures were labelled using nucleotide analogues of dCTP containing either Cy3 or Cy5 fluorescent dyes. The average signal intensity and local background correction were obtained using a commercially available software package from Biodiscovery, Inc (Imagene, version 4.0 and GeneSight, version 4). The mean values from each channel were log2 transformed and normalised using the Lowess method to remove intensity-dependent effects in the log2(ratios) values. The Cy3/Cy5 fluorescent ratios were calculated from the normalized values. C. jejuni NCTC11168 was grown in two parallel custom-built chemostats based on Sigma Proculture Dynalift spinner flasks with a working volume of 125 ml, and the dilution rate (which at steady state is equal to the specific growth rate) of 0.1 h-1. A gas mix of 10% O2, 10% CO2 and 80% N2 was passed into the head space of the culture vessel at a rate of 0.25 l min-1 to obtain a microaerobic atmosphere, and silicone water jackets maintained the temperature at 42 ÂºC. The flasks were placed on KMO 2 basic IKA-Werke stirrers (arbitrary setting, 260; Scientific Laboratory Supplies), which gaveeffective transfer of O2 from the gaseous atmosphere to the culture by virtue of a stable vortex. A fresh overflow sample was used to check the pH at the time of harvest (which was consistently between 6.6 and 7) and to ensure that there was no contamination by plating a 20 ?l sample onto both nutrient agar plates incubated at 37 Â°C at atmospheric oxygen, and Mueller-Hinton plates incubated at 42 Â°C in the microaerophilic work station. When staedy-state was reached, as verified by collecting at least 5 culture volumes, NOC-5 & -7 (final concentration 10 uM) was added to one culture. After a further 10 min the 125 ml volume was separated into 3 equal volumes and each was mixed immediately on ice with 3.56 ml 100% ethanol and 185 ml phenol to stabilize the RNA. The cells were subsequently harvested by centrifugation. Total RNA was purified by using a Qiagen RNeasy Mini kit as recommended by the supplier.   cDNA synthesis was carried out using 12 Âµg of starting material, which was primed with 9 Âµg of pd(N)6 random hexamers (Amersham Biosciences). Reaction mixtures (20 Âµl) containing 0.5 mM ATP, 0.5 mM TTP, 0.5 mM GTP, 0.2 mM CTP, and 1 mM Cy3- or 1 mM Cy5-dCTP were incubated for 2 h at 42 Â°C with 200 U of Superscript II RNase H reverse transcriptase (Invitrogen). Following synthesis, cDNA was purified by using a PCR purification kit (Qiagen) to remove the unincorporated deoxynucloside triphosphates, fluorescent dye, and primers. Equal volumes of cDNA were combined and evaporated to near dryness with an SPD121P Speed Vac (Thermon Savant). For hybridisation to the microarray slides, cDNA was resuspended in an appropriate volume of salt-based hybridization buffer (provided by Ocimum Biosolutions). Prior to addition to the slides, cDNA samples were heated to 95 Â°C for 3 min, and then placed on ice for no more than 3 min. The slides were placed in MWG Biotech hybridization chambers and incubated for 16 to 24 h in a shaking water bath at 110 rpm and 42 Â°C.  Following incubation , the slides were washed sequentially for 5 min in 2x SSC-1 % sodium dodecyl sulphate, 1x SSC, 0.5x SSC and 0.1x SSC at 37 Â°C with gentle agitation (1x SSC is 0.15 M NaCl plus 0.015 M sodium citrate). The slides were dried by centrifugation at 254 xg for 5 min and subsequently scanned with an Affymetrix 428 scanner.  The average signal intensity and local background correction were obtained by using commercially available software from Biodiscovery, Inc. (Imagine version 4.0, and Genesight, version 3.5). The mean values from each channel were log2 transformed and normalized by using the subtract-by-mean method to remove intensity-dependent effects in the log2 (ratio) values. The Cy3/Cy5 fluorescence ratios were calculated from the normalized values. Biological experiments (i.e. chemostat growth with and without 0.25 mM GSNO addition) were carried out three times and a dye swap was performed for two of the three experiments which provided two technical repeats, one each for two of the three biological repeats. Data from the independent experiments were combined. Genes that were differentially expressed ? twofold and displayed and P value of ? 0.05 (as determined by a t test) were defined as being statistically significantly differentially transcribed.","AE_experiment_type":"","AE_species":"NCBITAXON\u001fhttp://purl.obolibrary.org/obo/NCBITaxon_197","AE_name":"Global responses of Campylobacter jejuni to nitric oxide releasing compounds"},{"AE_description":"RNA-coimmunopurifications with TAP-tagged Puf proteins from Saccharomyces cereviseae. Untagged strain  (BY4741) served as a control. Cells were grown to midlog phase and harvested by centrifugation. TAP-tagged Puf proteins were affinity purified from cell-free extracts with IgG sepharose and eluted with TEV protease. RNA was isolated from extract (=input)and from purified protein samples by phenol-chloroform extraction. RNA samples were reverse transcribed using a mixture of oligo-dT and random nonamer oligos in the presence of amino-allyl dUTP/ dNTP mixture. cDNAs were fluorescently labeled and hybridized on yeast DNA microarrays over night at 65 degrees. For a detailed procedure see http://microarray-pubs.stanford.edu/yeast_puf and also Gerber AP et al. PLoS Biology, 2004.","AE_experiment_type":"EFO\u001fhttp://www.ebi.ac.uk/efo/EFO_0002760","AE_species":"NCBITAXON\u001fhttp://purl.obolibrary.org/obo/NCBITaxon_4932","AE_name":"Chromatin immunoprecipitation with TAP-tagged Puf proteins from Saccharomyces cereviseae"},{"AE_description":"Saxitoxin is a potent neurotoxin produced by several species of dinoflagellates and cyanobacteria. The molecular target of saxitoxin in higher eukaryotes is the voltage-gated sodium channel; however, its target in lower eukaryotic organisms remains unknown. The goal of this study was to obtain the transcriptional fingerprint of the model lower eukaryote Saccharomyces cerevisiae upon exposure to saxitoxin to identify potential genes suitable for biomarker development. Microarray analyses identified multiple genes associated with copper and iron homeostasis and sulfur metabolism as significantly differentially expressed upon exposure to saxitoxin; these results were verified with quantitative reverse-transcriptase PCR (qRT-PCR). Additionally, the qRT-PCR assays were used to generate expression profiles in a subset of the differentially regulated genes across multiple exposure times and concentrations, the results of which demonstrated that overall, genes tended to respond in a consistent manner to the toxin. In general, the genes encoding the metallothioneins CUP1 and CRS5 were induced following exposure to saxitoxin, while those encoding the ferric/cupric reductase FRE1 and the copper uptake transporter CTR1 were repressed. The gene encoding the multicopper ferroxidase FET3, part of the high-affinity iron uptake system, was also induced in all treatments, along with the STR3 gene, which codes for the cystathionine beta-lyase found in the methionine biosynthetic pathway.   Experiment Overall Design: Cultures of S. cerevisiae S288C were grown to OD600=1.0 in YPD (1% yeast extract, 2% peptone, 2% dextrose). Four hundred microliters from individual 10 mL overnight cultures were transferred to sterile tubes and saxitoxin added to a final concentration of 16 ÂµM. Sample were mixed gently and incubated at 30Â° C for 45 min. Controls consisted of deionized water. Three biological replicates were performed under these conditions for microarray analysis. Cells were harvested via sequential centrifugation and placed immediately in liquid nitrogen.Total RNA was extracted from frozen cell pellets using a protocol that combined the hot acid phenol method with the RNEasy kit. Total RNA was processed by the Affymetrix Core Facility at the University of Tennessee (Knoxville, TN) following the protocols of the Affymetrix GeneChip Expression Analysis Technical Manual. MAS5.0 was used to provide present/absent calls; genes absent in all six samples were then removed from the data set. Intensity values for all transcripts were obtained from the scanned image files of the chips. The intensity values of the six arrays were background-corrected and normalized with the GC-RMA algorithm.","AE_experiment_type":"EFO\u001fhttp://www.ebi.ac.uk/efo/EFO_0002768","AE_species":"NCBITAXON\u001fhttp://purl.obolibrary.org/obo/NCBITaxon_4932","AE_name":"Transcription orofiling of Saccharomyces cerevisiae upon exposure to aaxitoxin"},{"AE_description":"To identify mRNAs associated with human PUM2 protein, we used a modified Ribonucleoprotein-ImmunoPrecipitation Microarray (RIP-Chip) approach on HeLa S3 cancer cells that express PUM2. PUM ribonucleoprotein (RNP) complexes were captured from cell-free extracts with specific anti PUM2 antibody coupled to protein A sepharose beads, and then eluted with SDS-EDTA. To control for non-specifically enriched RNAs, the same procedure was performed with beads that were not coupled with immunoprecipitating antibodies (mock samples). Total RNA was isolated from cell extracts and immunopurified samples with the mirVanaTM PARISTM kit (Ambion). RNA was quantified with a NanoDrop device (Witeg AG). Poly-adenylated RNAs were amplified in the presence of aminoallyl-UTP with Amino Allyl MessageAmp II aRNA kit (Ambion). For this purpose, 500 ng total RNA from extracts and half (50-100 ng) of the immunopurified RNAs were used for amplification. 8 ug of the amplified RNAs (aaRNA) were fluorescently labeled with NHS-monoester Cy3 and Cy5 dyes (GE HealthSciences), except for mock RNA samples, where an aaRNA amount proportional to the yield obtained from corresponding PUM affinity isolates was used.For PUM2 RIPs, we performed four biological replicates but omitted the dye swaps due to the lower aaRNA obtained after amplification (~10 ug aaRNA from PUM2 RIPs, 9 ug aaRNA from mock RIPs). The Cy3- and Cy5-labeled aaRNA samples were mixed and hybridized to human cDNA microarrays. Detailed methods for microarray experiments are available at http://cmgm.stanford.edu/pbrown/protocols/index.html. cDNA microarrays were produced by the Stanford Functional Genomic Facility and contained 43,197 human probes representing 26,524 Unigene cluster IDs (12,466 ENSEMBL annotated genes) spotted on Corning Ultra GAPS slides. Spotted cDNAs were cross-linked with 65 mJ of UV irradiation on slides, which were then post-processed for 1 hour at 42Â°C in pre-hybridization solution (5x SSC, 0.1% SDS, 0.1 mg/ml BSA), washed twice in 400 ml of 0.1x SSC for 5 min, dunked in 400 ml ultrapure water for 30 sec, and dried by centrifugation at 550 rpm for 5 min. Slides were used the same day.  Cy3- and Cy5-labeled aaRNA probes were mixed and applied to arrays in hybridization solution (3x SSC, 20 ug poly(A) RNA [Invitrogen], 20 ug yeast tRNA [Invitrogen], 20 ug Human Cot-1 DNA [Invitrogen], 20 mM HEPES [pH 7.0] and 0.3% SDS) for 18 h at 65Â°C. The arrays were then washed sequentially in 400 ml of 2x SSC with 0.1% SDS, 1x SSC, and 0.2x SSC. The first wash was performed for 5 min at 65Â°C, the subsequent washes were performed for 5 min at RT. The arrays were dried by centrifugation and immediately scanned with an AxonScanner 4200A (Molecular Devices). Data were collected using GENEPIX 5.1 (Molecular Devices). Arrays were normalized computationally by the Stanford Microarray Database (SMD). The data were filtered for signal over background of greater than 1.5 in the channel measuring aaRNA from extract, and only features that met these criteria in > 50% of the arrays were included for further analysis. Log2 median ratios were retrieved and exported into Microsoft Excel.  To identify transcripts that were specifically enriched by association with PUM2, we performed two class Significance Analysis of Microarrays (SAM) on median centered arrays. A replicate experimental design type is where a series of replicates are performed to evaluate reproducibility or as a pilot study to determine the appropriate number of replicates for a subsequent experiments. Antibody used in IP: 50 ug rabbit anti-PUMILIO 2 (Bethyl Laboratories, #300-202A)  Keywords: replicate_design Computed","AE_experiment_type":"EFO\u001fhttp://www.ebi.ac.uk/efo/EFO_0002768","AE_species":"NCBITAXON\u001fhttp://purl.obolibrary.org/obo/NCBITaxon_9606","AE_name":"mRNAs associated with human PUM2 protein"},{"AE_description":"Pathogenic biofilms have been associated with persistent infections due to their high resistance to antimicrobial agents. To identify non-toxic biofilm inhibitors for enterohemorrhagic Escherichia coli O157:H7, indole-3-acetaldehyde was used and reduced E. coli O157:H7 biofilm formation. Global transcriptome analyses revealed that indole-3-acetaldehyde most repressed two curli operons, csgBAC and csgDEFG, and induced tryptophanase (tnaAB) in E. coli O157:H7 biofilm cells. Electron microscopy showed that indole-3-acetaldehyde reduced curli production in E. coli O157:H7. Together, this study shows that Actinomycetales are an important resource of biofilm inhibitors as well as antibiotics. For the microarray experiments, E. coli O157:H7 EDL933 was inoculated in 100 ml of LB in 250 ml flasks with overnight cultures that were diluted at 1:100. Cells were shaken with 10g of glass wool at 250 rpm and 37Â°C for 7 hrs.  Cells were immediately chilled with dry ice and 95% ethanol (to prevent RNA degradation) for 30 sec before centrifugation in 50 ml centrifuge tubes at 13,000 g for 2 min; cell pellets were frozen immediately with dry ice and stored -80Â°C.  RNA was isolated using Qiagen RNeasy mini Kit (Valencia, CA, USA). To eliminate DNA contamination, Qiagen RNase-free DNase I was used to digest DNA. RNA quality was assessed by Agilent 2100 bioanalyser using the RNA 6000 Nano Chip (Agilent Technologies, Amstelveen, The Netherlands), and quantity was determined by ND-1000 Spectrophotometer (NanoDrop Technologies, Inc., DE, USA).","AE_experiment_type":"EFO\u001fhttp://www.ebi.ac.uk/efo/EFO_0002768","AE_species":"NCBITAXON\u001fhttp://purl.obolibrary.org/obo/NCBITaxon_562","AE_name":"Indole-3-Acetaldehyde Inhibits Escherichia coli O157:H7 Biofilm Formation by Reducing Curli Formation"},{"AE_description":"Batch cultures of Wild-type C. jejuni NCTC11168 were grown in 150 ml volumes of Mueller-Hinton broth in 250 ml baffled flasks. Microaerophilic conditions were generated using a MACS-VA500 microaerophilic work station (10 % O2, 10 % CO2, 80 % N2) from Don Whitley Scientific, Ltd which also maintained the growth temperature at 42 ÂºC. When mid-exponential phase was reached 0.25 mM GSNO was added to half of the cultures. After a 10 minute exposure samples of both treated and untreated cells were harvested into phenol/ethenol to stabilize the RNA and total RNA was purified using Qiagenâ€™s RNeasy Mini kit (as recommended by the suppliers) prior to use in microarray analysis. A C. jejuni cgb mutant was grown in two parallel custom-built chemostats based on Sigma Proculture Dynalift spinner flasks with a working volume of 125 ml, and the dilution rate (which at steady state is equal to the specific growth rate) of 0.1 h-1. A gas mix of 10% O2, 10% CO2 and 80% N2 was passed into the head space of the culture vessel at a rate of 0.25 l min-1 to obtain a microaerobic atmosphere, and silicone water jackets maintained the temperature at 42 ÂºC. The flasks were placed on KMO 2 basic IKA-Werke stirrers (arbitrary setting, 260; Scientific Laboratory Supplies), which gaveeffective transfer of O2 from the gaseous atmosphere to the culture by virtue of a stable vortex. A fresh overflow sample was used to check the pH at the time of harvest (which was consistently between 6.6 and 7) and to ensure that there was no contamination by plating a 20 ?l sample onto both nutrient agar plates incubated at 37 Â°C at atmospheric oxygen, and Mueller-Hinton plates incubated at 42 Â°C in the microaerophilic work station. When staedy-state was reached, as verified by collecting at least 5 culture volumes, GSNO (final concentration 0.25 mM) was added to one culture. After a further 10 min the 125 ml volume was separated into 3 equal volumes and each was mixed immediately on ice with 3.56 ml 100% ethanol and 185 ml phenol to stabilize the RNA. The cells were subsequently harvested by centrifugation. Total RNA was purified by using a Qiagen RNeasy Mini kit as recommended by the supplier.   cDNA synthesis was carried out using 12 Âµg of starting material, which was primed with 9 Âµg of pd(N)6 random hexamers (Amersham Biosciences). Reaction mixtures (20 Âµl) containing 0.5 mM ATP, 0.5 mM TTP, 0.5 mM GTP, 0.2 mM CTP, and 1 mM Cy3- or 1 mM Cy5-dCTP were incubated for 2 h at 42 Â°C with 200 U of Superscript II RNase H reverse transcriptase (Invitrogen). Following synthesis, cDNA was purified by using a PCR purification kit (Qiagen) to remove the unincorporated deoxynucloside triphosphates, fluorescent dye, and primers. Equal volumes of cDNA were combined and evaporated to near dryness with an SPD121P Speed Vac (Thermon Savant). For hybridisation to the microarray slides, cDNA was resuspended in an appropriate volume of salt-based hybridization buffer (provided by Ocimum Biosolutions). Prior to addition to the slides, cDNA samples were heated to 95 Â°C for 3 min, and then placed on ice for no more than 3 min. The slides were placed in MWG Biotech hybridization chambers and incubated for 16 to 24 h in a shaking water bath at 110 rpm and 42 Â°C.  Following incubation , the slides were washed sequentially for 5 min in 2x SSC-1 % sodium dodecyl sulphate, 1x SSC, 0.5x SSC and 0.1x SSC at 37 Â°C with gentle agitation (1x SSC is 0.15 M NaCl plus 0.015 M sodium citrate). The slides were dried by centrifugation at 254 xg for 5 min and subsequently scanned with an Affymetrix 428 scanner.  The average signal intensity and local background correction were obtained by using commercially available software from Biodiscovery, Inc. (Imagine version 4.0, and Genesight, version 3.5). The mean values from each channel were log2 transformed and normalized by using the subtract-by-mean method to remove intensity-dependent effects in the log2 (ratio) values. The Cy3/Cy5 fluorescence ratios were calculated from the normalized values. Biological experiments (i.e. chemostat growth with and without 0.25 mM GSNO addition) were carried out three times and a dye swap was performed for two of the three experiments which provided two technical repeats, one each for two of the three biological repeats. Data from the independent experiments were combined. Genes that were differentially expressed ? twofold and displayed and P value of ? 0.05 (as determined by a t test) were defined as being statistically significantly differentially transcribed.","AE_experiment_type":"","AE_species":"NCBITAXON\u001fhttp://purl.obolibrary.org/obo/NCBITaxon_197","AE_name":"Transcriptome analysis of GSNO treated batch cultures of C. jejuni"}]}
  EOS
  DOCUMENT = %q({"documentId":"E-GEOD-38707","AE_description":"Malaria is caused by Plasmodium parasites, which are transmitted via the bites of infected Anopheline mosquitoes. Midgut invasion is a major bottleneck for Plasmodium development inside the mosquito vectors as a rapidly responding immune system recognizes ookinetes and recruits killing factors from the midgut and surrounding tissues, dramatically reducing the population of invading ookinetes before they can successfully traverse the midgut epithelium. Understanding molecular details of the parasite-vector interactions requires precise measurement of nascent protein synthesis in the mosquito during Plasmodium infection. Current expression profiling primarily monitors alterations in steady-state levels of mRNA, but does not address the equally critical issue of whether the proteins encoded by the mRNAs are actually synthesized. In this study, we used sucrose density gradient centrifugation to isolate actively translating Anopheles gambiae mRNAs based upon their association with polyribosomes (polysomes). The proportion of individual gene transcripts associated with polysomes, which is determined by RNA deep sequencing, reflects mRNA translational status. This approach led to identification of 1017 mosquito transcripts that were primarily regulated at the translational level after ingestion of Plasmodium falciparum-infected blood. Caspar, a negative regulator of the NF-kappaB transcription factor Rel2, appears to be substantially activated at the translational levels during Plasmodium infection. In addition, transcripts of Dcr1, Dcr2 and Drosha, which are involved in small RNA biosynthesis, exhibited enhanced associations with polysomes after P. falciparum challenge. This observation suggests that mosquito microRNAs may play an important role in reactions against Plasmodium invasion. We analyzed both total cellular mRNAs and mRNAs that are associated with polysomes to simultaneously monitor transcriptomes and nascent protein synthesis in the mosquito. This approach provides more accurate information regarding the rate of protein synthesis, and identifies some mosquito factors that might have gone unrecognized because expression of these proteins is regulated mainly at the translational level rather than at the transcriptional level after mosquitoes ingest a Plasmodium-infected blood meal. Midguts from female Anopheles gambiae mosquitoes were dissected at around 24 h after ingestion of P. falciparum-infected blood. Mosquitoes fed on uninfected blood were used as control. A portion of the midguts were used for isolation of total cellular RNA. Extracts of the remaining midguts were fractionated over sucrose density gradients, and fifteen fractions were collected from the top of each gradient. Non-polysomal fractions, as well as polysomal fractions were combined, respectively, to obtain two RNA pools per gradient for three independent experiments. The first, last and the sample between non-polysomal and polysomal, were all discarded to ensure pure pools from each set. A fourth experiment was conducted where the polysomal and non-polysomal samples were not pooled, to be used later for qRT-PCR. The mRNA levels of individual mosquito genes in polysome (PS) fractions, nonpolysome (NP) fractions and unfractionated steady-state (total) RNA were determined using high throughput RNA deep sequencing. Each RNA pool generated 2.1-9.8 million raw read .  Signal intensities of transcripts from each PS pool were compared to those of transcripts from a matching NP pool. To quantify the translational status of individual mRNA species, we define the relative PS loading [PL=P/(P+NP)] as the extent of mRNA association with polysomes. PL values were compared between the mosquitoes fed on P. falciparum-infected or -uninfected blood.","AE_experiment_type":"EFO\u001fhttp://www.ebi.ac.uk/efo/EFO_0003738","AE_species":"NCBITAXON\u001fhttp://purl.obolibrary.org/obo/NCBITaxon_7165","AE_name":"Translational regulation of Anopheles gambiae mRNAs in the midgut during Plasmodium falciparum infection"})

  def self.before_suite
    @@ont = self.new("before_suite").create_ontologies_and_submissions(ont_count: 1, submission_count: 1, process_submission: true)[2].first
  end

  def test_resources
    resources = MultiJson.load(RESOURCES)
    ResourceIndex::Resource.stub :populated, resources do
      get "/resource_index/resources"
      assert last_response.ok?
      assert_equal resources, MultiJson.load(last_response.body)
    end
  end

  def test_resource
    resources = resources_from_hash(MultiJson.load(RESOURCES))
    pm = resources.select {|r| r.acronym.eql?("PM")}.first
    ResourceIndex::Resource.stub :all, resources do
      get "/resource_index/resources/NOT_REAL"
      assert last_response.status == 404
      get "/resource_index/resources/PM"
      assert last_response.ok?
      pm_rest = MultiJson.load(last_response.body)
      assert_equal pm.acronym, pm_rest["acronym"]
      assert_equal pm.description, pm_rest["description"]
      assert_equal pm.mainField, pm_rest["mainField"]
      assert_equal pm.homepage, pm_rest["homepage"]
      assert_equal pm.lookupURL, pm_rest["lookupURL"]
      assert_equal pm.logo, pm_rest["logo"]
      assert_equal pm.count, pm_rest["count"]
      assert_equal pm.updated, pm_rest["updated"]
    end
  end

  def test_counts
    counts = MultiJson.load(COUNTS)
    Array.stub_any_instance :ri_counts, counts do
      get "/resource_index/counts?classes[TEST-ONT-0]=http://bro.com/resource"
      assert last_response.status == 404
      get "/resource_index/counts?classes[TEST-ONT-0]=http%3A%2F%2Fbioontology.org%2Fontologies%2FBiomedicalResourceOntology.owl%23Microarray_Scanner"
      assert last_response.ok?
      assert_equal counts, MultiJson.load(last_response.body)
    end
  end

  def test_resource_documents
    document = MultiJson.load(DOCUMENT)
    ResourceIndex::Document.stub :find, nil do
      get "/resource_index/resources/AE/documents/FAKE_DOC"
      assert last_response.status == 404
    end
    ResourceIndex::Document.stub :find, document do
      get "/resource_index/resources/AE/documents/E-GEOD-38707"
      assert last_response.ok?
      assert_equal document, MultiJson.load(last_response.body)
    end
  end

  def test_resource_search
    search = MultiJson.load(SEARCH)
    Array.stub_any_instance :ri_docs_page, search do
      get "/resource_index/resources/AE/search?classes[TEST-ONT-0]=http://bro.com/resource"
      assert last_response.status == 404
      get "/resource_index/resources/AE/search?classes[TEST-ONT-0]=http%3A%2F%2Fbioontology.org%2Fontologies%2FBiomedicalResourceOntology.owl%23Microarray_Scanner"
      assert last_response.ok?
      assert_equal search, MultiJson.load(last_response.body)
    end
  end

  def test_annotation_counts
    get "/resource_index/annotation_counts"
    assert last_response.ok?
    counts = MultiJson.load(last_response.body)
    assert_equal(["raw", "total", "time"].sort, counts.keys.sort)
  end

  private

  def resources_from_hash(resources)
    new_resources = []
    resources.each do |res_hash|
      new_resource = ResourceIndex::Resource.new
      res_hash.each {|k,v| new_resource.send("#{k}=", v)}
      new_resources << new_resource
    end
    new_resources
  end
end